@article{belbin_toward_2021,
	title = {Toward a fine-scale population health monitoring system},
	volume = {184},
	issn = {0092-8674},
	url = {https://www.sciencedirect.com/science/article/pii/S0092867421003652},
	doi = {10.1016/j.cell.2021.03.034},
	abstract = {Understanding population health disparities is an essential component of equitable precision health efforts. Epidemiology research often relies on definitions of race and ethnicity, but these population labels may not adequately capture disease burdens and environmental factors impacting specific sub-populations. Here, we propose a framework for repurposing data from electronic health records (EHRs) in concert with genomic data to explore the demographic ties that can impact disease burdens. Using data from a diverse biobank in New York City, we identified 17 communities sharing recent genetic ancestry. We observed 1,177 health outcomes that were statistically associated with a specific group and demonstrated significant differences in the segregation of genetic variants contributing to Mendelian diseases. We also demonstrated that fine-scale population structure can impact the prediction of complex disease risk within groups. This work reinforces the utility of linking genomic data to EHRs and provides a framework toward fine-scale monitoring of population health.},
	language = {en},
	number = {8},
	urldate = {2021-05-14},
	journal = {Cell},
	author = {Belbin, Gillian M. and Cullina, Sinead and Wenric, Stephane and Soper, Emily R. and Glicksberg, Benjamin S. and Torre, Denis and Moscati, Arden and Wojcik, Genevieve L. and Shemirani, Ruhollah and Beckmann, Noam D. and Cohain, Ariella and Sorokin, Elena P. and Park, Danny S. and Ambite, Jose-Luis and Ellis, Steve and Auton, Adam and Bottinger, Erwin P. and Cho, Judy H. and Loos, Ruth J. F. and Abul-Husn, Noura S. and Zaitlen, Noah A. and Gignoux, Christopher R. and Kenny, Eimear E.},
	month = apr,
	year = {2021},
	keywords = {biobanks, computational genomics, electronic health records, genetic ancestry, genomic medicine, health disparities, machine learning, population health},
	pages = {2068--2083.e11},
}

@article{littlejohns_uk_2020,
	title = {The {UK} {Biobank} imaging enhancement of 100,000 participants: rationale, data collection, management and future directions},
	volume = {11},
	copyright = {2020 The Author(s)},
	issn = {2041-1723},
	shorttitle = {The {UK} {Biobank} imaging enhancement of 100,000 participants},
	url = {https://www.nature.com/articles/s41467-020-15948-9},
	doi = {10.1038/s41467-020-15948-9},
	abstract = {UK Biobank is a population-based cohort of half a million participants aged 40–69 years recruited between 2006 and 2010. In 2014, UK Biobank started the world’s largest multi-modal imaging study, with the aim of re-inviting 100,000 participants to undergo brain, cardiac and abdominal magnetic resonance imaging, dual-energy X-ray absorptiometry and carotid ultrasound. The combination of large-scale multi-modal imaging with extensive phenotypic and genetic data offers an unprecedented resource for scientists to conduct health-related research. This article provides an in-depth overview of the imaging enhancement, including the data collected, how it is managed and processed, and future directions.},
	language = {en},
	number = {1},
	urldate = {2021-05-14},
	journal = {Nature Communications},
	author = {Littlejohns, Thomas J. and Holliday, Jo and Gibson, Lorna M. and Garratt, Steve and Oesingmann, Niels and Alfaro-Almagro, Fidel and Bell, Jimmy D. and Boultwood, Chris and Collins, Rory and Conroy, Megan C. and Crabtree, Nicola and Doherty, Nicola and Frangi, Alejandro F. and Harvey, Nicholas C. and Leeson, Paul and Miller, Karla L. and Neubauer, Stefan and Petersen, Steffen E. and Sellors, Jonathan and Sheard, Simon and Smith, Stephen M. and Sudlow, Cathie L. M. and Matthews, Paul M. and Allen, Naomi E.},
	month = may,
	year = {2020},
	note = {Number: 1
Publisher: Nature Publishing Group},
	pages = {2624},
}

@article{sudlow_uk_2015,
	title = {{UK} {Biobank}: {An} {Open} {Access} {Resource} for {Identifying} the {Causes} of a {Wide} {Range} of {Complex} {Diseases} of {Middle} and {Old} {Age}},
	volume = {12},
	issn = {1549-1676},
	shorttitle = {{UK} {Biobank}},
	url = {https://journals.plos.org/plosmedicine/article?id=10.1371/journal.pmed.1001779},
	doi = {10.1371/journal.pmed.1001779},
	abstract = {Cathie Sudlow and colleagues describe the UK Biobank, a large population-based prospective study, established to allow investigation of the genetic and non-genetic determinants of the diseases of middle and old age.},
	language = {en},
	number = {3},
	urldate = {2021-05-14},
	journal = {PLOS Medicine},
	author = {Sudlow, Cathie and Gallacher, John and Allen, Naomi and Beral, Valerie and Burton, Paul and Danesh, John and Downey, Paul and Elliott, Paul and Green, Jane and Landray, Martin and Liu, Bette and Matthews, Paul and Ong, Giok and Pell, Jill and Silman, Alan and Young, Alan and Sprosen, Tim and Peakman, Tim and Collins, Rory},
	month = mar,
	year = {2015},
	note = {Publisher: Public Library of Science},
	keywords = {Cohort studies, Global health, Intelligence tests, Magnetic resonance imaging, Prospective studies, Questionnaires, Research ethics, Scientists},
	pages = {e1001779},
}

@article{grinde_generalizing_2019,
	title = {Generalizing polygenic risk scores from {Europeans} to {Hispanics}/{Latinos}},
	volume = {43},
	issn = {1098-2272},
	url = {https://onlinelibrary.wiley.com/doi/abs/10.1002/gepi.22166},
	doi = {https://doi.org/10.1002/gepi.22166},
	abstract = {Polygenic risk scores (PRSs) are weighted sums of risk allele counts of single-nucleotide polymorphisms (SNPs) associated with a disease or trait. PRSs are typically constructed based on published results from Genome-Wide Association Studies (GWASs), and the majority of which has been performed in large populations of European ancestry (EA) individuals. Although many genotype-trait associations have generalized across populations, the optimal choice of SNPs and weights for PRSs may differ between populations due to different linkage disequilibrium (LD) and allele frequency patterns. We compare various approaches for PRS construction, using GWAS results from both large EA studies and a smaller study in Hispanics/Latinos: The Hispanic Community Health Study/Study of Latinos (HCHS/SOL, ). We consider multiple approaches for selecting SNPs and for computing SNP weights. We study the performance of the resulting PRSs in an independent study of Hispanics/Latinos from the Women’s Health Initiative (WHI, ). We support our investigation with simulation studies of potential genetic architectures in a single locus. We observed that selecting variants based on EA GWASs generally performs well, except for blood pressure trait. However, the use of EA GWASs for weight estimation was suboptimal. Using non-EA GWAS results to estimate weights improved results.},
	language = {en},
	number = {1},
	urldate = {2021-05-14},
	journal = {Genetic Epidemiology},
	author = {Grinde, Kelsey E. and Qi, Qibin and Thornton, Timothy A. and Liu, Simin and Shadyab, Aladdin H. and Chan, Kei Hang K. and Reiner, Alexander P. and Sofer, Tamar},
	year = {2019},
	note = {\_eprint: https://onlinelibrary.wiley.com/doi/pdf/10.1002/gepi.22166},
	keywords = {admixed populations, genetic diversity, linkage disequilibrium},
	pages = {50--62},
}

@article{cavazos_inclusion_2021,
	title = {Inclusion of variants discovered from diverse populations improves polygenic risk score transferability},
	volume = {2},
	issn = {2666-2477},
	url = {https://www.sciencedirect.com/science/article/pii/S2666247720300178},
	doi = {10.1016/j.xhgg.2020.100017},
	abstract = {The majority of polygenic risk scores (PRSs) have been developed and optimized in individuals of European ancestry and may have limited generalizability across other ancestral populations. Understanding aspects of PRSs that contribute to this issue and determining solutions is complicated by disease-specific genetic architecture and limited knowledge of sharing of causal variants and effect sizes across populations. Motivated by these challenges, we undertook a simulation study to assess the relationship between ancestry and the potential bias in PRSs developed in European ancestry populations. Our simulations show that the magnitude of this bias increases with increasing divergence from European ancestry, and this is attributed to population differences in linkage disequilibrium and allele frequencies of European-discovered variants, likely as a result of genetic drift. Importantly, we find that including into the PRS variants discovered in African ancestry individuals has the potential to achieve unbiased estimates of genetic risk across global populations and admixed individuals. We confirm our simulation findings in an analysis of hemoglobin A1c (HbA1c), asthma, and prostate cancer in the UK Biobank. Given the demonstrated improvement in PRS prediction accuracy, recruiting larger diverse cohorts will be crucial—and potentially even necessary—for enabling accurate and equitable genetic risk prediction across populations.},
	language = {en},
	number = {1},
	urldate = {2021-05-14},
	journal = {Human Genetics and Genomics Advances},
	author = {Cavazos, Taylor B. and Witte, John S.},
	month = jan,
	year = {2021},
	keywords = {GWAS, local ancestry, polygenic risk scores, population genetics, statistical genetics},
	pages = {100017},
}

@article{sirugo_missing_2019,
	title = {The {Missing} {Diversity} in {Human} {Genetic} {Studies}},
	volume = {177},
	issn = {0092-8674},
	url = {https://www.sciencedirect.com/science/article/pii/S0092867419302314},
	doi = {10.1016/j.cell.2019.02.048},
	abstract = {The majority of studies of genetic association with disease have been performed in Europeans. This European bias has important implications for risk prediction of diseases across global populations. In this commentary, we justify the need to study more diverse populations using both empirical examples and theoretical reasoning.},
	language = {en},
	number = {1},
	urldate = {2021-05-14},
	journal = {Cell},
	author = {Sirugo, Giorgio and Williams, Scott M. and Tishkoff, Sarah A.},
	month = mar,
	year = {2019},
	pages = {26--31},
}

@article{mordi_differential_2019,
	title = {Differential {Association} of {Genetic} {Risk} of {Coronary} {Artery} {Disease} with {Development} of {Heart} {Failure} with {Reduced} {Versus} {Preserved} {Ejection} {Fraction}: {A} {GoDARTS} {Mendelian} {Randomization} {Study} and {Meta}-{Analysis}},
	volume = {139},
	issn = {0009-7322},
	shorttitle = {Differential {Association} of {Genetic} {Risk} of {Coronary} {Artery} {Disease} with {Development} of {Heart} {Failure} with {Reduced} {Versus} {Preserved} {Ejection} {Fraction}},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6420119/},
	doi = {10.1161/CIRCULATIONAHA.118.038602},
	number = {7},
	urldate = {2021-05-13},
	journal = {Circulation},
	author = {Mordi, Ify R and Pearson, Ewan R and Palmer, Colin NA and Doney, Alexander SF and Lang, Chim C},
	month = feb,
	year = {2019},
	pmid = {30742529},
	pmcid = {PMC6420119},
	pages = {986--988},
}

@article{wand_improving_2021,
	title = {Improving reporting standards for polygenic scores in risk prediction studies},
	volume = {591},
	copyright = {2021 Springer Nature Limited},
	issn = {1476-4687},
	url = {https://www.nature.com/articles/s41586-021-03243-6},
	doi = {10.1038/s41586-021-03243-6},
	abstract = {Polygenic risk scores (PRSs), which often aggregate results from genome-wide association studies, can bridge the gap between initial discovery efforts and clinical applications for the estimation of disease risk using genetics. However, there is notable heterogeneity in the application and reporting of these risk scores, which hinders the translation of PRSs into clinical care. Here, in a collaboration between the Clinical Genome Resource (ClinGen) Complex Disease Working Group and the Polygenic Score (PGS) Catalog, we present the Polygenic Risk Score Reporting Standards (PRS-RS), in which we update the Genetic Risk Prediction Studies (GRIPS) Statement to reflect the present state of the field. Drawing on the input of experts in epidemiology, statistics, disease-specific applications, implementation and policy, this comprehensive reporting framework defines the minimal information that is needed to interpret and evaluate PRSs, especially with respect to downstream clinical applications. Items span detailed descriptions of study populations, statistical methods for the development and validation of PRSs and considerations for the potential limitations of these scores. In addition, we emphasize the need for data availability and transparency, and we encourage researchers to deposit and share PRSs through the PGS Catalog to facilitate reproducibility and comparative benchmarking. By providing these criteria in a structured format that builds on existing standards and ontologies, the use of this framework in publishing PRSs will facilitate translation into clinical care and progress towards defining best practice.},
	language = {en},
	number = {7849},
	urldate = {2021-05-13},
	journal = {Nature},
	author = {Wand, Hannah and Lambert, Samuel A. and Tamburro, Cecelia and Iacocca, Michael A. and O’Sullivan, Jack W. and Sillari, Catherine and Kullo, Iftikhar J. and Rowley, Robb and Dron, Jacqueline S. and Brockman, Deanna and Venner, Eric and McCarthy, Mark I. and Antoniou, Antonis C. and Easton, Douglas F. and Hegele, Robert A. and Khera, Amit V. and Chatterjee, Nilanjan and Kooperberg, Charles and Edwards, Karen and Vlessis, Katherine and Kinnear, Kim and Danesh, John N. and Parkinson, Helen and Ramos, Erin M. and Roberts, Megan C. and Ormond, Kelly E. and Khoury, Muin J. and Janssens, A. Cecile J. W. and Goddard, Katrina A. B. and Kraft, Peter and MacArthur, Jaqueline A. L. and Inouye, Michael and Wojcik, Genevieve L.},
	month = mar,
	year = {2021},
	note = {Number: 7849
Publisher: Nature Publishing Group},
	pages = {211--219},
}

@article{richter_genomic_2020,
	title = {Genomic analyses implicate noncoding de novo variants in congenital heart disease},
	volume = {52},
	issn = {1546-1718},
	doi = {10.1038/s41588-020-0652-z},
	abstract = {A genetic etiology is identified for one-third of patients with congenital heart disease (CHD), with 8\% of cases attributable to coding de novo variants (DNVs). To assess the contribution of noncoding DNVs to CHD, we compared genome sequences from 749 CHD probands and their parents with those from 1,611 unaffected trios. Neural network prediction of noncoding DNV transcriptional impact identified a burden of DNVs in individuals with CHD (n = 2,238 DNVs) compared to controls (n = 4,177; P = 8.7 × 10-4). Independent analyses of enhancers showed an excess of DNVs in associated genes (27 genes versus 3.7 expected, P = 1 × 10-5). We observed significant overlap between these transcription-based approaches (odds ratio (OR) = 2.5, 95\% confidence interval (CI) 1.1-5.0, P = 5.4 × 10-3). CHD DNVs altered transcription levels in 5 of 31 enhancers assayed. Finally, we observed a DNV burden in RNA-binding-protein regulatory sites (OR = 1.13, 95\% CI 1.1-1.2, P = 8.8 × 10-5). Our findings demonstrate an enrichment of potentially disruptive regulatory noncoding DNVs in a fraction of CHD at least as high as that observed for damaging coding DNVs.},
	language = {eng},
	number = {8},
	journal = {Nature Genetics},
	author = {Richter, Felix and Morton, Sarah U. and Kim, Seong Won and Kitaygorodsky, Alexander and Wasson, Lauren K. and Chen, Kathleen M. and Zhou, Jian and Qi, Hongjian and Patel, Nihir and DePalma, Steven R. and Parfenov, Michael and Homsy, Jason and Gorham, Joshua M. and Manheimer, Kathryn B. and Velinder, Matthew and Farrell, Andrew and Marth, Gabor and Schadt, Eric E. and Kaltman, Jonathan R. and Newburger, Jane W. and Giardini, Alessandro and Goldmuntz, Elizabeth and Brueckner, Martina and Kim, Richard and Porter, George A. and Bernstein, Daniel and Chung, Wendy K. and Srivastava, Deepak and Tristani-Firouzi, Martin and Troyanskaya, Olga G. and Dickel, Diane E. and Shen, Yufeng and Seidman, Jonathan G. and Seidman, Christine E. and Gelb, Bruce D.},
	month = aug,
	year = {2020},
	pmid = {32601476},
	pmcid = {PMC7415662},
	keywords = {Adolescent, Adult, Animals, Female, Genetic Predisposition to Disease, Genetic Variation, Genomics, Heart, Heart Defects, Congenital, Humans, Male, Mice, Middle Aged, Open Reading Frames, RNA, Untranslated, RNA-Binding Proteins, Transcription, Genetic, Young Adult},
	pages = {769--777},
}

@article{martin_critical_2018,
	title = {The critical needs and challenges for genetic architecture studies in {Africa}},
	volume = {53},
	issn = {0959-437X},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6494470/},
	doi = {10.1016/j.gde.2018.08.005},
	abstract = {Human genetic studies have long been vastly Eurocentric, raising a key question about the generalizability of these study findings to other populations. Because humans originated in Africa, these populations retain more genetic diversity, and yet individuals of African descent have been tremendously underrepresented in genetic studies. The diversity in Africa affords ample opportunities to improve fine-mapping resolution for associated loci, discover novel genetic associations with phenotypes, build more generalizable genetic risk prediction models, and better understand the genetic architecture of complex traits and diseases subject to varying environmental pressures. Thus, it is both ethically and scientifically imperative that geneticists globally surmount challenges that have limited progress in African genetic studies to date while meaningfully including African investigators, as greater inclusivity and enhanced research capacity affords enormous opportunities to accelerate genomic discoveries that translate more effectively to all populations. We review the advantages, challenges, and examples of genetic architecture studies of complex traits and diseases in Africa. For example, with greater genetic diversity comes greater ancestral heterogeneity; this higher level of understudied diversity can yield novel genetic findings, but some methods that assume homogeneous population structure and work well in European populations may work less well in the presence of greater heterogeneity in African populations. Consequently, we advocate for methodological development that will accelerate studies important for all populations, especially those currently underrepresented in genetics.},
	urldate = {2021-05-13},
	journal = {Current opinion in genetics \& development},
	author = {Martin, Alicia R. and Teferra, Solomon and Möller, Marlo and Hoal, Eileen G. and Daly, Mark J.},
	month = dec,
	year = {2018},
	pmid = {30240950},
	pmcid = {PMC6494470},
	pages = {113--120},
}

@article{majara_low_2021,
	title = {Low generalizability of polygenic scores in {African} populations due to genetic and environmental diversity},
	copyright = {© 2021, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/},
	url = {https://www.biorxiv.org/content/10.1101/2021.01.12.426453v1},
	doi = {10.1101/2021.01.12.426453},
	abstract = {{\textless}h3{\textgreater}Abstract{\textless}/h3{\textgreater} {\textless}p{\textgreater}African populations are vastly underrepresented in genetic studies but have the most genetic variation and face wide-ranging environmental exposures globally. Because systematic evaluations of genetic prediction had not yet been conducted in ancestries that span African diversity, we calculated polygenic risk scores (PRS) in simulations across Africa and in empirical data from South Africa, Uganda, and the UK to better understand the generalizability of genetic studies. PRS accuracy improves with ancestry-matched discovery cohorts more than from ancestry-mismatched studies. Within ancestrally and ethnically diverse South Africans, we find that PRS accuracy is low for all traits but varies across groups. Differences in African ancestries contribute more to variability in PRS accuracy than other large cohort differences considered between individuals in the UK versus Uganda. We computed PRS in African ancestry populations using existing European-only versus ancestrally diverse genetic studies; the increased diversity produced the largest accuracy gains for hemoglobin concentration and white blood cell count, reflecting large-effect ancestry-enriched variants in genes known to influence sickle cell anemia and the allergic response, respectively. Differences in PRS accuracy across African ancestries originating from diverse regions are as large as across out-of-Africa continental ancestries, requiring commensurate nuance.{\textless}/p{\textgreater}},
	language = {en},
	urldate = {2021-05-13},
	journal = {bioRxiv},
	author = {Majara, Lerato and Kalungi, Allan and Koen, Nastassja and Zar, Heather and Stein, Dan J. and Kinyanda, Eugene and Atkinson, Elizabeth G. and Martin, Alicia R.},
	month = jan,
	year = {2021},
	note = {Publisher: Cold Spring Harbor Laboratory
Section: New Results},
	pages = {2021.01.12.426453},
}

@article{levin_michael_g_polygenic_2020,
	title = {Polygenic {Risk} {Scores} and {Coronary} {Artery} {Disease}},
	volume = {141},
	url = {https://www.ahajournals.org/doi/full/10.1161/CIRCULATIONAHA.119.044770},
	doi = {10.1161/CIRCULATIONAHA.119.044770},
	number = {8},
	urldate = {2021-05-13},
	journal = {Circulation},
	author = {{Levin Michael G.} and {Rader Daniel J.}},
	month = feb,
	year = {2020},
	note = {Publisher: American Heart Association},
	pages = {637--640},
}

@article{sun_polygenic_2021,
	title = {Polygenic risk scores in cardiovascular risk prediction: {A} cohort study and modelling analyses},
	volume = {18},
	issn = {1549-1676},
	shorttitle = {Polygenic risk scores in cardiovascular risk prediction},
	url = {https://journals.plos.org/plosmedicine/article?id=10.1371/journal.pmed.1003498},
	doi = {10.1371/journal.pmed.1003498},
	abstract = {Background Polygenic risk scores (PRSs) can stratify populations into cardiovascular disease (CVD) risk groups. We aimed to quantify the potential advantage of adding information on PRSs to conventional risk factors in the primary prevention of CVD. Methods and findings Using data from UK Biobank on 306,654 individuals without a history of CVD and not on lipid-lowering treatments (mean age [SD]: 56.0 [8.0] years; females: 57\%; median follow-up: 8.1 years), we calculated measures of risk discrimination and reclassification upon addition of PRSs to risk factors in a conventional risk prediction model (i.e., age, sex, systolic blood pressure, smoking status, history of diabetes, and total and high-density lipoprotein cholesterol). We then modelled the implications of initiating guideline-recommended statin therapy in a primary care setting using incidence rates from 2.1 million individuals from the Clinical Practice Research Datalink. The C-index, a measure of risk discrimination, was 0.710 (95\% CI 0.703–0.717) for a CVD prediction model containing conventional risk predictors alone. Addition of information on PRSs increased the C-index by 0.012 (95\% CI 0.009–0.015), and resulted in continuous net reclassification improvements of about 10\% and 12\% in cases and non-cases, respectively. If a PRS were assessed in the entire UK primary care population aged 40–75 years, assuming that statin therapy would be initiated in accordance with the UK National Institute for Health and Care Excellence guidelines (i.e., for persons with a predicted risk of ≥10\% and for those with certain other risk factors, such as diabetes, irrespective of their 10-year predicted risk), then it could help prevent 1 additional CVD event for approximately every 5,750 individuals screened. By contrast, targeted assessment only among people at intermediate (i.e., 5\% to {\textless}10\%) 10-year CVD risk could help prevent 1 additional CVD event for approximately every 340 individuals screened. Such a targeted strategy could help prevent 7\% more CVD events than conventional risk prediction alone. Potential gains afforded by assessment of PRSs on top of conventional risk factors would be about 1.5-fold greater than those provided by assessment of C-reactive protein, a plasma biomarker included in some risk prediction guidelines. Potential limitations of this study include its restriction to European ancestry participants and a lack of health economic evaluation. Conclusions Our results suggest that addition of PRSs to conventional risk factors can modestly enhance prediction of first-onset CVD and could translate into population health benefits if used at scale.},
	language = {en},
	number = {1},
	urldate = {2021-05-13},
	journal = {PLOS Medicine},
	author = {Sun, Luanluan and Pennells, Lisa and Kaptoge, Stephen and Nelson, Christopher P. and Ritchie, Scott C. and Abraham, Gad and Arnold, Matthew and Bell, Steven and Bolton, Thomas and Burgess, Stephen and Dudbridge, Frank and Guo, Qi and Sofianopoulou, Eleni and Stevens, David and Thompson, John R. and Butterworth, Adam S. and Wood, Angela and Danesh, John and Samani, Nilesh J. and Inouye, Michael and Angelantonio, Emanuele Di},
	month = jan,
	year = {2021},
	note = {Publisher: Public Library of Science},
	keywords = {Blood pressure, Cardiovascular disease risk, Cardiovascular diseases, Cholesterol, Coronary heart disease, Diabetes mellitus, Ischemic stroke, Medical risk factors},
	pages = {e1003498},
}

@article{khan_respiratory_2020,
	title = {Respiratory and ventilator management of {COVID}-19},
	volume = {70(Suppl 3)},
	issn = {0030-9982},
	doi = {10.5455/JPMA.23},
	abstract = {The current pandemic of COVID-19 has infected around 2.5 million people with more than 125,000 deaths across the globe till date, and numbers are still rising. The causative organism is a virus of corona family. The International Committee on Taxonomy of Viruses (ICTV) named it severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) due to its similarities with the virus was caused SARS outbreak (SARS-CoV). Although most of the patients present with less severe symptoms like rhinitis, cough, fever, and mild flu-like symptoms, it may progress to severe acute respiratory illness, pneumonia or acute respiratory distress syndrome (ARDS) mainly in immunocompromised hosts. Severe infections mainly involve lungs, and compromise its capacity of ventilation. Respiratory and mechanical ventilation is one of the important parts of management.},
	language = {eng},
	number = {5},
	journal = {JPMA. The Journal of the Pakistan Medical Association},
	author = {Khan, Nafees Ahmad and Akhtar, Jamal},
	month = may,
	year = {2020},
	pmid = {32515384},
	keywords = {Betacoronavirus, COVID-19, Coronavirus Infections, Humans, Pandemics, Pneumonia, Viral, Respiratory Distress Syndrome, Respiratory Therapy, SARS CoV-2, Mechanical management, ARDS, Pandemic, Coronavirus., SARS-CoV-2},
	pages = {S60--S63},
}

@article{botta_ventilation_2021,
	title = {Ventilation management and clinical outcomes in invasively ventilated patients with {COVID}-19 ({PRoVENT}-{COVID}): a national, multicentre, observational cohort study},
	volume = {9},
	issn = {2213-2619},
	shorttitle = {Ventilation management and clinical outcomes in invasively ventilated patients with {COVID}-19 ({PRoVENT}-{COVID})},
	doi = {10.1016/S2213-2600(20)30459-8},
	abstract = {BACKGROUND: Little is known about the practice of ventilation management in patients with COVID-19. We aimed to describe the practice of ventilation management and to establish outcomes in invasively ventilated patients with COVID-19 in a single country during the first month of the outbreak.
METHODS: PRoVENT-COVID is a national, multicentre, retrospective observational study done at 18 intensive care units (ICUs) in the Netherlands. Consecutive patients aged at least 18 years were eligible for participation if they had received invasive ventilation for COVID-19 at a participating ICU during the first month of the national outbreak in the Netherlands. The primary outcome was a combination of ventilator variables and parameters over the first 4 calendar days of ventilation: tidal volume, positive end-expiratory pressure (PEEP), respiratory system compliance, and driving pressure. Secondary outcomes included the use of adjunctive treatments for refractory hypoxaemia and ICU complications. Patient-centred outcomes were ventilator-free days at day 28, duration of ventilation, duration of ICU and hospital stay, and mortality. PRoVENT-COVID is registered at ClinicalTrials.gov (NCT04346342).
FINDINGS: Between March 1 and April 1, 2020, 553 patients were included in the study. Median tidal volume was 6·3 mL/kg predicted bodyweight (IQR 5·7-7·1), PEEP was 14·0 cm H2O (IQR 11·0-15·0), and driving pressure was 14·0 cm H2O (11·2-16·0). Median respiratory system compliance was 31·9 mL/cm H2O (26·0-39·9). Of the adjunctive treatments for refractory hypoxaemia, prone positioning was most often used in the first 4 days of ventilation (283 [53\%] of 530 patients). The median number of ventilator-free days at day 28 was 0 (IQR 0-15); 186 (35\%) of 530 patients had died by day 28. Predictors of 28-day mortality were gender, age, tidal volume, respiratory system compliance, arterial pH, and heart rate on the first day of invasive ventilation.
INTERPRETATION: In patients with COVID-19 who were invasively ventilated during the first month of the outbreak in the Netherlands, lung-protective ventilation with low tidal volume and low driving pressure was broadly applied and prone positioning was often used. The applied PEEP varied widely, despite an invariably low respiratory system compliance. The findings of this national study provide a basis for new hypotheses and sample size calculations for future trials of invasive ventilation for COVID-19. These data could also help in the interpretation of findings from other studies of ventilation practice and outcomes in invasively ventilated patients with COVID-19.
FUNDING: Amsterdam University Medical Centers, location Academic Medical Center.},
	language = {eng},
	number = {2},
	journal = {The Lancet. Respiratory Medicine},
	author = {Botta, Michela and Tsonas, Anissa M. and Pillay, Janesh and Boers, Leonoor S. and Algera, Anna Geke and Bos, Lieuwe D. J. and Dongelmans, Dave A. and Hollmann, Marcus W. and Horn, Janneke and Vlaar, Alexander P. J. and Schultz, Marcus J. and Neto, Ary Serpa and Paulus, Frederique and {PRoVENT-COVID Collaborative Group}},
	month = feb,
	year = {2021},
	pmid = {33169671},
	pmcid = {PMC7584441},
	keywords = {Aged, COVID-19, Cohort Studies, Female, Humans, Male, Middle Aged, Netherlands, Respiration, Artificial, Retrospective Studies, Treatment Outcome},
	pages = {139--148},
}

@article{francois_challenge_2020,
	title = {The challenge of ventilator-associated pneumonia diagnosis in {COVID}-19 patients},
	volume = {24},
	issn = {1364-8535},
	url = {https://doi.org/10.1186/s13054-020-03013-2},
	doi = {10.1186/s13054-020-03013-2},
	number = {1},
	urldate = {2021-04-28},
	journal = {Critical Care},
	author = {François, Bruno and Laterre, Pierre-François and Luyt, Charles-Edouard and Chastre, Jean},
	month = jun,
	year = {2020},
	pages = {289},
}

@article{nicolau_topology_2011,
	title = {Topology based data analysis identifies a subgroup of breast cancers with a unique mutational profile and excellent survival},
	volume = {108},
	issn = {1091-6490},
	doi = {10.1073/pnas.1102826108},
	abstract = {High-throughput biological data, whether generated as sequencing, transcriptional microarrays, proteomic, or other means, continues to require analytic methods that address its high dimensional aspects. Because the computational part of data analysis ultimately identifies shape characteristics in the organization of data sets, the mathematics of shape recognition in high dimensions continues to be a crucial part of data analysis. This article introduces a method that extracts information from high-throughput microarray data and, by using topology, provides greater depth of information than current analytic techniques. The method, termed Progression Analysis of Disease (PAD), first identifies robust aspects of cluster analysis, then goes deeper to find a multitude of biologically meaningful shape characteristics in these data. Additionally, because PAD incorporates a visualization tool, it provides a simple picture or graph that can be used to further explore these data. Although PAD can be applied to a wide range of high-throughput data types, it is used here as an example to analyze breast cancer transcriptional data. This identified a unique subgroup of Estrogen Receptor-positive (ER(+)) breast cancers that express high levels of c-MYB and low levels of innate inflammatory genes. These patients exhibit 100\% survival and no metastasis. No supervised step beyond distinction between tumor and healthy patients was used to identify this subtype. The group has a clear and distinct, statistically significant molecular signature, it highlights coherent biology but is invisible to cluster methods, and does not fit into the accepted classification of Luminal A/B, Normal-like subtypes of ER(+) breast cancers. We denote the group as c-MYB(+) breast cancer.},
	language = {eng},
	number = {17},
	journal = {Proceedings of the National Academy of Sciences of the United States of America},
	author = {Nicolau, Monica and Levine, Arnold J. and Carlsson, Gunnar},
	month = apr,
	year = {2011},
	pmid = {21482760},
	pmcid = {PMC3084136},
	keywords = {Breast Neoplasms, Female, Gene Expression Regulation, Neoplastic, Humans, Models, Biological, Proto-Oncogene Proteins c-myb, Receptors, Estrogen, Survival Rate, Systems Biology},
	pages = {7265--7270},
}

@article{dahl_robust_2020,
	title = {A {Robust} {Method} {Uncovers} {Significant} {Context}-{Specific} {Heritability} in {Diverse} {Complex} {Traits}},
	volume = {106},
	issn = {0002-9297},
	url = {https://www.sciencedirect.com/science/article/pii/S0002929719304628},
	doi = {10.1016/j.ajhg.2019.11.015},
	abstract = {Gene-environment interactions (GxE) can be fundamental in applications ranging from functional genomics to precision medicine and is a conjectured source of substantial heritability. However, unbiased methods to profile GxE genome-wide are nascent and, as we show, cannot accommodate general environment variables, modest sample sizes, heterogeneous noise, and binary traits. To address this gap, we propose a simple, unifying mixed model for gene-environment interaction (GxEMM). In simulations and theory, we show that GxEMM can dramatically improve estimates and eliminate false positives when the assumptions of existing methods fail. We apply GxEMM to a range of human and model organism datasets and find broad evidence of context-specific genetic effects, including GxSex, GxAdversity, and GxDisease interactions across thousands of clinical and molecular phenotypes. Overall, GxEMM is broadly applicable for testing and quantifying polygenic interactions, which can be useful for explaining heritability and invaluable for determining biologically relevant environments.},
	language = {en},
	number = {1},
	urldate = {2021-04-16},
	journal = {The American Journal of Human Genetics},
	author = {Dahl, Andy and Nguyen, Khiem and Cai, Na and Gandal, Michael J. and Flint, Jonathan and Zaitlen, Noah},
	month = jan,
	year = {2020},
	keywords = {G-E correlation, GxE, disease subtypes, genetic heterogeneity, heritability, heteroskedasticity, linear mixed model, psychiatric disease},
	pages = {71--91},
}

@article{li_identification_2015,
	title = {Identification of type 2 diabetes subgroups through topological analysis of patient similarity},
	volume = {7},
	issn = {1946-6242},
	doi = {10.1126/scitranslmed.aaa9364},
	abstract = {Type 2 diabetes (T2D) is a heterogeneous complex disease affecting more than 29 million Americans alone with a rising prevalence trending toward steady increases in the coming decades. Thus, there is a pressing clinical need to improve early prevention and clinical management of T2D and its complications. Clinicians have understood that patients who carry the T2D diagnosis have a variety of phenotypes and susceptibilities to diabetes-related complications. We used a precision medicine approach to characterize the complexity of T2D patient populations based on high-dimensional electronic medical records (EMRs) and genotype data from 11,210 individuals. We successfully identified three distinct subgroups of T2D from topology-based patient-patient networks. Subtype 1 was characterized by T2D complications diabetic nephropathy and diabetic retinopathy; subtype 2 was enriched for cancer malignancy and cardiovascular diseases; and subtype 3 was associated most strongly with cardiovascular diseases, neurological diseases, allergies, and HIV infections. We performed a genetic association analysis of the emergent T2D subtypes to identify subtype-specific genetic markers and identified 1279, 1227, and 1338 single-nucleotide polymorphisms (SNPs) that mapped to 425, 322, and 437 unique genes specific to subtypes 1, 2, and 3, respectively. By assessing the human disease-SNP association for each subtype, the enriched phenotypes and biological functions at the gene level for each subtype matched with the disease comorbidities and clinical differences that we identified through EMRs. Our approach demonstrates the utility of applying the precision medicine paradigm in T2D and the promise of extending the approach to the study of other complex, multifactorial diseases.},
	language = {eng},
	number = {311},
	journal = {Science Translational Medicine},
	author = {Li, Li and Cheng, Wei-Yi and Glicksberg, Benjamin S. and Gottesman, Omri and Tamler, Ronald and Chen, Rong and Bottinger, Erwin P. and Dudley, Joel T.},
	month = oct,
	year = {2015},
	pmid = {26511511},
	pmcid = {PMC4780757},
	keywords = {Cardiovascular Diseases, Diabetes Mellitus, Type 2, Female, Genetic Association Studies, Genetic Predisposition to Disease, Genome-Wide Association Study, HIV Infections, Humans, Male, Middle Aged, Polymorphism, Single Nucleotide},
	pages = {311ra174},
}

@article{arnedo_uncovering_2015,
	title = {Uncovering the hidden risk architecture of the schizophrenias: confirmation in three independent genome-wide association studies},
	volume = {172},
	issn = {1535-7228},
	shorttitle = {Uncovering the hidden risk architecture of the schizophrenias},
	doi = {10.1176/appi.ajp.2014.14040435},
	abstract = {OBJECTIVE: The authors sought to demonstrate that schizophrenia is a heterogeneous group of heritable disorders caused by different genotypic networks that cause distinct clinical syndromes.
METHOD: In a large genome-wide association study of cases with schizophrenia and controls, the authors first identified sets of interacting single-nucleotide polymorphisms (SNPs) that cluster within particular individuals (SNP sets) regardless of clinical status. Second, they examined the risk of schizophrenia for each SNP set and tested replicability in two independent samples. Third, they identified genotypic networks composed of SNP sets sharing SNPs or subjects. Fourth, they identified sets of distinct clinical features that cluster in particular cases (phenotypic sets or clinical syndromes) without regard for their genetic background. Fifth, they tested whether SNP sets were associated with distinct phenotypic sets in a replicable manner across the three studies.
RESULTS: The authors identified 42 SNP sets associated with a 70\% or greater risk of schizophrenia, and confirmed 34 (81\%) or more with similar high risk of schizophrenia in two independent samples. Seventeen networks of SNP sets did not share any SNP or subject. These disjoint genotypic networks were associated with distinct gene products and clinical syndromes (i.e., the schizophrenias) varying in symptoms and severity. Associations between genotypic networks and clinical syndromes were complex, showing multifinality and equifinality. The interactive networks explained the risk of schizophrenia more than the average effects of all SNPs (24\%).
CONCLUSIONS: Schizophrenia is a group of heritable disorders caused by a moderate number of separate genotypic networks associated with several distinct clinical syndromes.},
	language = {eng},
	number = {2},
	journal = {The American Journal of Psychiatry},
	author = {Arnedo, Javier and Svrakic, Dragan M. and Del Val, Coral and Romero-Zaliz, Rocío and Hernández-Cuervo, Helena and {Molecular Genetics of Schizophrenia Consortium} and Fanous, Ayman H. and Pato, Michele T. and Pato, Carlos N. and de Erausquin, Gabriel A. and Cloninger, C. Robert and Zwir, Igor},
	month = feb,
	year = {2015},
	pmid = {25219520},
	keywords = {Adult, Female, Genetic Association Studies, Genetic Predisposition to Disease, Genome-Wide Association Study, Humans, Male, Neural Pathways, Polymorphism, Single Nucleotide, Psychiatric Status Rating Scales, Risk Assessment, Schizophrenia, Schizophrenic Psychology, Severity of Illness Index, Synaptic Transmission},
	pages = {139--153},
}

@article{choi_prsice-2_2019,
	title = {{PRSice}-2: {Polygenic} {Risk} {Score} software for biobank-scale data},
	volume = {8},
	issn = {2047-217X},
	shorttitle = {{PRSice}-2},
	doi = {10.1093/gigascience/giz082},
	abstract = {BACKGROUND: Polygenic risk score (PRS) analyses have become an integral part of biomedical research, exploited to gain insights into shared aetiology among traits, to control for genomic profile in experimental studies, and to strengthen causal inference, among a range of applications. Substantial efforts are now devoted to biobank projects to collect large genetic and phenotypic data, providing unprecedented opportunity for genetic discovery and applications. To process the large-scale data provided by such biobank resources, highly efficient and scalable methods and software are required.
RESULTS: Here we introduce PRSice-2, an efficient and scalable software program for automating and simplifying PRS analyses on large-scale data. PRSice-2 handles both genotyped and imputed data, provides empirical association P-values free from inflation due to overfitting, supports different inheritance models, and can evaluate multiple continuous and binary target traits simultaneously. We demonstrate that PRSice-2 is dramatically faster and more memory-efficient than PRSice-1 and alternative PRS software, LDpred and lassosum, while having comparable predictive power.
CONCLUSION: PRSice-2's combination of efficiency and power will be increasingly important as data sizes grow and as the applications of PRS become more sophisticated, e.g., when incorporated into high-dimensional or gene set-based analyses. PRSice-2 is written in C++, with an R script for plotting, and is freely available for download from http://PRSice.info.},
	language = {eng},
	number = {7},
	journal = {GigaScience},
	author = {Choi, Shing Wan and O'Reilly, Paul F.},
	month = jul,
	year = {2019},
	pmid = {31307061},
	pmcid = {PMC6629542},
	keywords = {Animals, Big Data, GWAS, Genome-Wide Association Study, Humans, Multifactorial Inheritance, Quantitative Trait Loci, Software, imputation, polygenic risk score},
}

@article{li_polygenic_2021,
	title = {Polygenic {Risk} {Scores} {Augment} {Stroke} {Subtyping}},
	volume = {7},
	copyright = {Copyright © 2021 The Author(s). Published by Wolters Kluwer Health, Inc. on behalf of the American Academy of Neurology.. This is an open access article distributed under the terms of the Creative Commons Attribution-NonCommercial-NoDerivatives License 4.0 (CC BY-NC-ND), which permits downloading and sharing the work provided it is properly cited. The work cannot be changed in any way or used commercially without permission from the journal.},
	issn = {2376-7839},
	url = {https://ng.neurology.org/content/7/2/e560},
	doi = {10.1212/NXG.0000000000000560},
	abstract = {Objective To determine whether the polygenic risk score (PRS) derived from MEGASTROKE is associated with ischemic stroke (IS) and its subtypes in an independent tertiary health care system and to identify the PRS derived from gene sets of known biological pathways associated with IS.
Methods Controls (n = 19,806/7,484, age ≥69/79 years) and cases (n = 1,184/951 for discovery/replication) of acute IS with European ancestry and clinical risk factors were identified by leveraging the Geisinger Electronic Health Record and chart review confirmation. All Geisinger MyCode patients with age ≥69/79 years and without any stroke-related diagnostic codes were included as low risk control. Genetic heritability and genetic correlation between Geisinger and MEGASTROKE (EUR) were calculated using the summary statistics of the genome-wide association study by linkage disequilibrium score regression. All PRS for any stroke (AS), any ischemic stroke (AIS), large artery stroke (LAS), cardioembolic stroke (CES), and small vessel stroke (SVS) were constructed by PRSice-2.
Results A moderate heritability (10\%–20\%) for Geisinger sample as well as the genetic correlation between MEGASTROKE and the Geisinger cohort was identified. Variation of all 5 PRS significantly explained some of the phenotypic variations of Geisinger IS, and the R2 increased by raising the cutoff for the age of controls. PRSLAS, PRSCES, and PRSSVS derived from low-frequency common variants provided the best fit for modeling (R2 = 0.015 for PRSLAS). Gene sets analyses highlighted the association of PRS with Gene Ontology terms (vascular endothelial growth factor, amyloid precursor protein, and atherosclerosis). The PRSLAS, PRSCES, and PRSSVS explained the most variance of the corresponding subtypes of Geisinger IS suggesting shared etiologies and corroborated Geisinger TOAST subtyping.
Conclusions We provide the first evidence that PRSs derived from MEGASTROKE have value in identifying shared etiologies and determining stroke subtypes.},
	language = {en},
	number = {2},
	urldate = {2021-04-16},
	journal = {Neurology Genetics},
	author = {Li, Jiang and Chaudhary, Durgesh P. and Khan, Ayesha and Griessenauer, Christoph and Carey, David J. and Zand, Ramin and Abedi, Vida},
	month = apr,
	year = {2021},
	note = {Publisher: Wolters Kluwer Health, Inc. on behalf of the American Academy of Neurology
Section: Article},
}

@article{mavaddat_polygenic_2019,
	title = {Polygenic {Risk} {Scores} for {Prediction} of {Breast} {Cancer} and {Breast} {Cancer} {Subtypes}},
	volume = {104},
	issn = {0002-9297},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6323553/},
	doi = {10.1016/j.ajhg.2018.11.002},
	abstract = {Stratification of women according to their risk of breast cancer based on polygenic risk scores (PRSs) could improve screening and prevention strategies. Our aim was to develop PRSs, optimized for prediction of estrogen receptor (ER)-specific disease, from the largest available genome-wide association dataset and to empirically validate the PRSs in prospective studies. The development dataset comprised 94,075 case subjects and 75,017 control subjects of European ancestry from 69 studies, divided into training and validation sets. Samples were genotyped using genome-wide arrays, and single-nucleotide polymorphisms (SNPs) were selected by stepwise regression or lasso penalized regression. The best performing PRSs were validated in an independent test set comprising 11,428 case subjects and 18,323 control subjects from 10 prospective studies and 190,040 women from UK Biobank (3,215 incident breast cancers). For the best PRSs (313 SNPs), the odds ratio for overall disease per 1 standard deviation in ten prospective studies was 1.61 (95\%CI: 1.57–1.65) with area under receiver-operator curve (AUC) = 0.630 (95\%CI: 0.628–0.651). The lifetime risk of overall breast cancer in the top centile of the PRSs was 32.6\%. Compared with women in the middle quintile, those in the highest 1\% of risk had 4.37- and 2.78-fold risks, and those in the lowest 1\% of risk had 0.16- and 0.27-fold risks, of developing ER-positive and ER-negative disease, respectively. Goodness-of-fit tests indicated that this PRS was well calibrated and predicts disease risk accurately in the tails of the distribution. This PRS is a powerful and reliable predictor of breast cancer risk that may improve breast cancer prevention programs.},
	number = {1},
	urldate = {2021-04-16},
	journal = {American Journal of Human Genetics},
	author = {Mavaddat, Nasim and Michailidou, Kyriaki and Dennis, Joe and Lush, Michael and Fachal, Laura and Lee, Andrew and Tyrer, Jonathan P. and Chen, Ting-Huei and Wang, Qin and Bolla, Manjeet K. and Yang, Xin and Adank, Muriel A. and Ahearn, Thomas and Aittomäki, Kristiina and Allen, Jamie and Andrulis, Irene L. and Anton-Culver, Hoda and Antonenkova, Natalia N. and Arndt, Volker and Aronson, Kristan J. and Auer, Paul L. and Auvinen, Päivi and Barrdahl, Myrto and Beane Freeman, Laura E. and Beckmann, Matthias W. and Behrens, Sabine and Benitez, Javier and Bermisheva, Marina and Bernstein, Leslie and Blomqvist, Carl and Bogdanova, Natalia V. and Bojesen, Stig E. and Bonanni, Bernardo and Børresen-Dale, Anne-Lise and Brauch, Hiltrud and Bremer, Michael and Brenner, Hermann and Brentnall, Adam and Brock, Ian W. and Brooks-Wilson, Angela and Brucker, Sara Y. and Brüning, Thomas and Burwinkel, Barbara and Campa, Daniele and Carter, Brian D. and Castelao, Jose E. and Chanock, Stephen J. and Chlebowski, Rowan and Christiansen, Hans and Clarke, Christine L. and Collée, J. Margriet and Cordina-Duverger, Emilie and Cornelissen, Sten and Couch, Fergus J. and Cox, Angela and Cross, Simon S. and Czene, Kamila and Daly, Mary B. and Devilee, Peter and Dörk, Thilo and dos-Santos-Silva, Isabel and Dumont, Martine and Durcan, Lorraine and Dwek, Miriam and Eccles, Diana M. and Ekici, Arif B. and Eliassen, A. Heather and Ellberg, Carolina and Engel, Christoph and Eriksson, Mikael and Evans, D. Gareth and Fasching, Peter A. and Figueroa, Jonine and Fletcher, Olivia and Flyger, Henrik and Försti, Asta and Fritschi, Lin and Gabrielson, Marike and Gago-Dominguez, Manuela and Gapstur, Susan M. and García-Sáenz, José A. and Gaudet, Mia M. and Georgoulias, Vassilios and Giles, Graham G. and Gilyazova, Irina R. and Glendon, Gord and Goldberg, Mark S. and Goldgar, David E. and González-Neira, Anna and Grenaker Alnæs, Grethe I. and Grip, Mervi and Gronwald, Jacek and Grundy, Anne and Guénel, Pascal and Haeberle, Lothar and Hahnen, Eric and Haiman, Christopher A. and Håkansson, Niclas and Hamann, Ute and Hankinson, Susan E. and Harkness, Elaine F. and Hart, Steven N. and He, Wei and Hein, Alexander and Heyworth, Jane and Hillemanns, Peter and Hollestelle, Antoinette and Hooning, Maartje J. and Hoover, Robert N. and Hopper, John L. and Howell, Anthony and Huang, Guanmengqian and Humphreys, Keith and Hunter, David J. and Jakimovska, Milena and Jakubowska, Anna and Janni, Wolfgang and John, Esther M. and Johnson, Nichola and Jones, Michael E. and Jukkola-Vuorinen, Arja and Jung, Audrey and Kaaks, Rudolf and Kaczmarek, Katarzyna and Kataja, Vesa and Keeman, Renske and Kerin, Michael J. and Khusnutdinova, Elza and Kiiski, Johanna I. and Knight, Julia A. and Ko, Yon-Dschun and Kosma, Veli-Matti and Koutros, Stella and Kristensen, Vessela N. and Krüger, Ute and Kühl, Tabea and Lambrechts, Diether and Le Marchand, Loic and Lee, Eunjung and Lejbkowicz, Flavio and Lilyquist, Jenna and Lindblom, Annika and Lindström, Sara and Lissowska, Jolanta and Lo, Wing-Yee and Loibl, Sibylle and Long, Jirong and Lubiński, Jan and Lux, Michael P. and MacInnis, Robert J. and Maishman, Tom and Makalic, Enes and Maleva Kostovska, Ivana and Mannermaa, Arto and Manoukian, Siranoush and Margolin, Sara and Martens, John W.M. and Martinez, Maria Elena and Mavroudis, Dimitrios and McLean, Catriona and Meindl, Alfons and Menon, Usha and Middha, Pooja and Miller, Nicola and Moreno, Fernando and Mulligan, Anna Marie and Mulot, Claire and Muñoz-Garzon, Victor M. and Neuhausen, Susan L. and Nevanlinna, Heli and Neven, Patrick and Newman, William G. and Nielsen, Sune F. and Nordestgaard, Børge G. and Norman, Aaron and Offit, Kenneth and Olson, Janet E. and Olsson, Håkan and Orr, Nick and Pankratz, V. Shane and Park-Simon, Tjoung-Won and Perez, Jose I.A. and Pérez-Barrios, Clara and Peterlongo, Paolo and Peto, Julian and Pinchev, Mila and Plaseska-Karanfilska, Dijana and Polley, Eric C. and Prentice, Ross and Presneau, Nadege and Prokofyeva, Darya and Purrington, Kristen and Pylkäs, Katri and Rack, Brigitte and Radice, Paolo and Rau-Murthy, Rohini and Rennert, Gad and Rennert, Hedy S. and Rhenius, Valerie and Robson, Mark and Romero, Atocha and Ruddy, Kathryn J. and Ruebner, Matthias and Saloustros, Emmanouil and Sandler, Dale P. and Sawyer, Elinor J. and Schmidt, Daniel F. and Schmutzler, Rita K. and Schneeweiss, Andreas and Schoemaker, Minouk J. and Schumacher, Fredrick and Schürmann, Peter and Schwentner, Lukas and Scott, Christopher and Scott, Rodney J. and Seynaeve, Caroline and Shah, Mitul and Sherman, Mark E. and Shrubsole, Martha J. and Shu, Xiao-Ou and Slager, Susan and Smeets, Ann and Sohn, Christof and Soucy, Penny and Southey, Melissa C. and Spinelli, John J. and Stegmaier, Christa and Stone, Jennifer and Swerdlow, Anthony J. and Tamimi, Rulla M. and Tapper, William J. and Taylor, Jack A. and Terry, Mary Beth and Thöne, Kathrin and Tollenaar, Rob A.E.M. and Tomlinson, Ian and Truong, Thérèse and Tzardi, Maria and Ulmer, Hans-Ulrich and Untch, Michael and Vachon, Celine M. and van Veen, Elke M. and Vijai, Joseph and Weinberg, Clarice R. and Wendt, Camilla and Whittemore, Alice S. and Wildiers, Hans and Willett, Walter and Winqvist, Robert and Wolk, Alicja and Yang, Xiaohong R. and Yannoukakos, Drakoulis and Zhang, Yan and Zheng, Wei and Ziogas, Argyrios and Dunning, Alison M. and Thompson, Deborah J. and Chenevix-Trench, Georgia and Chang-Claude, Jenny and Schmidt, Marjanka K. and Hall, Per and Milne, Roger L. and Pharoah, Paul D.P. and Antoniou, Antonis C. and Chatterjee, Nilanjan and Kraft, Peter and García-Closas, Montserrat and Simard, Jacques and Easton, Douglas F.},
	month = jan,
	year = {2019},
	pmid = {30554720},
	pmcid = {PMC6323553},
	pages = {21--34},
}

@article{dahl_reverse_2019,
	title = {Reverse {GWAS}: {Using} genetics to identify and model phenotypic subtypes},
	volume = {15},
	issn = {1553-7404},
	shorttitle = {Reverse {GWAS}},
	url = {https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008009},
	doi = {10.1371/journal.pgen.1008009},
	abstract = {Recent and classical work has revealed biologically and medically significant subtypes in complex diseases and traits. However, relevant subtypes are often unknown, unmeasured, or actively debated, making automated statistical approaches to subtype definition valuable. We propose reverse GWAS (RGWAS) to identify and validate subtypes using genetics and multiple traits: while GWAS seeks the genetic basis of a given trait, RGWAS seeks to define trait subtypes with distinct genetic bases. Unlike existing approaches relying on off-the-shelf clustering methods, RGWAS uses a novel decomposition, MFMR, to model covariates, binary traits, and population structure. We use extensive simulations to show that modelling these features can be crucial for power and calibration. We validate RGWAS in practice by recovering a recently discovered stress subtype in major depression. We then show the utility of RGWAS by identifying three novel subtypes of metabolic traits. We biologically validate these metabolic subtypes with SNP-level tests and a novel polygenic test: the former recover known metabolic GxE SNPs; the latter suggests subtypes may explain substantial missing heritability. Crucially, statins, which are widely prescribed and theorized to increase diabetes risk, have opposing effects on blood glucose across metabolic subtypes, suggesting the subtypes have potential translational value.},
	language = {en},
	number = {4},
	urldate = {2021-04-16},
	journal = {PLOS Genetics},
	author = {Dahl, Andy and Cai, Na and Ko, Arthur and Laakso, Markku and Pajukanta, Päivi and Flint, Jonathan and Zaitlen, Noah},
	month = apr,
	year = {2019},
	note = {Publisher: Public Library of Science},
	keywords = {Genetics, Genetics of disease, Genome-wide association studies, Glucose metabolism, Heredity, Phenotypes, Single nucleotide polymorphisms, Statins},
	pages = {e1008009},
}

@article{choi_tutorial_2020,
	title = {Tutorial: a guide to performing polygenic risk score analyses},
	volume = {15},
	copyright = {2020 The Author(s), under exclusive licence to Springer Nature Limited},
	issn = {1750-2799},
	shorttitle = {Tutorial},
	url = {https://www.nature.com/articles/s41596-020-0353-1},
	doi = {10.1038/s41596-020-0353-1},
	abstract = {A polygenic score (PGS) or polygenic risk score (PRS) is an estimate of an individual’s genetic liability to a trait or disease, calculated according to their genotype profile and relevant genome-wide association study (GWAS) data. While present PRSs typically explain only a small fraction of trait variance, their correlation with the single largest contributor to phenotypic variation—genetic liability—has led to the routine application of PRSs across biomedical research. Among a range of applications, PRSs are exploited to assess shared etiology between phenotypes, to evaluate the clinical utility of genetic data for complex disease and as part of experimental studies in which, for example, experiments are performed that compare outcomes (e.g., gene expression and cellular response to treatment) between individuals with low and high PRS values. As GWAS sample sizes increase and PRSs become more powerful, PRSs are set to play a key role in research and stratified medicine. However, despite the importance and growing application of PRSs, there are limited guidelines for performing PRS analyses, which can lead to inconsistency between studies and misinterpretation of results. Here, we provide detailed guidelines for performing and interpreting PRS analyses. We outline standard quality control steps, discuss different methods for the calculation of PRSs, provide an introductory online tutorial, highlight common misconceptions relating to PRS results, offer recommendations for best practice and discuss future challenges.},
	language = {en},
	number = {9},
	urldate = {2021-04-15},
	journal = {Nature Protocols},
	author = {Choi, Shing Wan and Mak, Timothy Shin-Heng and O’Reilly, Paul F.},
	month = sep,
	year = {2020},
	note = {Number: 9
Publisher: Nature Publishing Group},
	pages = {2759--2772},
}

@article{rappaport_malacards_2017,
	title = {{MalaCards}: an amalgamated human disease compendium with diverse clinical and genetic annotation and structured search},
	volume = {45},
	issn = {0305-1048},
	shorttitle = {{MalaCards}},
	url = {https://doi.org/10.1093/nar/gkw1012},
	doi = {10.1093/nar/gkw1012},
	abstract = {The MalaCards human disease database (http://www.malacards.org/) is an integrated compendium of annotated diseases mined from 68 data sources. MalaCards has a web card for each of ∼20 000 disease entries, in six global categories. It portrays a broad array of annotation topics in 15 sections, including Summaries, Symptoms, Anatomical Context, Drugs, Genetic Tests, Variations and Publications. The Aliases and Classifications section reflects an algorithm for disease name integration across often-conflicting sources, providing effective annotation consolidation. A central feature is a balanced Genes section, with scores reflecting the strength of disease-gene associations. This is accompanied by other gene-related disease information such as pathways, mouse phenotypes and GO-terms, stemming from MalaCards’ affiliation with the GeneCards Suite of databases. MalaCards’ capacity to inter-link information from complementary sources, along with its elaborate search function, relational database infrastructure and convenient data dumps, allows it to tackle its rich disease annotation landscape, and facilitates systems analyses and genome sequence interpretation. MalaCards adopts a ‘flat’ disease-card approach, but each card is mapped to popular hierarchical ontologies (e.g. International Classification of Diseases, Human Phenotype Ontology and Unified Medical Language System) and also contains information about multi-level relations among diseases, thereby providing an optimal tool for disease representation and scrutiny.},
	number = {D1},
	urldate = {2021-03-26},
	journal = {Nucleic Acids Research},
	author = {Rappaport, Noa and Twik, Michal and Plaschkes, Inbar and Nudel, Ron and Iny Stein, Tsippi and Levitt, Jacob and Gershoni, Moran and Morrey, C. Paul and Safran, Marilyn and Lancet, Doron},
	month = jan,
	year = {2017},
	pages = {D877--D887},
}

@article{uren_putting_2020,
	title = {Putting {RFMix} and {ADMIXTURE} to the test in a complex admixed population},
	volume = {21},
	issn = {1471-2156},
	url = {https://doi.org/10.1186/s12863-020-00845-3},
	doi = {10.1186/s12863-020-00845-3},
	abstract = {Global and local ancestry inference in admixed human populations can be performed using computational tools implementing distinct algorithms. The development and resulting accuracy of these tools has been tested largely on populations with relatively straightforward admixture histories but little is known about how well they perform in more complex admixture scenarios.},
	number = {1},
	urldate = {2020-07-27},
	journal = {BMC Genetics},
	author = {Uren, Caitlin and Hoal, Eileen G. and Möller, Marlo},
	month = apr,
	year = {2020},
	pages = {40},
}

@article{belbin_towards_2019,
	title = {Towards a fine-scale population health monitoring system},
	copyright = {© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},
	url = {https://www.biorxiv.org/content/10.1101/780668v1},
	doi = {10.1101/780668},
	abstract = {{\textless}h3{\textgreater}Abstract{\textless}/h3{\textgreater} {\textless}p{\textgreater}Understanding population health disparities is an essential component of equitable precision health efforts. Epidemiology research often relies on definitions of race and ethnicity, but these population labels may not adequately capture disease burdens specific to sub-populations. Here we propose a framework for repurposing data from Electronic Health Records (EHRs) in concert with genomic data to explore enrichment of disease within sub-populations. Using data from a diverse biobank in New York City, we genetically identified 17 sub-populations, and noted the presence of genetic founder effects in 7. By then linking community membership to the EHR, we were able to identify over 600 health outcomes that were statistically enriched within a specific population, with many representing known associations, and many others being novel. This work reinforces the utility of linking genomic data to EHRs, and provides a framework towards fine-scale monitoring of population health.{\textless}/p{\textgreater}},
	language = {en},
	urldate = {2020-07-22},
	journal = {bioRxiv},
	author = {Belbin, Gillian M. and Wenric, Stephane and Cullina, Sinead and Glicksberg, Benjamin S. and Moscati, Arden and Wojcik, Genevieve L. and Shemirani, Ruhollah and Beckmann, Noam D. and Cohain, Ariella and Sorokin, Elena P. and Park, Danny S. and Ambite, Jose-Luis and Ellis, Steve and Auton, Adam and Team, CBIPM Genomics and Team, CBIPM Genomics and Center, Regeneron Genetics and Bottinger, Erwin P. and Cho, Judy H. and Loos, Ruth JF and Abul-husn, Noura S. and Zaitlen, Noah A. and Gignoux, Christopher R. and Kenny, Eimear E.},
	month = sep,
	year = {2019},
	pages = {780668},
}

@misc{noauthor_towards_nodate,
	title = {Towards a fine-scale population health monitoring system {\textbar} {bioRxiv}},
	url = {http://webcache.googleusercontent.com/search?q=cache:0gBTn1glXYwJ:https://www.biorxiv.org/content/10.1101/780668v1&hl=en&gl=us&strip=0&vwsrc=0},
	urldate = {2020-07-22},
}

@article{marnetto_ancestry_2020,
	title = {Ancestry deconvolution and partial polygenic score can improve susceptibility predictions in recently admixed individuals},
	volume = {11},
	copyright = {2020 The Author(s)},
	issn = {2041-1723},
	url = {https://www.nature.com/articles/s41467-020-15464-w},
	doi = {10.1038/s41467-020-15464-w},
	abstract = {Polygenic Scores (PSs) describe the genetic component of an individual’s quantitative phenotype or their susceptibility to diseases with a genetic basis. Currently, PSs rely on population-dependent contributions of many associated alleles, with limited applicability to understudied populations and recently admixed individuals. Here we introduce a combination of local ancestry deconvolution and partial PS computation to account for the population-specific nature of the association signals in individuals with admixed ancestry. We demonstrate partial PS to be a proxy for the total PS and that a portion of the genome is enough to improve susceptibility predictions for the traits we test. By combining partial PSs from different populations, we are able to improve trait predictability in admixed individuals with some European ancestry. These results may extend the applicability of PSs to subjects with a complex history of admixture, where current methods cannot be applied.},
	language = {en},
	number = {1},
	urldate = {2020-07-22},
	journal = {Nature Communications},
	author = {Marnetto, Davide and Pärna, Katri and Läll, Kristi and Molinaro, Ludovica and Montinaro, Francesco and Haller, Toomas and Metspalu, Mait and Mägi, Reedik and Fischer, Krista and Pagani, Luca},
	month = apr,
	year = {2020},
	pages = {1628},
}

@article{maples_rfmix_2013,
	title = {{RFMix}: {A} {Discriminative} {Modeling} {Approach} for {Rapid} and {Robust} {Local}-{Ancestry} {Inference}},
	volume = {93},
	issn = {0002-9297},
	shorttitle = {{RFMix}},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3738819/},
	doi = {10.1016/j.ajhg.2013.06.020},
	abstract = {Local-ancestry inference is an important step in the genetic analysis of fully sequenced human genomes. Current methods can only detect continental-level ancestry (i.e., European versus African versus Asian) accurately even when using millions of markers. Here, we present RFMix, a powerful discriminative modeling approach that is faster (∼30×) and more accurate than existing methods. We accomplish this by using a conditional random field parameterized by random forests trained on reference panels. RFMix is capable of learning from the admixed samples themselves to boost performance and autocorrect phasing errors. RFMix shows high sensitivity and specificity in simulated Hispanics/Latinos and African Americans and admixed Europeans, Africans, and Asians. Finally, we demonstrate that African Americans in HapMap contain modest (but nonzero) levels of Native American ancestry (∼0.4\%).},
	number = {2},
	urldate = {2020-07-22},
	journal = {American Journal of Human Genetics},
	author = {Maples, Brian K. and Gravel, Simon and Kenny, Eimear E. and Bustamante, Carlos D.},
	month = aug,
	year = {2013},
	pmid = {23910464},
	pmcid = {PMC3738819},
	pages = {278--288},
}

@article{linda_kao_genome-wide_2008,
	title = {A genome-wide admixture scan identifies {MYH9} as a candidate locus associated with non-diabetic end stage renal disease in {African} {Americans}},
	volume = {40},
	issn = {1061-4036},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614692/},
	doi = {10.1038/ng.232},
	abstract = {End stage renal disease (ESRD) has a four times higher incidence in African Americans compared to European Americans. This led to the hypothesis that susceptibility alleles for ESRD have a higher frequency in West African than European gene pool. We performed a genome-wide admixture scan in 1,372 ESRD cases and 806 controls and demonstrated a highly significant association between excess African ancestry and non-diabetic ESRD (LOD 5.70) but not diabetic ESRD (LOD 0.47) on chromosome 22q12. Each copy of the European ancestral allele conferred a relative risk of 0.50 (95\% credible interval 0.39 – 0.63) compared to African ancestry. Multiple common SNPs (allele frequency ranging from 0.2 to 0.6) in the gene that encodes non-muscle myosin heavy chain type II isoform A (MYH9) were associated with 2-4 times greater risk of non-diabetic ESRD and accounted for a large proportion of the excess risk of ESRD observed in African compared to European Americans.},
	number = {10},
	urldate = {2020-07-22},
	journal = {Nature genetics},
	author = {Linda Kao, WH and Klag, Michael J and Meoni, Lucy A and Reich, David and Berthier-Schaad, Yvette and Li, Man and Coresh, Josef and Patterson, Nick and Tandon, Arti and Powe, Neil R and Fink, Nancy E and Sadler, John H and Weir, Matthew R and Abboud, Hanna E and Adler, Sharon and Divers, Jasmin and Iyengar, Sudha K and Freedman, Barry I and Kimmel, Paul L and Knowler, William C and Kohn, Orly F and Kramp, Kristopher and Leehey, David J and Nicholas, Susanne and Pahl, Madeleine and Schelling, Jeffrey R and Sedor, John R and Thornley-Brown, Denyse and Winkler, Cheryl A and Smith, Michael W. and Parekh, Rulan S.},
	month = oct,
	year = {2008},
	pmid = {18794854},
	pmcid = {PMC2614692},
	pages = {1185--1192},
}

@article{ioannidis_native_2020,
	title = {Native {American} gene flow into {Polynesia} predating {Easter} {Island} settlement},
	copyright = {2020 The Author(s), under exclusive licence to Springer Nature Limited},
	issn = {1476-4687},
	url = {https://www.nature.com/articles/s41586-020-2487-2},
	doi = {10.1038/s41586-020-2487-2},
	abstract = {The possibility of voyaging contact between prehistoric Polynesian and Native American populations has long intrigued researchers. Proponents have pointed to the existence of New World crops, such as the sweet potato and bottle gourd, in the Polynesian archaeological record, but nowhere else outside the pre-Columbian Americas1–6, while critics have argued that these botanical dispersals need not have been human mediated7. The Norwegian explorer Thor Heyerdahl controversially suggested that prehistoric South American populations had an important role in the settlement of east Polynesia and particularly of Easter Island (Rapa Nui)2. Several limited molecular genetic studies have reached opposing conclusions, and the possibility continues to be as hotly contested today as it was when first suggested8–12. Here we analyse genome-wide variation in individuals from islands across Polynesia for signs of Native American admixture, analysing 807 individuals from 17 island populations and 15 Pacific coast Native American groups. We find conclusive evidence for prehistoric contact of Polynesian individuals with Native American individuals (around ad 1200) contemporaneous with the settlement of remote Oceania13–15. Our analyses suggest strongly that a single contact event occurred in eastern Polynesia, before the settlement of Rapa Nui, between Polynesian individuals and a Native American group most closely related to the indigenous inhabitants of present-day Colombia.},
	language = {en},
	urldate = {2020-07-22},
	journal = {Nature},
	author = {Ioannidis, Alexander G. and Blanco-Portillo, Javier and Sandoval, Karla and Hagelberg, Erika and Miquel-Poblete, Juan Francisco and Moreno-Mayar, J. Víctor and Rodríguez-Rodríguez, Juan Esteban and Quinto-Cortés, Consuelo D. and Auckland, Kathryn and Parks, Tom and Robson, Kathryn and Hill, Adrian V. S. and Avila-Arcos, María C. and Sockell, Alexandra and Homburger, Julian R. and Wojcik, Genevieve L. and Barnes, Kathleen C. and Herrera, Luisa and Berríos, Soledad and Acuña, Mónica and Llop, Elena and Eng, Celeste and Huntsman, Scott and Burchard, Esteban G. and Gignoux, Christopher R. and Cifuentes, Lucía and Verdugo, Ricardo A. and Moraga, Mauricio and Mentzer, Alexander J. and Bustamante, Carlos D. and Moreno-Estrada, Andrés},
	month = jul,
	year = {2020},
	pages = {1--6},
}

@misc{noauthor_zotero_nodate,
	title = {Zotero {\textbar} {Groups} {\textgreater} {New} group},
	url = {https://www.zotero.org/groups/new/},
	urldate = {2020-07-22},
}

@misc{noauthor_zotero_nodate-1,
	title = {Zotero {\textbar} {Groups} {\textgreater} {New} group},
	url = {https://www.zotero.org/groups/new/},
	urldate = {2020-07-22},
}

@article{belbin_towards_2019-1,
	title = {Towards a fine-scale population health monitoring system},
	copyright = {© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},
	url = {https://www.biorxiv.org/content/10.1101/780668v1},
	doi = {10.1101/780668},
	abstract = {{\textless}h3{\textgreater}Abstract{\textless}/h3{\textgreater} {\textless}p{\textgreater}Understanding population health disparities is an essential component of equitable precision health efforts. Epidemiology research often relies on definitions of race and ethnicity, but these population labels may not adequately capture disease burdens specific to sub-populations. Here we propose a framework for repurposing data from Electronic Health Records (EHRs) in concert with genomic data to explore enrichment of disease within sub-populations. Using data from a diverse biobank in New York City, we genetically identified 17 sub-populations, and noted the presence of genetic founder effects in 7. By then linking community membership to the EHR, we were able to identify over 600 health outcomes that were statistically enriched within a specific population, with many representing known associations, and many others being novel. This work reinforces the utility of linking genomic data to EHRs, and provides a framework towards fine-scale monitoring of population health.{\textless}/p{\textgreater}},
	language = {en},
	urldate = {2020-07-06},
	journal = {bioRxiv},
	author = {Belbin, Gillian M. and Wenric, Stephane and Cullina, Sinead and Glicksberg, Benjamin S. and Moscati, Arden and Wojcik, Genevieve L. and Shemirani, Ruhollah and Beckmann, Noam D. and Cohain, Ariella and Sorokin, Elena P. and Park, Danny S. and Ambite, Jose-Luis and Ellis, Steve and Auton, Adam and Team, CBIPM Genomics and Team, CBIPM Genomics and Center, Regeneron Genetics and Bottinger, Erwin P. and Cho, Judy H. and Loos, Ruth JF and Abul-husn, Noura S. and Zaitlen, Noah A. and Gignoux, Christopher R. and Kenny, Eimear E.},
	month = sep,
	year = {2019},
	note = {Publisher: Cold Spring Harbor Laboratory
Section: New Results},
	pages = {780668},
}

@article{marnetto_ancestry_2020-1,
	title = {Ancestry deconvolution and partial polygenic score can improve susceptibility predictions in recently admixed individuals},
	volume = {11},
	copyright = {2020 The Author(s)},
	issn = {2041-1723},
	url = {https://www.nature.com/articles/s41467-020-15464-w},
	doi = {10.1038/s41467-020-15464-w},
	abstract = {Polygenic Scores (PSs) describe the genetic component of an individual’s quantitative phenotype or their susceptibility to diseases with a genetic basis. Currently, PSs rely on population-dependent contributions of many associated alleles, with limited applicability to understudied populations and recently admixed individuals. Here we introduce a combination of local ancestry deconvolution and partial PS computation to account for the population-specific nature of the association signals in individuals with admixed ancestry. We demonstrate partial PS to be a proxy for the total PS and that a portion of the genome is enough to improve susceptibility predictions for the traits we test. By combining partial PSs from different populations, we are able to improve trait predictability in admixed individuals with some European ancestry. These results may extend the applicability of PSs to subjects with a complex history of admixture, where current methods cannot be applied.},
	language = {en},
	number = {1},
	urldate = {2020-07-06},
	journal = {Nature Communications},
	author = {Marnetto, Davide and Pärna, Katri and Läll, Kristi and Molinaro, Ludovica and Montinaro, Francesco and Haller, Toomas and Metspalu, Mait and Mägi, Reedik and Fischer, Krista and Pagani, Luca},
	month = apr,
	year = {2020},
	note = {Number: 1
Publisher: Nature Publishing Group},
	pages = {1628},
}

@misc{noauthor_zotero_nodate-2,
	title = {Zotero {\textbar} {Your} personal research assistant},
	url = {https://www.zotero.org/start},
	urldate = {2020-07-06},
}

@article{kimura_stepping_1964,
	title = {The {Stepping} {Stone} {Model} of {Population} {Structure} and the {Decrease} of {Genetic} {Correlation} with {Distance}},
	volume = {49},
	issn = {0016-6731},
	language = {eng},
	number = {4},
	journal = {Genetics},
	author = {Kimura, M. and Weiss, G. H.},
	month = apr,
	year = {1964},
	pmid = {17248204},
	pmcid = {PMC1210594},
	pages = {561--576},
}

@misc{noauthor_stepping_nodate,
	title = {{THE} {STEPPING} {STONE} {MODEL} {OF} {POPULATION} {STRUCTURE} {AND} {THE} {DECREASE} {OF} {GENETIC} {CORRELATION} {WITH} {DISTANCE} {\textbar} {Genetics}},
	url = {https://www.genetics.org/content/49/4/561},
	urldate = {2020-06-04},
}

@article{kiss_disease_2005,
	title = {Disease contact tracing in random and clustered networks},
	volume = {272},
	issn = {0962-8452},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1560336/},
	doi = {10.1098/rspb.2005.3092},
	abstract = {The efficacy of contact tracing, be it between individuals (e.g. sexually transmitted diseases or severe acute respiratory syndrome) or between groups of individuals (e.g. foot-and-mouth disease; FMD), is difficult to evaluate without precise knowledge of the underlying contact structure; i.e. who is connected to whom? Motivated by the 2001 FMD epidemic in the UK, we determine, using stochastic simulations and deterministic ‘moment closure’ models of disease transmission on networks of premises (nodes), network and disease properties that are important for contact tracing efficiency. For random networks with a high average number of connections per node, little clustering of connections and short latency periods, contact tracing is typically ineffective. In this case, isolation of infected nodes is the dominant factor in determining disease epidemic size and duration. If the latency period is longer and the average number of connections per node small, or if the network is spatially clustered, then the contact tracing performs better and an overall reduction in the proportion of nodes that are removed during an epidemic is observed.},
	number = {1570},
	urldate = {2020-06-04},
	journal = {Proceedings of the Royal Society B: Biological Sciences},
	author = {Kiss, Istvan Z and Green, Darren M and Kao, Rowland R},
	month = jul,
	year = {2005},
	pmid = {16006334},
	pmcid = {PMC1560336},
	pages = {1407--1414},
}

@incollection{keeling_spatial_2008,
	title = {Spatial {Models}},
	isbn = {978-0-691-11617-4},
	url = {https://www.jstor.org/stable/j.ctvcm4gk0.10},
	abstract = {It is intuitive that in most circumstances disease transmission is predominantly a localized process. For directly transmitted diseases, for example, transmission is most likely between individuals with the most intense interaction, which generally implies those in the same location. Additionally, movement of individuals between population centers facilitates the geographical spread of infectious diseases. This chapter is concerned with capturing these host population characteristics, enabling us to address issues such as: determining the rate of spatial spread of a pathogen, calculating the influence of large populations on smaller ones, and finding optimally targeted control measures that take into account the local},
	urldate = {2020-06-04},
	booktitle = {Modeling {Infectious} {Diseases} in {Humans} and {Animals}},
	publisher = {Princeton University Press},
	author = {Keeling, Matt J. and Rohani, Pejman},
	year = {2008},
	doi = {10.2307/j.ctvcm4gk0.10},
	pages = {232--290},
}

@article{paeng_continuous_2017,
	title = {Continuous and discrete {SIR}-models with spatial distributions},
	volume = {74},
	issn = {1432-1416},
	url = {https://doi.org/10.1007/s00285-016-1071-8},
	doi = {10.1007/s00285-016-1071-8},
	abstract = {The SIR-model is a basic epidemic model that classifies a population into three subgroups: susceptible S, infected I and removed R. This model does not take into consideration the spatial distribution of each subgroup, but considers the total number of individuals belonging to each subgroup. There are many variants of the SIR-model. For studying the spatial distribution, stochastic processes have often been introduced to describe the dispersion of individuals. Such assumptions do not seem to be applicable to humans, because almost everyone moves within a small fixed radius in practice. Even if individuals do not disperse, the transmission of disease occurs. In this paper, we do not assume the dispersion of individuals, and instead use the infectious radius. Then, we propose simple continuous and discrete SIR-models that show spatial distributions. The results of our simulations show that the propagation speed and size of an epidemic depend on the population density and the infectious radius.},
	language = {en},
	number = {7},
	urldate = {2020-06-04},
	journal = {Journal of Mathematical Biology},
	author = {Paeng, Seong-Hun and Lee, Jonggul},
	month = jun,
	year = {2017},
	pages = {1709--1727},
}

@article{hamada_equilibrium_2019,
	title = {Equilibrium properties of the spatial {SIS} model as a point pattern dynamics - {How} is infection distributed over space?},
	volume = {468},
	issn = {0022-5193},
	url = {http://www.sciencedirect.com/science/article/pii/S0022519319300608},
	doi = {10.1016/j.jtbi.2019.02.005},
	abstract = {We revisit the classical epidemiological SIS model as a stochastic point pattern dynamics with special focus on its spatial distribution at equilibrium. In this model, each point on a continuous space is either susceptible S or infectious I, and infection occurs with an infection kernel as a function of distance from I to S. This stochastic process has been mathematically described by the hierarchical dynamics of the probabilities that a point, a pair made by two points, and a triplet made by three points, etc., is in a specific configuration of status. Using a simple closure thereby triplet probabilities that appear in the dynamics are approximated, we show that the average singlet probabilities and the pair probabilities that describe spatial distributions of Ss and Is at equilibrium can be explicitly derived using the infection kernel; Is are spatially clustered in the same order of the infection kernel. The results highlight the advantage of point pattern approach to model spatial population dynamics in general ecology where local interactions among individuals likely depend on distance between them.},
	language = {en},
	urldate = {2020-06-04},
	journal = {Journal of Theoretical Biology},
	author = {Hamada, Miki and Takasu, Fugo},
	month = may,
	year = {2019},
	keywords = {Distance-dependent interactions, Epidemiology, Spatial ecology, Spatial population dynamics},
	pages = {12--26},
}

@article{kao_impact_2003,
	title = {The impact of local heterogeneity on alternative control strategies for foot-and-mouth disease},
	volume = {270},
	issn = {0962-8452},
	doi = {10.1098/rspb.2003.2546},
	abstract = {The 2001 epidemic of foot-and-mouth disease (FMD) in the UK resulted in the death of nearly 10 million livestock at a cost that was estimated to be up to 8 billion pounds. Owing to the controversy surrounding the epidemic, the question of whether or not alternative policies would have resulted in significantly better control of the epidemic remains of great interest. A hexagonal lattice simulation of FMD in Cumbria is used to address the central question of whether or not better use could have been made of expert knowledge of FMD transmission to target pre-emptive culling, by assuming that the premises at greatest risk of becoming infected can be targeted for culling. The 2000 UK census and the epidemiological database collected during the epidemic are used to describe key characteristics of disease transmission, and the model is fit to the epidemic time-series. Under the assumptions of the model, the parameters that best fit the epidemic in Cumbria indicate that a policy based on expert knowledge would have exacerbated the epidemic compared with the policy as implemented. However, targeting more distant, high-risk farms could be more valuable under different epidemic conditions, notably, if risk factors of sufficient magnitude could be identified to aid in prioritizing vaccination or culling of farms at high risk of becoming infected.},
	language = {eng},
	number = {1533},
	journal = {Proceedings. Biological Sciences},
	author = {Kao, Rowland R.},
	month = dec,
	year = {2003},
	pmid = {14728777},
	pmcid = {PMC1691549},
	keywords = {Animals, Cattle, Demography, Disease Outbreaks, Disease Transmission, Infectious, Foot-and-Mouth Disease, Models, Biological, United Kingdom, Vaccination},
	pages = {2557--2564},
}

@article{hayama_mathematical_2013,
	title = {Mathematical model of the 2010 foot-and-mouth disease epidemic in {Japan} and evaluation of control measures},
	volume = {112},
	issn = {0167-5877},
	url = {http://www.sciencedirect.com/science/article/pii/S0167587713002754},
	doi = {10.1016/j.prevetmed.2013.08.010},
	abstract = {A large-scale foot-and-mouth disease (FMD) epidemic occurred in Japan in 2010. The epidemic arose in an area densely populated with cattle and pigs, continued for 3 months, and was contained by emergency vaccination. In this study, a mathematical simulation model of FMD transmission between farms was developed to generate the disease spread in the affected area. First, a farm-distance-based transmission kernel was estimated using the epidemic data. The estimated transmission kernel was then incorporated into the transmission model to evaluate the effectiveness of several control measures. The baseline model provided a good fit to the observed data during the period from imposition of movement restrictions until the implementation of vaccination. Our simulation results demonstrated that prompt culling on infected farms after detection could contribute to reducing the disease spread. The number of infected farms decreased to 30\% of the baseline model by applying the 24-h prompt culling scenario. The early detection scenario resulted in a smaller-sized epidemic. However, the results of this scenario included a 35\% chance of large-scale epidemic (more than 500 infected farms), even when the disease was detected 14 days earlier than in the baseline model. As additional options, preemptive culling could halt the epidemic more effectively. However, the preemptive culling scenario required substantial resources for culling operations. The 1-km preemptive scenario involved more than 50 farms remaining to be culled per day. Therefore, preemptive culling scenarios accompanied some difficulties in maintaining a sufficient capacity for culling in the affected area. A 10-km vaccination 7 days after the first detection of the disease demonstrated the potential to contain the epidemic to a small scale, while implementation of a 3-km vaccination on the same day could not effectively reduce epidemic size. In vaccination scenarios, the total number of farms that were either culled or vaccinated exceeded that of the baseline model. Vaccination scenarios therefore posed a problem of appropriate management of many vaccinated animals, whether these vaccinated animals would be culled or not. The present FMD transmission model developed using the 2010 FMD epidemic data in Japan provides useful information for consideration of suitable control strategies against FMD.},
	language = {en},
	number = {3},
	urldate = {2020-06-04},
	journal = {Preventive Veterinary Medicine},
	author = {Hayama, Y. and Yamamoto, T. and Kobayashi, S. and Muroga, N. and Tsutsui, T.},
	month = nov,
	year = {2013},
	keywords = {Control measures, FMD epidemic in Japan 2010, Mathematical model, Transmission kernel},
	pages = {183--193},
}

@article{hayama_mathematical_2013-1,
	title = {Mathematical model of the 2010 foot-and-mouth disease epidemic in {Japan} and evaluation of control measures},
	volume = {112},
	issn = {0167-5877},
	url = {http://www.sciencedirect.com/science/article/pii/S0167587713002754},
	doi = {10.1016/j.prevetmed.2013.08.010},
	abstract = {A large-scale foot-and-mouth disease (FMD) epidemic occurred in Japan in 2010. The epidemic arose in an area densely populated with cattle and pigs, continued for 3 months, and was contained by emergency vaccination. In this study, a mathematical simulation model of FMD transmission between farms was developed to generate the disease spread in the affected area. First, a farm-distance-based transmission kernel was estimated using the epidemic data. The estimated transmission kernel was then incorporated into the transmission model to evaluate the effectiveness of several control measures. The baseline model provided a good fit to the observed data during the period from imposition of movement restrictions until the implementation of vaccination. Our simulation results demonstrated that prompt culling on infected farms after detection could contribute to reducing the disease spread. The number of infected farms decreased to 30\% of the baseline model by applying the 24-h prompt culling scenario. The early detection scenario resulted in a smaller-sized epidemic. However, the results of this scenario included a 35\% chance of large-scale epidemic (more than 500 infected farms), even when the disease was detected 14 days earlier than in the baseline model. As additional options, preemptive culling could halt the epidemic more effectively. However, the preemptive culling scenario required substantial resources for culling operations. The 1-km preemptive scenario involved more than 50 farms remaining to be culled per day. Therefore, preemptive culling scenarios accompanied some difficulties in maintaining a sufficient capacity for culling in the affected area. A 10-km vaccination 7 days after the first detection of the disease demonstrated the potential to contain the epidemic to a small scale, while implementation of a 3-km vaccination on the same day could not effectively reduce epidemic size. In vaccination scenarios, the total number of farms that were either culled or vaccinated exceeded that of the baseline model. Vaccination scenarios therefore posed a problem of appropriate management of many vaccinated animals, whether these vaccinated animals would be culled or not. The present FMD transmission model developed using the 2010 FMD epidemic data in Japan provides useful information for consideration of suitable control strategies against FMD.},
	language = {en},
	number = {3},
	urldate = {2020-06-04},
	journal = {Preventive Veterinary Medicine},
	author = {Hayama, Y. and Yamamoto, T. and Kobayashi, S. and Muroga, N. and Tsutsui, T.},
	month = nov,
	year = {2013},
	keywords = {Control measures, FMD epidemic in Japan 2010, Mathematical model, Transmission kernel},
	pages = {183--193},
}

@article{saint_pierre_how_2014,
	title = {How important are rare variants in common disease?},
	volume = {13},
	issn = {2041-2649},
	url = {https://academic.oup.com/bfg/article/13/5/353/2846072},
	doi = {10.1093/bfgp/elu025},
	abstract = {Abstract.  Genome-wide association studies have uncovered hundreds of common genetic variants involved in complex diseases. However, for most complex diseases,},
	language = {en},
	number = {5},
	urldate = {2020-05-25},
	journal = {Briefings in Functional Genomics},
	author = {Saint Pierre, Aude and Génin, Emmanuelle},
	month = sep,
	year = {2014},
	note = {Publisher: Oxford Academic},
	pages = {353--361},
}

@article{lander_initial_2001,
	title = {Initial sequencing and analysis of the human genome},
	volume = {409},
	copyright = {2001 Macmillan Magazines Ltd.},
	issn = {1476-4687},
	url = {https://www.nature.com/articles/35057062},
	doi = {10.1038/35057062},
	abstract = {The human genome holds an extraordinary trove of information about human development, physiology, medicine and evolution. Here we report the results of an international collaboration to produce and make freely available a draft sequence of the human genome. We also present an initial analysis of the data, describing some of the insights that can be gleaned from the sequence.},
	language = {en},
	number = {6822},
	urldate = {2020-05-25},
	journal = {Nature},
	author = {Lander, Eric S. and Linton, Lauren M. and Birren, Bruce and Nusbaum, Chad and Zody, Michael C. and Baldwin, Jennifer and Devon, Keri and Dewar, Ken and Doyle, Michael and FitzHugh, William and Funke, Roel and Gage, Diane and Harris, Katrina and Heaford, Andrew and Howland, John and Kann, Lisa and Lehoczky, Jessica and LeVine, Rosie and McEwan, Paul and McKernan, Kevin and Meldrim, James and Mesirov, Jill P. and Miranda, Cher and Morris, William and Naylor, Jerome and Raymond, Christina and Rosetti, Mark and Santos, Ralph and Sheridan, Andrew and Sougnez, Carrie and Stange-Thomann, Nicole and Stojanovic, Nikola and Subramanian, Aravind and Wyman, Dudley and Rogers, Jane and Sulston, John and Ainscough, Rachael and Beck, Stephan and Bentley, David and Burton, John and Clee, Christopher and Carter, Nigel and Coulson, Alan and Deadman, Rebecca and Deloukas, Panos and Dunham, Andrew and Dunham, Ian and Durbin, Richard and French, Lisa and Grafham, Darren and Gregory, Simon and Hubbard, Tim and Humphray, Sean and Hunt, Adrienne and Jones, Matthew and Lloyd, Christine and McMurray, Amanda and Matthews, Lucy and Mercer, Simon and Milne, Sarah and Mullikin, James C. and Mungall, Andrew and Plumb, Robert and Ross, Mark and Shownkeen, Ratna and Sims, Sarah and Waterston, Robert H. and Wilson, Richard K. and Hillier, LaDeana W. and McPherson, John D. and Marra, Marco A. and Mardis, Elaine R. and Fulton, Lucinda A. and Chinwalla, Asif T. and Pepin, Kymberlie H. and Gish, Warren R. and Chissoe, Stephanie L. and Wendl, Michael C. and Delehaunty, Kim D. and Miner, Tracie L. and Delehaunty, Andrew and Kramer, Jason B. and Cook, Lisa L. and Fulton, Robert S. and Johnson, Douglas L. and Minx, Patrick J. and Clifton, Sandra W. and Hawkins, Trevor and Branscomb, Elbert and Predki, Paul and Richardson, Paul and Wenning, Sarah and Slezak, Tom and Doggett, Norman and Cheng, Jan-Fang and Olsen, Anne and Lucas, Susan and Elkin, Christopher and Uberbacher, Edward and Frazier, Marvin and Gibbs, Richard A. and Muzny, Donna M. and Scherer, Steven E. and Bouck, John B. and Sodergren, Erica J. and Worley, Kim C. and Rives, Catherine M. and Gorrell, James H. and Metzker, Michael L. and Naylor, Susan L. and Kucherlapati, Raju S. and Nelson, David L. and Weinstock, George M. and Sakaki, Yoshiyuki and Fujiyama, Asao and Hattori, Masahira and Yada, Tetsushi and Toyoda, Atsushi and Itoh, Takehiko and Kawagoe, Chiharu and Watanabe, Hidemi and Totoki, Yasushi and Taylor, Todd and Weissenbach, Jean and Heilig, Roland and Saurin, William and Artiguenave, Francois and Brottier, Philippe and Bruls, Thomas and Pelletier, Eric and Robert, Catherine and Wincker, Patrick and Rosenthal, André and Platzer, Matthias and Nyakatura, Gerald and Taudien, Stefan and Rump, Andreas and Smith, Douglas R. and Doucette-Stamm, Lynn and Rubenfield, Marc and Weinstock, Keith and Lee, Hong Mei and Dubois, JoAnn and Yang, Huanming and Yu, Jun and Wang, Jian and Huang, Guyang and Gu, Jun and Hood, Leroy and Rowen, Lee and Madan, Anup and Qin, Shizen and Davis, Ronald W. and Federspiel, Nancy A. and Abola, A. Pia and Proctor, Michael J. and Roe, Bruce A. and Chen, Feng and Pan, Huaqin and Ramser, Juliane and Lehrach, Hans and Reinhardt, Richard and McCombie, W. Richard and de la Bastide, Melissa and Dedhia, Neilay and Blöcker, Helmut and Hornischer, Klaus and Nordsiek, Gabriele and Agarwala, Richa and Aravind, L. and Bailey, Jeffrey A. and Bateman, Alex and Batzoglou, Serafim and Birney, Ewan and Bork, Peer and Brown, Daniel G. and Burge, Christopher B. and Cerutti, Lorenzo and Chen, Hsiu-Chuan and Church, Deanna and Clamp, Michele and Copley, Richard R. and Doerks, Tobias and Eddy, Sean R. and Eichler, Evan E. and Furey, Terrence S. and Galagan, James and Gilbert, James G. R. and Harmon, Cyrus and Hayashizaki, Yoshihide and Haussler, David and Hermjakob, Henning and Hokamp, Karsten and Jang, Wonhee and Johnson, L. Steven and Jones, Thomas A. and Kasif, Simon and Kaspryzk, Arek and Kennedy, Scot and Kent, W. James and Kitts, Paul and Koonin, Eugene V. and Korf, Ian and Kulp, David and Lancet, Doron and Lowe, Todd M. and McLysaght, Aoife and Mikkelsen, Tarjei and Moran, John V. and Mulder, Nicola and Pollara, Victor J. and Ponting, Chris P. and Schuler, Greg and Schultz, Jörg and Slater, Guy and Smit, Arian F. A. and Stupka, Elia and Szustakowki, Joseph and Thierry-Mieg, Danielle and Thierry-Mieg, Jean and Wagner, Lukas and Wallis, John and Wheeler, Raymond and Williams, Alan and Wolf, Yuri I. and Wolfe, Kenneth H. and Yang, Shiaw-Pyng and Yeh, Ru-Fang and Collins, Francis and Guyer, Mark S. and Peterson, Jane and Felsenfeld, Adam and Wetterstrand, Kris A. and Myers, Richard M. and Schmutz, Jeremy and Dickson, Mark and Grimwood, Jane and Cox, David R. and Olson, Maynard V. and Kaul, Rajinder and Raymond, Christopher and Shimizu, Nobuyoshi and Kawasaki, Kazuhiko and Minoshima, Shinsei and Evans, Glen A. and Athanasiou, Maria and Schultz, Roger and Patrinos, Aristides and Morgan, Michael J. and {International Human Genome Sequencing Consortium} and Whitehead Institute for Biomedical Research, Center for Genome Research: and {The Sanger Centre:} and {Washington University Genome Sequencing Center} and {US DOE Joint Genome Institute:} and {Baylor College of Medicine Human Genome Sequencing Center:} and {RIKEN Genomic Sciences Center:} and {Genoscope and CNRS UMR-8030:} and Department of Genome Analysis, Institute of Molecular Biotechnology: and {GTC Sequencing Center:} and {Beijing Genomics Institute/Human Genome Center:} and Multimegabase Sequencing Center, The Institute for Systems Biology: and {Stanford Genome Technology Center:} and {University of Oklahoma's Advanced Center for Genome Technology:} and {Max Planck Institute for Molecular Genetics:} and Cold Spring Harbor Laboratory, Lita Annenberg Hazen Genome Center: and {GBF—German Research Centre for Biotechnology:} and *Genome Analysis Group (listed in alphabetical order, also includes individuals listed under other headings): and Scientific management: National Human Genome Research Institute, US National Institutes of Health: and {Stanford Human Genome Center:} and {University of Washington Genome Center:} and Department of Molecular Biology, Keio University School of Medicine: and {University of Texas Southwestern Medical Center at Dallas:} and Office of Science, US Department of Energy: and {The Wellcome Trust:}},
	month = feb,
	year = {2001},
	note = {Number: 6822
Publisher: Nature Publishing Group},
	pages = {860--921},
}

@article{ahearn_height_2009,
	title = {Height and the {Normal} {Distribution}: {Evidence} from {Italian} {Military} {Data}},
	volume = {46},
	issn = {0070-3370},
	shorttitle = {Height and the {Normal} {Distribution}},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2831262/},
	abstract = {Researchers modeling historical heights have typically relied on the restrictive assumption of a normal distribution, only the mean of which is affected by age, income, nutrition, disease, and similar influences. To avoid these restrictive assumptions, we develop a new semiparametric approach in which covariates are allowed to affect the entire distribution without imposing any parametric shape. We apply our method to a new database of height distributions for Italian provinces, drawn from conscription records, of unprecedented length and geographical disaggregation. Our method allows us to standardize distributions to a single age and calculate moments of the distribution that are comparable through time. Our method also allows us to generate counterfactual distributions for a range of ages, from which we derive age-height profiles. These profiles reveal how the adolescent growth spurt (AGS) distorts the distribution of stature, and they document the earlier and earlier onset of the AGS as living conditions improved over the second half of the nineteenth century. Our new estimates of provincial mean height also reveal a previously unnoticed “regime switch” from regional convergence to divergence in this period.},
	number = {1},
	urldate = {2020-05-25},
	journal = {Demography},
	author = {A’HEARN, BRIAN and PERACCHI, FRANCO and VECCHI, GIOVANNI},
	month = feb,
	year = {2009},
	pmid = {19348106},
	pmcid = {PMC2831262},
	pages = {1--25},
}

@article{boyle_expanded_2017,
	title = {An expanded view of complex traits: from polygenic to omnigenic},
	volume = {169},
	issn = {0092-8674},
	shorttitle = {An expanded view of complex traits},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5536862/},
	doi = {10.1016/j.cell.2017.05.038},
	abstract = {A central goal of genetics is to understand the links between genetic variation and disease. Intuitively, one might expect disease-causing variants to cluster into key pathways that drive disease etiology. But for complex traits, association signals tend to be spread across most of the genome–including near many genes without an obvious connection to disease. We propose that gene regulatory networks are sufficiently interconnected that all genes expressed in disease-relevant cells are liable to affect the functions of core disease-related genes and that most heritability can be explained by effects on genes outside core pathways. We refer to this hypothesis as an “omnigenic” model.},
	number = {7},
	urldate = {2020-05-25},
	journal = {Cell},
	author = {Boyle, Evan A. and Li, Yang I. and Pritchard, Jonathan K.},
	month = jun,
	year = {2017},
	pmid = {28622505},
	pmcid = {PMC5536862},
	pages = {1177--1186},
}

@article{boyle_expanded_2017-1,
	title = {An expanded view of complex traits: from polygenic to omnigenic},
	volume = {169},
	issn = {0092-8674},
	shorttitle = {An expanded view of complex traits},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5536862/},
	doi = {10.1016/j.cell.2017.05.038},
	abstract = {A central goal of genetics is to understand the links between genetic variation and disease. Intuitively, one might expect disease-causing variants to cluster into key pathways that drive disease etiology. But for complex traits, association signals tend to be spread across most of the genome–including near many genes without an obvious connection to disease. We propose that gene regulatory networks are sufficiently interconnected that all genes expressed in disease-relevant cells are liable to affect the functions of core disease-related genes and that most heritability can be explained by effects on genes outside core pathways. We refer to this hypothesis as an “omnigenic” model.},
	number = {7},
	urldate = {2020-05-25},
	journal = {Cell},
	author = {Boyle, Evan A. and Li, Yang I. and Pritchard, Jonathan K.},
	month = jun,
	year = {2017},
	pmid = {28622505},
	pmcid = {PMC5536862},
	pages = {1177--1186},
}

@article{lapierre_accuracy_2017,
	title = {Accuracy of {Demographic} {Inferences} from the {Site} {Frequency} {Spectrum}: {The} {Case} of the {Yoruba} {Population}},
	volume = {206},
	copyright = {Copyright © 2017 by the Genetics Society of America},
	issn = {0016-6731, 1943-2631},
	shorttitle = {Accuracy of {Demographic} {Inferences} from the {Site} {Frequency} {Spectrum}},
	url = {https://www.genetics.org/content/206/1/439},
	doi = {10.1534/genetics.116.192708},
	abstract = {Some methods for demographic inference based on the observed genetic diversity of current populations rely on the use of summary statistics such as the Site Frequency Spectrum (SFS). Demographic models can be either model-constrained with numerous parameters, such as growth rates, timing of demographic events, and migration rates, or model-flexible, with an unbounded collection of piecewise constant sizes. It is still debated whether demographic histories can be accurately inferred based on the SFS. Here, we illustrate this theoretical issue on an example of demographic inference for an African population. The SFS of the Yoruba population (data from the 1000 Genomes Project) is fit to a simple model of population growth described with a single parameter (e.g., founding time). We infer a time to the most recent common ancestor of 1.7 million years (MY) for this population. However, we show that the Yoruba SFS is not informative enough to discriminate between several different models of growth. We also show that for such simple demographies, the fit of one-parameter models outperforms the stairway plot, a recently developed model-flexible method. The use of this method on simulated data suggests that it is biased by the noise intrinsically present in the data.},
	language = {en},
	number = {1},
	urldate = {2020-05-20},
	journal = {Genetics},
	author = {Lapierre, Marguerite and Lambert, Amaury and Achaz, Guillaume},
	month = may,
	year = {2017},
	pmid = {28341655},
	note = {Publisher: Genetics
Section: Investigations},
	keywords = {coalescent theory, human demography, model identifiability, site frequency spectrum},
	pages = {439--449},
}

@article{haller_slim_2019,
	title = {{SLiM} 3: {Forward} {Genetic} {Simulations} {Beyond} the {Wright}–{Fisher} {Model}},
	volume = {36},
	issn = {0737-4038},
	shorttitle = {{SLiM} 3},
	url = {https://academic.oup.com/mbe/article/36/3/632/5229931},
	doi = {10.1093/molbev/msy228},
	abstract = {Abstract.  With the desire to model population genetic processes under increasingly realistic scenarios, forward genetic simulations have become a critical part},
	language = {en},
	number = {3},
	urldate = {2020-05-18},
	journal = {Molecular Biology and Evolution},
	author = {Haller, Benjamin C. and Messer, Philipp W.},
	month = mar,
	year = {2019},
	note = {Publisher: Oxford Academic},
	pages = {632--637},
}

@article{manolio_finding_2009,
	title = {Finding the missing heritability of complex diseases},
	volume = {461},
	issn = {0028-0836},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2831613/},
	doi = {10.1038/nature08494},
	abstract = {Genome-wide association studies have identified hundreds of genetic variants associated with complex human diseases and traits, and have provided valuable insights into their genetic architecture. Most variants identified so far confer relatively small increments in risk, and explain only a small proportion of familial clustering, leading many to question how the remaining, ‘missing’ heritability can be explained. Here we examine potential sources of missing heritability and propose research strategies, including and extending beyond current genome-wide association approaches, to illuminate the genetics of complex diseases and enhance its potential to enable effective disease prevention or treatment.},
	number = {7265},
	urldate = {2020-05-18},
	journal = {Nature},
	author = {Manolio, Teri A. and Collins, Francis S. and Cox, Nancy J. and Goldstein, David B. and Hindorff, Lucia A. and Hunter, David J. and McCarthy, Mark I. and Ramos, Erin M. and Cardon, Lon R. and Chakravarti, Aravinda and Cho, Judy H. and Guttmacher, Alan E. and Kong, Augustine and Kruglyak, Leonid and Mardis, Elaine and Rotimi, Charles N. and Slatkin, Montgomery and Valle, David and Whittemore, Alice S. and Boehnke, Michael and Clark, Andrew G. and Eichler, Evan E. and Gibson, Greg and Haines, Jonathan L. and Mackay, Trudy F. C. and McCarroll, Steven A. and Visscher, Peter M.},
	month = oct,
	year = {2009},
	pmid = {19812666},
	pmcid = {PMC2831613},
	pages = {747--753},
}

@article{marcus_visualizing_2017,
	title = {Visualizing the geography of genetic variants},
	volume = {33},
	issn = {1367-4811},
	doi = {10.1093/bioinformatics/btw643},
	abstract = {Summary: One of the key characteristics of any genetic variant is its geographic distribution. The geographic distribution can shed light on where an allele first arose, what populations it has spread to, and in turn on how migration, genetic drift, and natural selection have acted. The geographic distribution of a genetic variant can also be of great utility for medical/clinical geneticists and collectively many genetic variants can reveal population structure. Here we develop an interactive visualization tool for rapidly displaying the geographic distribution of genetic variants. Through a REST API and dynamic front-end, the Geography of Genetic Variants (GGV) browser ( http://popgen.uchicago.edu/ggv/ ) provides maps of allele frequencies in populations distributed across the globe.
Availability and Implementation: GGV is implemented as a website ( http://popgen.uchicago.edu/ggv/ ) which employs an API to access frequency data ( http://popgen.uchicago.edu/freq\_api/ ). Python and javascript source code for the website and the API are available at: http://github.com/NovembreLab/ggv/ and http://github.com/NovembreLab/ggv-api/ .
Contact: jnovembre@uchicago.edu.
Supplementary information: Supplementary data are available at Bioinformatics online.},
	language = {eng},
	number = {4},
	journal = {Bioinformatics (Oxford, England)},
	author = {Marcus, Joseph H. and Novembre, John},
	year = {2017},
	pmid = {27742697},
	pmcid = {PMC5408806},
	keywords = {Genetic Variation, Genome, Human, Genomics, Humans, Phylogeography, Software},
	pages = {594--595},
}

@article{fisher_mathematical_1922,
	title = {On the mathematical foundations of theoretical statistics},
	volume = {222},
	url = {https://royalsocietypublishing.org/doi/abs/10.1098/rsta.1922.0009},
	doi = {10.1098/rsta.1922.0009},
	abstract = {Several reasons have contributed to the prolonged neglect into which the study of statistics, in its theoretical aspects, has fallen. In spite of the immense amount of fruitful labour which has been expended in its practical applications, the basic principles of this organ of science are still in a state of obscurity, and it cannot be denied that, during the recent rapid development of practical methods, fundamental problems have been ignored and fundamental paradoxes left unresolved. This anomalous state of statistical science is strikingly exemplified by a recent paper entitled "The Fundamental Problem of Practical Statistics," in which one of the most eminent of modern statisticians presents what purports to be a general proof of BAYES' postulate, a proof which, in the opinion of a second statistician of equal eminence, "seems to rest upon a very peculiar -- not to say hardly supposable -- relation."},
	number = {594-604},
	urldate = {2020-05-18},
	journal = {Philosophical Transactions of the Royal Society of London. Series A, Containing Papers of a Mathematical or Physical Character},
	author = {Fisher, R. A. and Russell, Edward John},
	month = jan,
	year = {1922},
	note = {Publisher: Royal Society},
	pages = {309--368},
}

@article{sawyer_branching_1976,
	title = {Branching diffusion processes in population genetics},
	volume = {8},
	issn = {0001-8678, 1475-6064},
	url = {https://www.cambridge.org/core/journals/advances-in-applied-probability/article/branching-diffusion-processes-in-population-genetics/4A1AA03D0DB6326864045195759FA156},
	doi = {10.2307/1425929},
	abstract = {A branching random field is considered as a model of either of two situations in genetics in which migration or dispersion plays a role. Specifically we consider the expected number of individuals NA
 in a (geographical) set A at time t, the covariance of NA
 and NB
 for two sets A, B, and the probability I(x, y, u) that two individuals found at locations x, y at time t are of the same genetic type if the population is subject to a selectively neutral mutation rate u. The last also leads to limit laws for the average degree of relationship of individuals in various types of branching random fields. We also find the equations that the mean and bivariate densities satisfy, and explicit formulas when the underlying migration process is Brownian motion.},
	language = {en},
	number = {4},
	urldate = {2020-05-18},
	journal = {Advances in Applied Probability},
	author = {Sawyer, Stanley},
	month = dec,
	year = {1976},
	note = {Publisher: Cambridge University Press},
	keywords = {BRANCHING PROCESS, BRANCHING RANDOM FIELD, GENETICS, IDENTITY BY DESCENT, MIGRATION, POINT PROCESS, POISSON RANDOM FIELD, STEPPING-STONE MODEL},
	pages = {659--689},
}

@article{malecot_heterozygosity_1975,
	title = {Heterozygosity and relationship in regularly subdivided populations},
	volume = {8},
	issn = {0040-5809},
	doi = {10.1016/0040-5809(75)90033-7},
	language = {eng},
	number = {2},
	journal = {Theoretical Population Biology},
	author = {Malécot, G.},
	month = oct,
	year = {1975},
	pmid = {1198353},
	keywords = {Fourier Analysis, Gene Frequency, Genetics, Population, Heterozygote, Models, Biological, Mutation},
	pages = {212--241},
}

@article{felsenstein_genetic_1975,
	title = {Genetic drift in clines which are maintained by migration and natural selection},
	volume = {81},
	issn = {0016-6731},
	abstract = {Genetic drift will cause a migration-selection cline to wobble about its expected position. A rough linear approximation is developed, valid when local populations are large. This is used to calculate effects of genetic drift on clines in a stepping-stone model with abrupt and with gradual changes of selection coefficients at a single haploid locus. Among the quantities calculated are measures of slope, standardized variation of gene frequencies around their expected values, and correlation among neighboring populations with respect to deviations from the expected gene frequencies. These quantities appear to be primarily functions of Ns and Nm for a given pattern of selection. Computer simulation gives rough confirmation of these results. Standardized variances of gene frequencies and correlation of neighbors differ along the cline in the case of smooth changes in selection. In no case is pathological behavior of gene frequency deviations found near the boundaries of selective regions. Local behavior of gene frequences of nearby colonies is approximately predicted by a simple adaptation of the stepping-stone theory of Kimura and Weiss. Approximate measures of the lateral variation of the midpoint of a cline and the probability of non-monotonicity are also calculated and discussed.},
	language = {eng},
	number = {1},
	journal = {Genetics},
	author = {Felsenstein, J.},
	month = sep,
	year = {1975},
	pmid = {1205125},
	pmcid = {PMC1213383},
	keywords = {Gene Frequency, Selection, Genetic, Statistics as Topic},
	pages = {191--207},
}

@article{nelson_population_2008,
	title = {The {Population} {Reference} {Sample}, {POPRES}: a resource for population, disease, and pharmacological genetics research},
	volume = {83},
	issn = {1537-6605},
	shorttitle = {The {Population} {Reference} {Sample}, {POPRES}},
	doi = {10.1016/j.ajhg.2008.08.005},
	abstract = {Technological and scientific advances, stemming in large part from the Human Genome and HapMap projects, have made large-scale, genome-wide investigations feasible and cost effective. These advances have the potential to dramatically impact drug discovery and development by identifying genetic factors that contribute to variation in disease risk as well as drug pharmacokinetics, treatment efficacy, and adverse drug reactions. In spite of the technological advancements, successful application in biomedical research would be limited without access to suitable sample collections. To facilitate exploratory genetics research, we have assembled a DNA resource from a large number of subjects participating in multiple studies throughout the world. This growing resource was initially genotyped with a commercially available genome-wide 500,000 single-nucleotide polymorphism panel. This project includes nearly 6,000 subjects of African-American, East Asian, South Asian, Mexican, and European origin. Seven informative axes of variation identified via principal-component analysis (PCA) of these data confirm the overall integrity of the data and highlight important features of the genetic structure of diverse populations. The potential value of such extensively genotyped collections is illustrated by selection of genetically matched population controls in a genome-wide analysis of abacavir-associated hypersensitivity reaction. We find that matching based on country of origin, identity-by-state distance, and multidimensional PCA do similarly well to control the type I error rate. The genotype and demographic data from this reference sample are freely available through the NCBI database of Genotypes and Phenotypes (dbGaP).},
	language = {eng},
	number = {3},
	journal = {American Journal of Human Genetics},
	author = {Nelson, Matthew R. and Bryc, Katarzyna and King, Karen S. and Indap, Amit and Boyko, Adam R. and Novembre, John and Briley, Linda P. and Maruyama, Yuka and Waterworth, Dawn M. and Waeber, Gérard and Vollenweider, Peter and Oksenberg, Jorge R. and Hauser, Stephen L. and Stirnadel, Heide A. and Kooner, Jaspal S. and Chambers, John C. and Jones, Brendan and Mooser, Vincent and Bustamante, Carlos D. and Roses, Allen D. and Burns, Daniel K. and Ehm, Margaret G. and Lai, Eric H.},
	month = sep,
	year = {2008},
	pmid = {18760391},
	pmcid = {PMC2556436},
	keywords = {Case-Control Studies, Databases, Genetic, Dideoxynucleosides, Drug Hypersensitivity, European Continental Ancestry Group, Female, Genetics, Population, Genome, Human, Genotype, Humans, Male, Pharmacogenetics, Polymorphism, Single Nucleotide, Population Groups},
	pages = {347--358},
}

@article{simons_population_2018,
	title = {A population genetic interpretation of {GWAS} findings for human quantitative traits},
	volume = {16},
	issn = {1545-7885},
	doi = {10.1371/journal.pbio.2002985},
	abstract = {Human genome-wide association studies (GWASs) are revealing the genetic architecture of anthropomorphic and biomedical traits, i.e., the frequencies and effect sizes of variants that contribute to heritable variation in a trait. To interpret these findings, we need to understand how genetic architecture is shaped by basic population genetics processes-notably, by mutation, natural selection, and genetic drift. Because many quantitative traits are subject to stabilizing selection and because genetic variation that affects one trait often affects many others, we model the genetic architecture of a focal trait that arises under stabilizing selection in a multidimensional trait space. We solve the model for the phenotypic distribution and allelic dynamics at steady state and derive robust, closed-form solutions for summary statistics of the genetic architecture. Our results provide a simple interpretation for missing heritability and why it varies among traits. They predict that the distribution of variances contributed by loci identified in GWASs is well approximated by a simple functional form that depends on a single parameter: the expected contribution to genetic variance of a strongly selected site affecting the trait. We test this prediction against the results of GWASs for height and body mass index (BMI) and find that it fits the data well, allowing us to make inferences about the degree of pleiotropy and mutational target size for these traits. Our findings help to explain why the GWAS for height explains more of the heritable variance than the similarly sized GWAS for BMI and to predict the increase in explained heritability with study sample size. Considering the demographic history of European populations, in which these GWASs were performed, we further find that most of the associations they identified likely involve mutations that arose shortly before or during the Out-of-Africa bottleneck at sites with selection coefficients around s = 10-3.},
	language = {eng},
	number = {3},
	journal = {PLoS biology},
	author = {Simons, Yuval B. and Bullaughey, Kevin and Hudson, Richard R. and Sella, Guy},
	year = {2018},
	pmid = {29547617},
	pmcid = {PMC5871013},
	keywords = {Body Height, Body Mass Index, Genetic Drift, Genetic Variation, Genetics, Population, Genome-Wide Association Study, Humans, Models, Genetic, Phenotype, Quantitative Trait Loci, Selection, Genetic},
	pages = {e2002985},
}

@article{pritchard_are_2001,
	title = {Are rare variants responsible for susceptibility to complex diseases?},
	volume = {69},
	issn = {0002-9297},
	doi = {10.1086/321272},
	abstract = {Little is known about the nature of genetic variation underlying complex diseases in humans. One popular view proposes that mapping efforts should focus on identification of susceptibility mutations that are relatively old and at high frequency. It is generally assumed-at least for modeling purposes-that selection against complex disease mutations is so weak that it can be ignored. In this article, I propose an explicit model for the evolution of complex disease loci, incorporating mutation, random genetic drift, and the possibility of purifying selection against susceptibility mutations. I show that, for the most plausible range of mutation rates, neutral susceptibility alleles are unlikely to be at intermediate frequencies and contribute little to the overall genetic variance for the disease. Instead, it seems likely that the bulk of genetic variance underlying diseases is due to loci where susceptibility mutations are mildly deleterious and where there is a high overall mutation rate to the susceptible class. At such loci, the total frequency of susceptibility mutations may be quite high, but there is likely to be extensive allelic heterogeneity at many of these loci. I discuss some practical implications of these results for gene mapping efforts.},
	language = {eng},
	number = {1},
	journal = {American Journal of Human Genetics},
	author = {Pritchard, J. K.},
	month = jul,
	year = {2001},
	pmid = {11404818},
	pmcid = {PMC1226027},
	keywords = {Alleles, Chromosome Mapping, Computer Simulation, Female, Gene Frequency, Genetic Diseases, Inborn, Genetic Heterogeneity, Genetic Linkage, Genetic Predisposition to Disease, Genetic Variation, Humans, Male, Models, Genetic, Mutagenesis, Mutation, Pedigree, Phenotype, Polymorphism, Genetic, Probability, Selection, Genetic, Software},
	pages = {124--137},
}

@article{eyre-walker_evolution_2010,
	title = {Evolution in health and medicine {Sackler} colloquium: {Genetic} architecture of a complex trait and its implications for fitness and genome-wide association studies},
	volume = {107 Suppl 1},
	issn = {1091-6490},
	shorttitle = {Evolution in health and medicine {Sackler} colloquium},
	doi = {10.1073/pnas.0906182107},
	abstract = {A model is investigated in which mutations that affect a complex trait (e.g., heart disease) also affect fitness because the trait is a component of fitness or because the mutations have pleiotropic effects on fitness. The model predicts that the genetic variance, and hence the heritability, in the trait is contributed by mutations at low frequency in the population, unless the mean strength of selection of mutations that affect the trait is very small or weakly selected mutations tend to contribute disproportionately to the trait compared with strongly selected mutations. Furthermore, it is shown that each rare mutation tends to contribute more to the variance than each common mutation. These results may explain why most genome-wide association studies have failed to find associations that explain much of the variance. It is also shown that most of the variance in fitness contributed by new nonsynonymous mutations is caused by mutations at very low frequency in the population. This implies that most low-frequency SNPs, which are observed in current resequencing studies of, for example, 100 chromosomes, probably have little impact on the variance in fitness or traits. Finally, it is shown that the variance contributed by a category of mutations (e.g., coding or regulatory) depends largely upon the mean strength of selection; this has implications for understanding which types of mutations are likely to be responsible for the variance in fitness and inherited disease.},
	language = {eng},
	journal = {Proceedings of the National Academy of Sciences of the United States of America},
	author = {Eyre-Walker, Adam},
	month = jan,
	year = {2010},
	pmid = {20133822},
	pmcid = {PMC2868283},
	keywords = {Genetic Variation, Genome-Wide Association Study, Humans, Mutation, Selection, Genetic},
	pages = {1752--1756},
}

@article{gilad_reanalysis_2015,
	title = {A reanalysis of mouse {ENCODE} comparative gene expression data},
	volume = {4},
	issn = {2046-1402},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4516019/},
	doi = {10.12688/f1000research.6536.1},
	abstract = {Recently, the Mouse ENCODE Consortium reported that comparative gene expression data from human and mouse tend to cluster more by species rather than by tissue. This observation was surprising, as it contradicted much of the comparative gene regulatory data collected previously, as well as the common notion that major developmental pathways are highly conserved across a wide range of species, in particular across mammals. Here we show that the Mouse ENCODE gene expression data were collected using a flawed study design, which confounded sequencing batch (namely, the assignment of samples to sequencing flowcells and lanes) with species. When we account for the batch effect, the corrected comparative gene expression data from human and mouse tend to cluster by tissue, not by species.},
	urldate = {2020-05-18},
	journal = {F1000Research},
	author = {Gilad, Yoav and Mizrahi-Man, Orna},
	month = may,
	year = {2015},
	pmid = {26236466},
	pmcid = {PMC4516019},
}

@article{mcvean_genealogical_2009,
	title = {A {Genealogical} {Interpretation} of {Principal} {Components} {Analysis}},
	volume = {5},
	issn = {1553-7404},
	url = {https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000686},
	doi = {10.1371/journal.pgen.1000686},
	abstract = {Principal components analysis, PCA, is a statistical method commonly used in population genetics to identify structure in the distribution of genetic variation across geographical location and ethnic background. However, while the method is often used to inform about historical demographic processes, little is known about the relationship between fundamental demographic parameters and the projection of samples onto the primary axes. Here I show that for SNP data the projection of samples onto the principal components can be obtained directly from considering the average coalescent times between pairs of haploid genomes. The result provides a framework for interpreting PCA projections in terms of underlying processes, including migration, geographical isolation, and admixture. I also demonstrate a link between PCA and Wright's fst and show that SNP ascertainment has a largely simple and predictable effect on the projection of samples. Using examples from human genetics, I discuss the application of these results to empirical data and the implications for inference.},
	language = {en},
	number = {10},
	urldate = {2020-05-18},
	journal = {PLOS Genetics},
	author = {McVean, Gil},
	month = oct,
	year = {2009},
	note = {Publisher: Public Library of Science},
	keywords = {Eigenvectors, Genetic polymorphism, Haplotypes, Molecular genetics, Phylogeography, Population genetics, Principal component analysis, Variant genotypes},
	pages = {e1000686},
}

@misc{noauthor_genealogical_nodate,
	title = {A {Genealogical} {Interpretation} of {Principal} {Components} {Analysis}},
	url = {https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000686},
	urldate = {2020-05-18},
}

@article{sigwart_coalescent_2009,
	title = {Coalescent {Theory}: {An} {Introduction}},
	volume = {58},
	issn = {1063-5157},
	url = {https://doi.org/10.1093/schbul/syp004},
	doi = {10.1093/schbul/syp004},
	abstract = {“The Coalescent” is a powerful extension of classical population genetics because it is a collection of mathematical models that can accommodate biological phenomena as reflected in genomic data. The theory was initially developed by Kingman (1982) in 3 papers published in probability theory journals, which outline the foundation of coalescent theory as a suite of probability models. Recent publications continue to address the mathematical development of the coalescent in mathematical journals (e.g., Sagitov and Jagers 2005), as well as addressing questions in real biological systems. The 2 important points to take from these facts are that, first, coalescent theory is still an active and exciting topic that continues to be developed in its fundamentals and applications and, second, that any book on coalescent theory is going to be heavy on the mathematics.},
	number = {1},
	journal = {Systematic Biology},
	author = {Sigwart, Julia},
	year = {2009},
	note = {\_eprint: https://academic.oup.com/sysbio/article-pdf/58/1/162/24204974/syp004.pdf},
	pages = {162--165},
}

@article{kingman_coalescent_1982,
	title = {The coalescent},
	volume = {13},
	issn = {0304-4149},
	url = {http://www.sciencedirect.com/science/article/pii/0304414982900114},
	doi = {10.1016/0304-4149(82)90011-4},
	abstract = {The n-coalescent is a continuous-time Markov chain on a finite set of states, which describes the family relationships among a sample of n members drawn from a large haploid population. Its transition probabilities can be calculated from a factorization of the chain into two independent components, a pure death process and a discrete-time jump chain. For a deeper study, it is useful to construct a more complicated Markov process in which n-coalescents for all values of n are embedded in a natural way.},
	language = {en},
	number = {3},
	urldate = {2020-05-18},
	journal = {Stochastic Processes and their Applications},
	author = {Kingman, J. F. C.},
	month = sep,
	year = {1982},
	keywords = {Genetical models, Markov process, coupling, exchangeability, haploid genealogy, jump chain, random equivalent relations},
	pages = {235--248},
}

@article{novembre_genes_2008,
	title = {Genes mirror geography within {Europe}},
	volume = {456},
	copyright = {2008 Macmillan Publishers Limited. All rights reserved},
	issn = {1476-4687},
	url = {https://www.nature.com/articles/nature07331},
	doi = {10.1038/nature07331},
	abstract = {The power of the latest massively parallel synthetic DNA sequencing technologies is demonstrated in two major collaborations that shed light on the nature of genomic variation with ethnicity. The first describes the genomic characterization of an individual from the Yoruba ethnic group of west Africa. The second reports a personal genome of a Han Chinese, the group comprising 30\% of the world's population. These new resources can now be used in conjunction with the Venter, Watson and NIH reference sequences. A separate study looked at genetic ethnicity on the continental scale, based on data from 1,387 individuals from more than 30 European countries. Overall there was little genetic variation between countries, but the differences that do exist correspond closely to the geographic map. Statistical analysis of the genome data places 50\% of the individuals within 310 km of their reported origin. As well as its relevance for testing genetic ancestry, this work has implications for evaluating genome-wide association studies that link genes with diseases.},
	language = {en},
	number = {7218},
	urldate = {2020-05-18},
	journal = {Nature},
	author = {Novembre, John and Johnson, Toby and Bryc, Katarzyna and Kutalik, Zoltán and Boyko, Adam R. and Auton, Adam and Indap, Amit and King, Karen S. and Bergmann, Sven and Nelson, Matthew R. and Stephens, Matthew and Bustamante, Carlos D.},
	month = nov,
	year = {2008},
	note = {Number: 7218
Publisher: Nature Publishing Group},
	pages = {98--101},
}

@article{lin_comparison_1994,
	title = {Comparison of 79 {DNA} polymorphisms tested in {Australians}, {Japanese} and {Papua} {New} {Guineans} with those of five other human populations},
	volume = {8},
	issn = {0394-249X},
	abstract = {Seventy-nine DNA polymorphisms from 57 loci (28 genes and 29 anonymous DNA segments) have been typed in eight human populations. Here we present allele frequencies for three populations (Japanese, New Guineans, and Australians) as well as revised frequencies for a Chinese sample: allele frequencies for five additional populations (Biaka and Mbuti Pygmies, Melanesians, Chinese, and Europeans) were described previously [Bowcock et al 1991a]. Evaluation of Hardy-Weinberg equilibrium for these polymorphisms suggested that the New Guinean sample may be from a highly substructured population. Average FST value for the 79 markers (polymorphisms) was 0.147 +/- 0.011 across the eight populations: Fst values for some markers changed dramatically with the addition of three populations--in particular, Australians and New Guineans. Average heterozygosity for eight populations was 0.307 +/- 0.014. Genetic distances indicated that the Australian sample may have some European ancestry. An average linkage tree inferred from these distances suggested that the first split of modern humans was between Africans and non-Africans, while the second major split was between Australians/New Guineans and all other non-Africans. The neighbor-joining tree also separated the African populations from all others. European polymorphism ascertainment bias and European admixture appear to have influenced both estimation of population heterozygosity and tree inference.},
	language = {eng},
	number = {3},
	journal = {Gene Geography: A Computerized Bulletin on Human Gene Frequencies},
	author = {Lin, A. A. and Hebert, J. M. and Mountain, J. L. and Cavalli-Sforza, L. L.},
	month = dec,
	year = {1994},
	pmid = {7662611},
	keywords = {Alleles, Animals, Australia, Chromosome Mapping, DNA, Gene Frequency, Genetic Markers, Heterozygote, Humans, Japan, Molecular Probes, Pan troglodytes, Papua New Guinea, Polymorphism, Genetic},
	pages = {191--214},
}

@article{popejoy_genomics_2016,
	title = {Genomics is failing on diversity},
	volume = {538},
	url = {http://www.nature.com/news/genomics-is-failing-on-diversity-1.20759},
	doi = {10.1038/538161a},
	abstract = {An analysis by Alice B. Popejoy and Stephanie M. Fullerton indicates that some populations are still being left behind on the road to precision medicine.},
	language = {en},
	number = {7624},
	urldate = {2020-05-18},
	journal = {Nature News},
	author = {Popejoy, Alice B. and Fullerton, Stephanie M.},
	month = oct,
	year = {2016},
	note = {Section: Comment},
	pages = {161},
}

@article{need_next_2009,
	title = {Next generation disparities in human genomics: concerns and remedies},
	volume = {25},
	issn = {0168-9525},
	shorttitle = {Next generation disparities in human genomics},
	url = {http://www.sciencedirect.com/science/article/pii/S0168952509001851},
	doi = {10.1016/j.tig.2009.09.012},
	abstract = {Studies of human genetics, particularly genome-wide association studies (GWAS), have concentrated heavily on European populations, with individuals of African ancestry rarely represented. Reasons for this include the distribution of biomedical funding and the increased population structure and reduced linkage disequilibrium in African populations. Currently, few GWAS findings have clinical utility and, therefore, the field has not yet contributed to health-care disparities. As human genomics research progresses towards the whole-genome sequencing era, however, more clinically relevant results are likely to be discovered. As we discuss here, to avoid the genetics community contributing to healthcare disparities, it is important to adopt measures to ensure that populations of diverse ancestry are included in genomic studies, and that no major population groups are excluded.},
	language = {en},
	number = {11},
	urldate = {2020-05-18},
	journal = {Trends in Genetics},
	author = {Need, Anna C. and Goldstein, David B.},
	month = nov,
	year = {2009},
	pages = {489--494},
}

@article{han_fine_2014,
	title = {Fine mapping seronegative and seropositive rheumatoid arthritis to shared and distinct {HLA} alleles by adjusting for the effects of heterogeneity},
	volume = {94},
	issn = {1537-6605},
	doi = {10.1016/j.ajhg.2014.02.013},
	abstract = {Despite progress in defining human leukocyte antigen (HLA) alleles for anti-citrullinated-protein-autoantibody-positive (ACPA(+)) rheumatoid arthritis (RA), identifying HLA alleles for ACPA-negative (ACPA(-)) RA has been challenging because of clinical heterogeneity within clinical cohorts. We imputed 8,961 classical HLA alleles, amino acids, and SNPs from Immunochip data in a discovery set of 2,406 ACPA(-) RA case and 13,930 control individuals. We developed a statistical approach to identify and adjust for clinical heterogeneity within ACPA(-) RA and observed independent associations for serine and leucine at position 11 in HLA-DRβ1 (p = 1.4 × 10(-13), odds ratio [OR] = 1.30) and for aspartate at position 9 in HLA-B (p = 2.7 × 10(-12), OR = 1.39) within the peptide binding grooves. These amino acid positions induced associations at HLA-DRB1(∗)03 (encoding serine at 11) and HLA-B(∗)08 (encoding aspartate at 9). We validated these findings in an independent set of 427 ACPA(-) case subjects, carefully phenotyped with a highly sensitive ACPA assay, and 1,691 control subjects (HLA-DRβ1 Ser11+Leu11: p = 5.8 × 10(-4), OR = 1.28; HLA-B Asp9: p = 2.6 × 10(-3), OR = 1.34). Although both amino acid sites drove risk of ACPA(+) and ACPA(-) disease, the effects of individual residues at HLA-DRβ1 position 11 were distinct (p {\textless} 2.9 × 10(-107)). We also identified an association with ACPA(+) RA at HLA-A position 77 (p = 2.7 × 10(-8), OR = 0.85) in 7,279 ACPA(+) RA case and 15,870 control subjects. These results contribute to mounting evidence that ACPA(+) and ACPA(-) RA are genetically distinct and potentially have separate autoantigens contributing to pathogenesis. We expect that our approach might have broad applications in analyzing clinical conditions with heterogeneity at both major histocompatibility complex (MHC) and non-MHC regions.},
	language = {eng},
	number = {4},
	journal = {American Journal of Human Genetics},
	author = {Han, Buhm and Diogo, Dorothée and Eyre, Steve and Kallberg, Henrik and Zhernakova, Alexandra and Bowes, John and Padyukov, Leonid and Okada, Yukinori and González-Gay, Miguel A. and Rantapää-Dahlqvist, Solbritt and Martin, Javier and Huizinga, Tom W. J. and Plenge, Robert M. and Worthington, Jane and Gregersen, Peter K. and Klareskog, Lars and de Bakker, Paul I. W. and Raychaudhuri, Soumya},
	month = apr,
	year = {2014},
	pmid = {24656864},
	pmcid = {PMC3980428},
	keywords = {Alleles, Arthritis, Rheumatoid, Case-Control Studies, Genetic Heterogeneity, HLA Antigens, Humans},
	pages = {522--532},
}

@article{abraham_accurate_2014,
	title = {Accurate and robust genomic prediction of celiac disease using statistical learning},
	volume = {10},
	issn = {1553-7404},
	doi = {10.1371/journal.pgen.1004137},
	abstract = {Practical application of genomic-based risk stratification to clinical diagnosis is appealing yet performance varies widely depending on the disease and genomic risk score (GRS) method. Celiac disease (CD), a common immune-mediated illness, is strongly genetically determined and requires specific HLA haplotypes. HLA testing can exclude diagnosis but has low specificity, providing little information suitable for clinical risk stratification. Using six European cohorts, we provide a proof-of-concept that statistical learning approaches which simultaneously model all SNPs can generate robust and highly accurate predictive models of CD based on genome-wide SNP profiles. The high predictive capacity replicated both in cross-validation within each cohort (AUC of 0.87-0.89) and in independent replication across cohorts (AUC of 0.86-0.9), despite differences in ethnicity. The models explained 30-35\% of disease variance and up to ∼43\% of heritability. The GRS's utility was assessed in different clinically relevant settings. Comparable to HLA typing, the GRS can be used to identify individuals without CD with ≥99.6\% negative predictive value however, unlike HLA typing, fine-scale stratification of individuals into categories of higher-risk for CD can identify those that would benefit from more invasive and costly definitive testing. The GRS is flexible and its performance can be adapted to the clinical situation by adjusting the threshold cut-off. Despite explaining a minority of disease heritability, our findings indicate a genomic risk score provides clinically relevant information to improve upon current diagnostic pathways for CD and support further studies evaluating the clinical utility of this approach in CD and other complex diseases.},
	language = {eng},
	number = {2},
	journal = {PLoS genetics},
	author = {Abraham, Gad and Tye-Din, Jason A. and Bhalala, Oneil G. and Kowalczyk, Adam and Zobel, Justin and Inouye, Michael},
	month = feb,
	year = {2014},
	pmid = {24550740},
	pmcid = {PMC3923679},
	keywords = {Alleles, Biometry, Celiac Disease, Female, Genetic Predisposition to Disease, Genome, Human, Genomics, HLA Antigens, Haplotypes, Humans, Polymorphism, Single Nucleotide, Risk},
	pages = {e1004137},
}

@misc{noauthor_genetic_nodate,
	title = {Genetic identification of a common collagen disease in {Puerto} {Ricans} via identity-by-descent mapping in a health system {\textbar} {eLife}},
	url = {https://elifesciences.org/articles/25060},
	urldate = {2020-05-18},
}

@article{visscher_five_2012,
	title = {Five {Years} of {GWAS} {Discovery}},
	volume = {90},
	issn = {0002-9297},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3257326/},
	doi = {10.1016/j.ajhg.2011.11.029},
	abstract = {The past five years have seen many scientific and biological discoveries made through the experimental design of genome-wide association studies (GWASs). These studies were aimed at detecting variants at genomic loci that are associated with complex traits in the population and, in particular, at detecting associations between common single-nucleotide polymorphisms (SNPs) and common diseases such as heart disease, diabetes, auto-immune diseases, and psychiatric disorders. We start by giving a number of quotes from scientists and journalists about perceived problems with GWASs. We will then briefly give the history of GWASs and focus on the discoveries made through this experimental design, what those discoveries tell us and do not tell us about the genetics and biology of complex traits, and what immediate utility has come out of these studies. Rather than giving an exhaustive review of all reported findings for all diseases and other complex traits, we focus on the results for auto-immune diseases and metabolic diseases. We return to the perceived failure or disappointment about GWASs in the concluding section.},
	number = {1},
	urldate = {2020-05-18},
	journal = {American Journal of Human Genetics},
	author = {Visscher, Peter M. and Brown, Matthew A. and McCarthy, Mark I. and Yang, Jian},
	month = jan,
	year = {2012},
	pmid = {22243964},
	pmcid = {PMC3257326},
	pages = {7--24},
}

@article{ganna_large-scale_2019,
	title = {Large-scale {GWAS} reveals insights into the genetic architecture of same-sex sexual behavior},
	volume = {365},
	copyright = {Copyright © 2019 The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. No claim to original U.S. Government Works. http://www.sciencemag.org/about/science-licenses-journal-article-reuseThis is an article distributed under the terms of the Science Journals Default License.},
	issn = {0036-8075, 1095-9203},
	url = {https://science.sciencemag.org/content/365/6456/eaat7693},
	doi = {10.1126/science.aat7693},
	abstract = {The genetics of sexual orientation
Twin studies and other analyses of inheritance of sexual orientation in humans has indicated that same-sex sexual behavior has a genetic component. Previous searches for the specific genes involved have been underpowered and thus unable to detect genetic signals. Ganna et al. perform a genome-wide association study on 493,001 participants from the United States, the United Kingdom, and Sweden to study genes associated with sexual orientation (see the Perspective by Mills). They find multiple loci implicated in same-sex sexual behavior indicating that, like other behavioral traits, nonheterosexual behavior is polygenic.
Science, this issue p. eaat7693; see also p. 869
Structured Abstract
INTRODUCTIONAcross human societies and in both sexes, some 2 to 10\% of individuals report engaging in sex with same-sex partners, either exclusively or in addition to sex with opposite-sex partners. Twin and family studies have shown that same-sex sexual behavior is partly genetically influenced, but previous searches for the specific genes involved have been underpowered to detect effect sizes realistic for complex traits.
RATIONALEFor the first time, new large-scale datasets afford sufficient statistical power to identify genetic variants associated with same-sex sexual behavior (ever versus never had a same-sex partner), estimate the proportion of variation in the trait accounted for by all variants in aggregate, estimate the genetic correlation of same-sex sexual behavior with other traits, and probe the biology and complexity of the trait. To these ends, we performed genome-wide association discovery analyses on 477,522 individuals from the United Kingdom and United States, replication analyses in 15,142 individuals from the United States and Sweden, and follow-up analyses using different aspects of sexual preference.
RESULTSIn the discovery samples (UK Biobank and 23andMe), five autosomal loci were significantly associated with same-sex sexual behavior. Follow-up of these loci suggested links to biological pathways that involve sex hormone regulation and olfaction. Three of the loci were significant in a meta-analysis of smaller, independent replication samples. Although only a few loci passed the stringent statistical corrections for genome-wide multiple testing and were replicated in other samples, our analyses show that many loci underlie same-sex sexual behavior in both sexes. In aggregate, all tested genetic variants accounted for 8 to 25\% of variation in male and female same-sex sexual behavior, and the genetic influences were positively but imperfectly correlated between the sexes [genetic correlation coefficient (rg)= 0.63; 95\% confidence intervals, 0.48 to 0.78]. These aggregate genetic influences partly overlapped with those on a variety of other traits, including externalizing behaviors such as smoking, cannabis use, risk-taking, and the personality trait “openness to experience.” Additional analyses suggested that sexual behavior, attraction, identity, and fantasies are influenced by a similar set of genetic variants (rg {\textgreater} 0.83); however, the genetic effects that differentiate heterosexual from same-sex sexual behavior are not the same as those that differ among nonheterosexuals with lower versus higher proportions of same-sex partners, which suggests that there is no single continuum from opposite-sex to same-sex preference.
CONCLUSIONSame-sex sexual behavior is influenced by not one or a few genes but many. Overlap with genetic influences on other traits provides insights into the underlying biology of same-sex sexual behavior, and analysis of different aspects of sexual preference underscore its complexity and call into question the validity of bipolar continuum measures such as the Kinsey scale. Nevertheless, many uncertainties remain to be explored, including how sociocultural influences on sexual preference might interact with genetic influences. To help communicate our study to the broader public, we organized workshops in which representatives of the public, activists, and researchers discussed the rationale, results, and implications of our study. {\textless}img class="fragment-image" aria-describedby="F1-caption" src="https://science.sciencemag.org/content/sci/365/6456/eaat7693/F1.medium.gif"/{\textgreater} Download high-res image Open in new tab Download Powerpoint A genome-wide association study (GWAS) of same-sex sexual behavior reveals five loci and high polygenicity.Follow-up analyses show potential biological pathways; show genetic correlations with various traits; and indicate that sexual preference is a complex, heterogeneous phenotype.
Twin and family studies have shown that same-sex sexual behavior is partly genetically influenced, but previous searches for specific genes involved have been underpowered. We performed a genome-wide association study (GWAS) on 477,522 individuals, revealing five loci significantly associated with same-sex sexual behavior. In aggregate, all tested genetic variants accounted for 8 to 25\% of variation in same-sex sexual behavior, only partially overlapped between males and females, and do not allow meaningful prediction of an individual’s sexual behavior. Comparing these GWAS results with those for the proportion of same-sex to total number of sexual partners among nonheterosexuals suggests that there is no single continuum from opposite-sex to same-sex sexual behavior. Overall, our findings provide insights into the genetics underlying same-sex sexual behavior and underscore the complexity of sexuality.
A genome-wide association study of same-sex sexual behavior identifies loci associated with human sexual orientation.
A genome-wide association study of same-sex sexual behavior identifies loci associated with human sexual orientation.},
	language = {en},
	number = {6456},
	urldate = {2020-05-18},
	journal = {Science},
	author = {Ganna, Andrea and Verweij, Karin J. H. and Nivard, Michel G. and Maier, Robert and Wedow, Robbee and Busch, Alexander S. and Abdellaoui, Abdel and Guo, Shengru and Sathirapongsasuti, J. Fah and Team16, 23andMe Research and Lichtenstein, Paul and Lundström, Sebastian and Långström, Niklas and Auton, Adam and Harris, Kathleen Mullan and Beecham, Gary W. and Martin, Eden R. and Sanders, Alan R. and Perry, John R. B. and Neale, Benjamin M. and Zietsch, Brendan P.},
	month = aug,
	year = {2019},
	pmid = {31467194},
	note = {Publisher: American Association for the Advancement of Science
Section: Research Article},
}

@article{reich_opinion_2018,
	chapter = {Opinion},
	title = {Opinion {\textbar} {How} {Genetics} {Is} {Changing} {Our} {Understanding} of ‘{Race}’},
	issn = {0362-4331},
	url = {https://www.nytimes.com/2018/03/23/opinion/sunday/genetics-race.html},
	abstract = {If scientists avoid discussing the topic candidly, racist theories will fill the vacuum.},
	language = {en-US},
	urldate = {2020-05-18},
	journal = {The New York Times},
	author = {Reich, David},
	month = mar,
	year = {2018},
	keywords = {Genetics and Heredity, Race and Ethnicity},
}

@article{wojcik_genetic_2019,
	title = {Genetic analyses of diverse populations improves discovery for complex traits},
	volume = {570},
	copyright = {2019 The Author(s), under exclusive licence to Springer Nature Limited},
	issn = {1476-4687},
	url = {https://www.nature.com/articles/s41586-019-1310-4},
	doi = {10.1038/s41586-019-1310-4},
	abstract = {Genome-wide association studies (GWAS) have laid the foundation for investigations into the biology of complex traits, drug development and clinical guidelines. However, the majority of discovery efforts are based on data from populations of European ancestry1–3. In light of the differential genetic architecture that is known to exist between populations, bias in representation can exacerbate existing disease and healthcare disparities. Critical variants may be missed if they have a low frequency or are completely absent in European populations, especially as the field shifts its attention towards rare variants, which are more likely to be population-specific4–10. Additionally, effect sizes and their derived risk prediction scores derived in one population may not accurately extrapolate to other populations11,12. Here we demonstrate the value of diverse, multi-ethnic participants in large-scale genomic studies. The Population Architecture using Genomics and Epidemiology (PAGE) study conducted a GWAS of 26 clinical and behavioural phenotypes in 49,839 non-European individuals. Using strategies tailored for analysis of multi-ethnic and admixed populations, we describe a framework for analysing diverse populations, identify 27 novel loci and 38 secondary signals at known loci, as well as replicate 1,444 GWAS catalogue associations across these traits. Our data show evidence of effect-size heterogeneity across ancestries for published GWAS associations, substantial benefits for fine-mapping using diverse cohorts and insights into clinical implications. In the United States—where minority populations have a disproportionately higher burden of chronic conditions13—the lack of representation of diverse populations in genetic research will result in inequitable access to precision medicine for those with the highest burden of disease. We strongly advocate for continued, large genome-wide efforts in diverse populations to maximize genetic discovery and reduce health disparities.},
	language = {en},
	number = {7762},
	urldate = {2020-05-18},
	journal = {Nature},
	author = {Wojcik, Genevieve L. and Graff, Mariaelisa and Nishimura, Katherine K. and Tao, Ran and Haessler, Jeffrey and Gignoux, Christopher R. and Highland, Heather M. and Patel, Yesha M. and Sorokin, Elena P. and Avery, Christy L. and Belbin, Gillian M. and Bien, Stephanie A. and Cheng, Iona and Cullina, Sinead and Hodonsky, Chani J. and Hu, Yao and Huckins, Laura M. and Jeff, Janina and Justice, Anne E. and Kocarnik, Jonathan M. and Lim, Unhee and Lin, Bridget M. and Lu, Yingchang and Nelson, Sarah C. and Park, Sung-Shim L. and Poisner, Hannah and Preuss, Michael H. and Richard, Melissa A. and Schurmann, Claudia and Setiawan, Veronica W. and Sockell, Alexandra and Vahi, Karan and Verbanck, Marie and Vishnu, Abhishek and Walker, Ryan W. and Young, Kristin L. and Zubair, Niha and Acuña-Alonso, Victor and Ambite, Jose Luis and Barnes, Kathleen C. and Boerwinkle, Eric and Bottinger, Erwin P. and Bustamante, Carlos D. and Caberto, Christian and Canizales-Quinteros, Samuel and Conomos, Matthew P. and Deelman, Ewa and Do, Ron and Doheny, Kimberly and Fernández-Rhodes, Lindsay and Fornage, Myriam and Hailu, Benyam and Heiss, Gerardo and Henn, Brenna M. and Hindorff, Lucia A. and Jackson, Rebecca D. and Laurie, Cecelia A. and Laurie, Cathy C. and Li, Yuqing and Lin, Dan-Yu and Moreno-Estrada, Andres and Nadkarni, Girish and Norman, Paul J. and Pooler, Loreall C. and Reiner, Alexander P. and Romm, Jane and Sabatti, Chiara and Sandoval, Karla and Sheng, Xin and Stahl, Eli A. and Stram, Daniel O. and Thornton, Timothy A. and Wassel, Christina L. and Wilkens, Lynne R. and Winkler, Cheryl A. and Yoneyama, Sachi and Buyske, Steven and Haiman, Christopher A. and Kooperberg, Charles and Le Marchand, Loic and Loos, Ruth J. F. and Matise, Tara C. and North, Kari E. and Peters, Ulrike and Kenny, Eimear E. and Carlson, Christopher S.},
	month = jun,
	year = {2019},
	note = {Number: 7762
Publisher: Nature Publishing Group},
	pages = {514--518},
}

@article{bycroft_uk_2018,
	title = {The {UK} {Biobank} resource with deep phenotyping and genomic data},
	volume = {562},
	copyright = {2018 Springer Nature Limited},
	issn = {1476-4687},
	url = {https://www.nature.com/articles/s41586-018-0579-z},
	doi = {10.1038/s41586-018-0579-z},
	abstract = {The UK Biobank project is a prospective cohort study with deep genetic and phenotypic data collected on approximately 500,000 individuals from across the United Kingdom, aged between 40 and 69 at recruitment. The open resource is unique in its size and scope. A rich variety of phenotypic and health-related information is available on each participant, including biological measurements, lifestyle indicators, biomarkers in blood and urine, and imaging of the body and brain. Follow-up information is provided by linking health and medical records. Genome-wide genotype data have been collected on all participants, providing many opportunities for the discovery of new genetic associations and the genetic bases of complex traits. Here we describe the centralized analysis of the genetic data, including genotype quality, properties of population structure and relatedness of the genetic data, and efficient phasing and genotype imputation that increases the number of testable variants to around 96 million. Classical allelic variation at 11 human leukocyte antigen genes was imputed, resulting in the recovery of signals with known associations between human leukocyte antigen alleles and many diseases.},
	language = {en},
	number = {7726},
	urldate = {2020-05-18},
	journal = {Nature},
	author = {Bycroft, Clare and Freeman, Colin and Petkova, Desislava and Band, Gavin and Elliott, Lloyd T. and Sharp, Kevin and Motyer, Allan and Vukcevic, Damjan and Delaneau, Olivier and O’Connell, Jared and Cortes, Adrian and Welsh, Samantha and Young, Alan and Effingham, Mark and McVean, Gil and Leslie, Stephen and Allen, Naomi and Donnelly, Peter and Marchini, Jonathan},
	month = oct,
	year = {2018},
	note = {Number: 7726
Publisher: Nature Publishing Group},
	pages = {203--209},
}

@article{sigwart_coalescent_2009-1,
	title = {Coalescent {Theory}: {An} {Introduction}},
	volume = {58},
	issn = {1063-5157},
	shorttitle = {Coalescent {Theory}},
	url = {https://academic.oup.com/sysbio/article/58/1/162/1673216},
	doi = {10.1093/schbul/syp004},
	abstract = {“The Coalescent” is a powerful extension of classical population genetics because it is a collection of mathematical models that can accommodate biological phen},
	language = {en},
	number = {1},
	urldate = {2020-05-18},
	journal = {Systematic Biology},
	author = {Sigwart, Julia},
	month = feb,
	year = {2009},
	note = {Publisher: Oxford Academic},
	pages = {162--165},
}

@misc{noauthor_alianza_nodate,
	title = {Alianza {SIDALC}},
	url = {http://www.sidalc.net/cgi-bin/wxis.exe/?IsisScript=FCL.xis&method=post&formato=2&cantidad=1&expresion=mfn=010195},
	urldate = {2020-05-18},
}

@misc{noauthor_coalescent_nodate,
	title = {The coalescent - {ScienceDirect}},
	url = {https://www.sciencedirect.com/science/article/pii/0304414982900114},
	urldate = {2020-05-18},
}

@article{petkova_visualizing_2016,
	title = {Visualizing spatial population structure with estimated effective migration surfaces},
	volume = {48},
	issn = {1061-4036},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4696895/},
	doi = {10.1038/ng.3464},
	abstract = {Genetic data often exhibit patterns broadly consistent with “isolation by distance” – a phenomenon where genetic similarity decays with geographic distance. In a heterogeneous habitat this may occur more quickly in some regions than others: for example, barriers to gene flow can accelerate differentiation between neighboring groups. We use the concept of “effective migration” to model the relationship between genetics and geography: in this paradigm, effective migration is low in regions where genetic similarity decays quickly. We present a method to visualize variation in effective migration across the habitat from geographically indexed genetic data. Our approach uses a population genetic model to relate effective migration rates to expected genetic dissimilarities. We illustrate its potential and limitations using simulations and data from elephant, human and A. thaliana populations. The resulting visualizations highlight important spatial features of population structure that are difficult to discern using existing methods for summarizing genetic variation.},
	number = {1},
	urldate = {2020-05-18},
	journal = {Nature genetics},
	author = {Petkova, Desislava and Novembre, John and Stephens, Matthew},
	month = jan,
	year = {2016},
	pmid = {26642242},
	pmcid = {PMC4696895},
	pages = {94--100},
}

@article{noauthor_global_2015,
	title = {A global reference for human genetic variation},
	volume = {526},
	issn = {0028-0836},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4750478/},
	doi = {10.1038/nature15393},
	abstract = {The 1000 Genomes Project set out to provide a comprehensive description of common human genetic variation by applying whole-genome sequencing to a diverse set of individuals from multiple populations. Here we report completion of the project, having reconstructed the genomes of 2,504 individuals from 26 populations using a combination of low-coverage whole-genome sequencing, deep exome sequencing, and dense microarray genotyping. We characterized a broad spectrum of genetic variation, in total over 88 million variants (84.7 million single nucleotide polymorphisms (SNPs), 3.6 million short insertions/deletions (indels), and 60,000 structural variants), all phased onto high-quality haplotypes. This resource includes {\textgreater}99\% of SNP variants with a frequency of {\textgreater}1\% for a variety of ancestries. We describe the distribution of genetic variation across the global sample, and discuss the implications for common disease studies.},
	number = {7571},
	urldate = {2020-05-18},
	journal = {Nature},
	month = oct,
	year = {2015},
	pmid = {26432245},
	pmcid = {PMC4750478},
	pages = {68--74},
}

@misc{noauthor_comparison_nodate,
	title = {Comparison of 79 {DNA} polymorphisms tested in {Australians}, {Japanese} and {Papua} {New} {Guineans} with those of five other human populations. - {Abstract} - {Europe} {PMC}},
	url = {https://europepmc.org/article/med/7662611},
	urldate = {2020-05-18},
}

@book{ginsburgh_palgrave_2016,
	title = {The {Palgrave} {Handbook} of {Economics} and {Language}},
	isbn = {978-1-137-32505-1},
	abstract = {In this handbook, Victor Ginsburgh and Shlomo Weber bring together methodological, theoretical, and empirical studies in the economics of language in a single framework of linguistic diversity that reflects the history and contemporary study of the topic. The impact of linguistic diversity on economic outcomes and public policies has been studied not only by economists and other social scientists in the contemporary era, but all the way back to the 19th century by geographer and naturalist, Alexander von Humboldt, who emphasized the importance of language in the framework of cultural experience. This interdependence of language and culture is reflected in the chapters in this handbook, which have been written by leading economists, linguists, and political scientists from universities in the United States, Australia, Russia, Israel, the United Kingdom, France, Spain, Belgium, Germany, Switzerland, Denmark, Finland, Hungary, and the Czech Republic. The contributions are divided into four parts. Part I examines linguistic concepts that forge common ground between economists, political scientists, sociologists, and linguists, and introduces the notion of linguistic proximity extensively utilized in various chapters of the volume. Part II assesses the impact of languages on market interactions, including international trade, patent protection, migration, and use of languages in ancient and modern business environments. Part III focuses on the link between linguistic policies and economic development, including the analysis of regional development in Asia, Africa, Europe and Russia. Part IV addresses issues of globalization, minority languages, and the protection of linguistic rights in multilingual societies.},
	language = {en},
	publisher = {Springer},
	author = {Ginsburgh, V. and Weber, S.},
	month = apr,
	year = {2016},
	note = {Google-Books-ID: ya3tCwAAQBAJ},
	keywords = {Business \& Economics / Economics / Theory, Language Arts \& Disciplines / Linguistics / General, Language Arts \& Disciplines / Linguistics / Historical \& Comparative, Language Arts \& Disciplines / Linguistics / Sociolinguistics},
}

@misc{noauthor_sewall_nodate,
	title = {Sewall {Wright} and {Gustave} {Malécot} on {Isolation} by {Distance} {\textbar} {Philosophy} of {Science}: {Vol} 76, {No} 5},
	url = {https://www.journals.uchicago.edu/doi/10.1086/605802},
	urldate = {2020-05-18},
}

@article{rohlf_investigation_1971,
	title = {An {Investigation} of the {Isolation}-{By}-{Distance} {Model}},
	volume = {105},
	issn = {0003-0147},
	url = {https://www.journals.uchicago.edu/doi/10.1086/282727},
	doi = {10.1086/282727},
	abstract = {Wright's isolation-by-distance model was investigated using techniques of simulation on a digital computer. We examined the manner in which a population as a whole differentiated in time, as well as the distributional pattern of gene frequencies in the population. Both the areal and the linear isolation-by-distance models were investigated. The observed rate of increase in the inbreeding coefficient did not agree well with that predicted by the results of Wright (although the differences may be due to changes which take place in only the first few generations). It was also found that the particular patterns of highs and lows of gene frequencies over a geographic area became established quickly and persisted for a large number of generations, particularly near the periphery of the population. Implications of our results for interpreting geographic variation analyses in terms of differential selection are discussed.},
	number = {944},
	urldate = {2020-05-18},
	journal = {The American Naturalist},
	author = {Rohlf, F. James and Schnell, Gary D.},
	month = jul,
	year = {1971},
	note = {Publisher: The University of Chicago Press},
	pages = {295--324},
}

@article{slatkin_isolation_1993,
	title = {Isolation by {Distance} in {Equilibrium} and {Non}-{Equilibrium} {Populations}},
	volume = {47},
	issn = {0014-3820},
	url = {https://www.jstor.org/stable/2410134},
	doi = {10.2307/2410134},
	abstract = {It is shown that for allele frequency data a useful measure of the extent of gene flow between a pair of populations is M̂ = (1/FST - 1)/4, which is the estimated level of gene flow in an island model at equilibrium. For DNA sequence data, the same formula can be used if FST is replaced by NST. In a population with restricted dispersal, analytic theory shows that there is a simple relationship between M and geographic distance in both equilibrium and non-equilibrium populations and that this relationship is approximately independent of mutation rate when the mutation rate is small. Simulation results show that with reasonable sample sizes, isolation by distance can indeed be detected and that, at least in some cases, non-equilibrium patterns can be distinguished. This approach to analyzing isolation by distance is used for two allozyme data sets, one from gulls and one from pocket gophers.},
	number = {1},
	urldate = {2020-05-18},
	journal = {Evolution},
	author = {Slatkin, Montgomery},
	year = {1993},
	note = {Publisher: [Society for the Study of Evolution, Wiley]},
	pages = {264--279},
}

@article{rice_distinguishing_2018,
	title = {Distinguishing multiple-merger from {Kingman} coalescence using two-site frequency spectra},
	copyright = {© 2018, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NoDerivs 4.0 International), CC BY-ND 4.0, as described at http://creativecommons.org/licenses/by-nd/4.0/},
	url = {https://www.biorxiv.org/content/10.1101/461517v1},
	doi = {10.1101/461517},
	abstract = {{\textless}h3{\textgreater}Abstract{\textless}/h3{\textgreater} {\textless}p{\textgreater}Demographic inference methods in population genetics typically assume that the ancestry of a sample can be modeled by the Kingman coalescent. A defining feature of this stochastic process is that it generates genealogies that are binary trees: no more than two ancestral lineages may coalesce at the same time. However, this assumption breaks down under several scenarios. For example, pervasive natural selection and extreme variation in offspring number can both generate genealogies with “multiple-merger” events in which more than two lineages coalesce instantaneously. Therefore, detecting multiple mergers is important both for understanding which forces have shaped the diversity of a population and for avoiding fitting misspecified models to data. Current methods to detect multiple mergers in genomic data rely on the site frequency spectrum (SFS). However, the signatures of multiple mergers in the SFS are also consistent with a Kingman coalescent with a time-varying population size. Here, we present a new method for detecting multiple mergers based on the pointwise mutual information of the two-site frequency spectrum for pairs of linked sites. Unlike the SFS, the pointwise mutual information depends mostly on the topologies of genealogies rather than on their branch lengths and is therefore largely insensitive to population size change. This statistic is global in the sense that it can detect when the genome-wide genetic diversity is inconsistent with the Kingman coalescent, rather than detecting outlier regions, as in selection scan methods. Finally, we demonstrate a graphical model-checking procedure based on the point-wise mutual information using genomic diversity data from \textit{Drosophila melanogaster}.{\textless}/p{\textgreater}},
	language = {en},
	urldate = {2020-05-18},
	journal = {bioRxiv},
	author = {Rice, Daniel P. and Novembre, John and Desai, Michael M.},
	month = nov,
	year = {2018},
	note = {Publisher: Cold Spring Harbor Laboratory
Section: New Results},
	pages = {461517},
}

@article{mcdonald_sex_2016,
	title = {Sex speeds adaptation by altering the dynamics of molecular evolution},
	volume = {531},
	copyright = {2016 Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.},
	issn = {1476-4687},
	url = {https://www.nature.com/articles/nature17143},
	doi = {10.1038/nature17143},
	abstract = {In a comparison between replicate sexual and asexual populations of Saccharomyces cerevisiae, sexual reproduction increases fitness by reducing clonal interference and alters the type of mutations that get fixed by natural selection.},
	language = {en},
	number = {7593},
	urldate = {2020-05-18},
	journal = {Nature},
	author = {McDonald, Michael J. and Rice, Daniel P. and Desai, Michael M.},
	month = mar,
	year = {2016},
	note = {Number: 7593
Publisher: Nature Publishing Group},
	pages = {233--236},
}

@misc{noauthor_distinguishing_nodate,
	title = {Distinguishing multiple-merger from {Kingman} coalescence using two-site frequency spectra {\textbar} {bioRxiv}},
	url = {https://www.biorxiv.org/content/10.1101/461517v1.abstract},
	urldate = {2020-05-18},
}

@techreport{rice_distinguishing_2018-1,
	type = {preprint},
	title = {Distinguishing multiple-merger from {Kingman} coalescence using two-site frequency spectra},
	url = {http://biorxiv.org/lookup/doi/10.1101/461517},
	abstract = {Abstract
          
            Demographic inference methods in population genetics typically assume that the ancestry of a sample can be modeled by the Kingman coalescent. A defining feature of this stochastic process is that it generates genealogies that are binary trees: no more than two ancestral lineages may coalesce at the same time. However, this assumption breaks down under several scenarios. For example, pervasive natural selection and extreme variation in offspring number can both generate genealogies with “multiple-merger” events in which more than two lineages coalesce instantaneously. Therefore, detecting multiple mergers is important both for understanding which forces have shaped the diversity of a population and for avoiding fitting misspecified models to data. Current methods to detect multiple mergers in genomic data rely on the site frequency spectrum (SFS). However, the signatures of multiple mergers in the SFS are also consistent with a Kingman coalescent with a time-varying population size. Here, we present a new method for detecting multiple mergers based on the pointwise mutual information of the two-site frequency spectrum for pairs of linked sites. Unlike the SFS, the pointwise mutual information depends mostly on the topologies of genealogies rather than on their branch lengths and is therefore largely insensitive to population size change. This statistic is global in the sense that it can detect when the genome-wide genetic diversity is inconsistent with the Kingman coalescent, rather than detecting outlier regions, as in selection scan methods. Finally, we demonstrate a graphical model-checking procedure based on the point-wise mutual information using genomic diversity data from
            Drosophila melanogaster
            .},
	language = {en},
	urldate = {2020-05-18},
	institution = {Evolutionary Biology},
	author = {Rice, Daniel P. and Novembre, John and Desai, Michael M.},
	month = nov,
	year = {2018},
	doi = {10.1101/461517},
}

@techreport{rice_distinguishing_2018-2,
	type = {preprint},
	title = {Distinguishing multiple-merger from {Kingman} coalescence using two-site frequency spectra},
	url = {http://biorxiv.org/lookup/doi/10.1101/461517},
	abstract = {Abstract
          
            Demographic inference methods in population genetics typically assume that the ancestry of a sample can be modeled by the Kingman coalescent. A defining feature of this stochastic process is that it generates genealogies that are binary trees: no more than two ancestral lineages may coalesce at the same time. However, this assumption breaks down under several scenarios. For example, pervasive natural selection and extreme variation in offspring number can both generate genealogies with “multiple-merger” events in which more than two lineages coalesce instantaneously. Therefore, detecting multiple mergers is important both for understanding which forces have shaped the diversity of a population and for avoiding fitting misspecified models to data. Current methods to detect multiple mergers in genomic data rely on the site frequency spectrum (SFS). However, the signatures of multiple mergers in the SFS are also consistent with a Kingman coalescent with a time-varying population size. Here, we present a new method for detecting multiple mergers based on the pointwise mutual information of the two-site frequency spectrum for pairs of linked sites. Unlike the SFS, the pointwise mutual information depends mostly on the topologies of genealogies rather than on their branch lengths and is therefore largely insensitive to population size change. This statistic is global in the sense that it can detect when the genome-wide genetic diversity is inconsistent with the Kingman coalescent, rather than detecting outlier regions, as in selection scan methods. Finally, we demonstrate a graphical model-checking procedure based on the point-wise mutual information using genomic diversity data from
            Drosophila melanogaster
            .},
	language = {en},
	urldate = {2020-05-18},
	institution = {Evolutionary Biology},
	author = {Rice, Daniel P. and Novembre, John and Desai, Michael M.},
	month = nov,
	year = {2018},
	doi = {10.1101/461517},
}

@book{gillespie_population_2004,
	title = {Population {Genetics}: {A} {Concise} {Guide}},
	isbn = {978-1-4214-0170-6},
	shorttitle = {Population {Genetics}},
	abstract = {This concise introduction offers students and researchers an overview of the discipline that connects genetics and evolution. Addressing the theories behind population genetics and relevant empirical evidence, John Gillespie discusses genetic drift, natural selection, nonrandom mating, quantitative genetics, and the evolutionary advantage of sex. First published to wide acclaim in 1998, this brilliant primer has been updated to include new sections on molecular evolution, genetic drift, genetic load, the stationary distribution, and two-locus dynamics. This book is indispensable for students working in a laboratory setting or studying free-ranging populations.},
	language = {en},
	publisher = {JHU Press},
	author = {Gillespie, John H.},
	month = aug,
	year = {2004},
	note = {Google-Books-ID: KAcAfiyHpcoC},
	keywords = {Medical / Genetics, Science / Life Sciences / Biology, Science / Life Sciences / Ecology},
}

@book{fisher_genetical_nodate,
	title = {The genetical theory of natural selection},
	isbn = {978-1-176-62502-0},
	language = {en},
	publisher = {Рипол Классик},
	author = {Fisher, R. A.},
	note = {Google-Books-ID: WPfvAgAAQBAJ},
	keywords = {History / General},
}

@article{gillham_battle_2015,
	title = {The {Battle} {Between} the {Biometricians} and the {Mendelians}: {How} {Sir} {Francis} {Galton}’s {Work} {Caused} his {Disciples} to {Reach} {Conflicting} {Conclusions} {About} the {Hereditary} {Mechanism}},
	volume = {24},
	issn = {1573-1901},
	shorttitle = {The {Battle} {Between} the {Biometricians} and the {Mendelians}},
	url = {https://doi.org/10.1007/s11191-013-9642-1},
	doi = {10.1007/s11191-013-9642-1},
	abstract = {Francis Galton, Charles Darwin’s cousin, had wide and varied interests. They ranged from exploration and travel writing to fingerprinting and the weather. After reading Darwin’s On the Origin of Species, Galton reached the conclusion that it should be possible to improve the human stock through selective breeding, as was the case for domestic animals and cultivated plants. Much of the latter half of Galton’s career was devoted to trying to devise methods to distinguish men of good stock and then to show that these qualities were inherited. But along the way he invented two important statistical methods: regression and correlation. He also discovered regression to the mean. This led Galton to believe that evolution could not proceed by the small steps envisioned by Darwin, but must proceed by discontinuous changes. Galton’s book Natural Inheritance (1889) served as the inspiration for Karl Pearson, W.F.R. Weldon and William Bateson. Pearson and Weldon were interested in continuously varying characters and the application of statistical techniques to their study. Bateson was fascinated by discontinuities and the role they might play in evolution. Galton proposed his Law of Ancestral Heredity in the last decade of the nineteenth century. At first this seemed to work well as an explanation for continuously varying traits of the type that interested Pearson and Weldon. In contrast, Bateson had published a book on discontinuously varying traits so he was in a position to understand and embrace Mendel’s principles of inheritance when they were rediscovered in 1900. The subsequent battle between Weldon and Pearson, the biometricians, and Bateson, the Mendelian, went on acrimoniously for several years at the beginning of the twentieth century before Mendelian theory finally won out.},
	language = {en},
	number = {1},
	urldate = {2020-05-18},
	journal = {Science \& Education},
	author = {Gillham, Nicholas W.},
	month = jan,
	year = {2015},
	pages = {61--75},
}

@article{bateson_reports_1902,
	title = {Reports to the {Evolution} {Committee} of the {Royal} {Society},},
	language = {en},
	author = {Bateson, William and Saunders, E R},
	year = {1902},
	pages = {41},
}

@article{young_solving_2019,
	title = {Solving the missing heritability problem},
	volume = {15},
	issn = {1553-7404},
	url = {https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008222},
	doi = {10.1371/journal.pgen.1008222},
	language = {en},
	number = {6},
	urldate = {2020-05-17},
	journal = {PLOS Genetics},
	author = {Young, Alexander I.},
	month = jun,
	year = {2019},
	note = {Publisher: Public Library of Science},
	keywords = {Genetic polymorphism, Genome-wide association studies, Genotyping, Heredity, Population genetics, Principal component analysis, Twin studies, Twins},
	pages = {e1008222},
}

@misc{noauthor_finding_nodate,
	title = {Finding the missing heritability of complex diseases {\textbar} {Nature}},
	url = {https://www.nature.com/articles/nature08494},
	urldate = {2020-05-17},
}

@article{maher_personal_2008,
	title = {Personal genomes: {The} case of the missing heritability},
	volume = {456},
	issn = {0028-0836, 1476-4687},
	shorttitle = {Personal genomes},
	url = {http://www.nature.com/articles/456018a},
	doi = {10.1038/456018a},
	language = {en},
	number = {7218},
	urldate = {2020-05-17},
	journal = {Nature},
	author = {Maher, Brendan},
	month = nov,
	year = {2008},
	pages = {18--21},
}

@article{kumar_mutation_2002,
	title = {Mutation rates in mammalian genomes},
	volume = {99},
	copyright = {Copyright © 2002, The National Academy of Sciences},
	issn = {0027-8424, 1091-6490},
	url = {https://www.pnas.org/content/99/2/803},
	doi = {10.1073/pnas.022629899},
	abstract = {Knowledge of the rate of point mutation is of fundamental importance, because mutations are a vital source of genetic novelty and a significant cause of human diseases. Currently, mutation rate is thought to vary many fold among genes within a genome and among lineages in mammals. We have conducted a computational analysis of 5,669 genes (17,208 sequences) from species representing major groups of placental mammals to characterize the extent of mutation rate differences among genes in a genome and among diverse mammalian lineages. We find that mutation rate is approximately constant per year and largely similar among genes. Similarity of mutation rates among lineages with vastly different generation lengths and physiological attributes points to a much greater contribution of replication-independent mutational processes to the overall mutation rate. Our results suggest that the average mammalian genome mutation rate is 2.2 × 10−9 per base pair per year, which provides further opportunities for estimating species and population divergence times by using molecular clocks.},
	language = {en},
	number = {2},
	urldate = {2020-05-17},
	journal = {Proceedings of the National Academy of Sciences},
	author = {Kumar, Sudhir and Subramanian, Sankar},
	month = jan,
	year = {2002},
	pmid = {11792858},
	note = {Publisher: National Academy of Sciences
Section: Biological Sciences},
	keywords = {neutral evolution‖substitution pattern‖disparity index‖generation length‖molecular clock},
	pages = {803--808},
}

@article{scally_mutation_2016,
	series = {Genetics of human origin},
	title = {The mutation rate in human evolution and demographic inference},
	volume = {41},
	issn = {0959-437X},
	url = {http://www.sciencedirect.com/science/article/pii/S0959437X16301010},
	doi = {10.1016/j.gde.2016.07.008},
	abstract = {The germline mutation rate has long been a major source of uncertainty in human evolutionary and demographic analyses based on genetic data, but estimates have improved substantially in recent years. I discuss our current knowledge of the mutation rate in humans and the underlying biological factors affecting it, which include generation time, parental age and other developmental and reproductive timescales. There is good evidence for a slowdown in mean mutation rate during great ape evolution, but not for a more recent change within the timescale of human genetic diversity. Hence, pending evidence to the contrary, it is reasonable to use a present-day rate of approximately 0.5×10−9bp−1year−1 in all human or hominin demographic analyses.},
	language = {en},
	urldate = {2020-05-17},
	journal = {Current Opinion in Genetics \& Development},
	author = {Scally, Aylwyn},
	month = dec,
	year = {2016},
	pages = {36--43},
}

@article{desai_beneficial_2007,
	title = {Beneficial {Mutation}–{Selection} {Balance} and the {Effect} of {Linkage} on {Positive} {Selection}},
	volume = {176},
	copyright = {Copyright © 2007 by the Genetics Society of America},
	issn = {0016-6731, 1943-2631},
	url = {https://www.genetics.org/content/176/3/1759},
	doi = {10.1534/genetics.106.067678},
	abstract = {When beneficial mutations are rare, they accumulate by a series of selective sweeps. But when they are common, many beneficial mutations will occur before any can fix, so there will be many different mutant lineages in the population concurrently. In an asexual population, these different mutant lineages interfere and not all can fix simultaneously. In addition, further beneficial mutations can accumulate in mutant lineages while these are still a minority of the population. In this article, we analyze the dynamics of such multiple mutations and the interplay between multiple mutations and interference between clones. These result in substantial variation in fitness accumulating within a single asexual population. The amount of variation is determined by a balance between selection, which destroys variation, and beneficial mutations, which create more. The behavior depends in a subtle way on the population parameters: the population size, the beneficial mutation rate, and the distribution of the fitness increments of the potential beneficial mutations. The mutation–selection balance leads to a continually evolving population with a steady-state fitness variation. This variation increases logarithmically with both population size and mutation rate and sets the rate at which the population accumulates beneficial mutations, which thus also grows only logarithmically with population size and mutation rate. These results imply that mutator phenotypes are less effective in larger asexual populations. They also have consequences for the advantages (or disadvantages) of sex via the Fisher–Muller effect; these are discussed briefly.},
	language = {en},
	number = {3},
	urldate = {2020-05-17},
	journal = {Genetics},
	author = {Desai, Michael M. and Fisher, Daniel S.},
	month = jul,
	year = {2007},
	pmid = {17483432},
	note = {Publisher: Genetics
Section: Investigations},
	pages = {1759--1798},
}

@article{koster_snakemakescalable_2012,
	title = {Snakemake—a scalable bioinformatics workflow engine},
	volume = {28},
	issn = {1367-4803},
	url = {https://academic.oup.com/bioinformatics/article/28/19/2520/290322},
	doi = {10.1093/bioinformatics/bts480},
	abstract = {Abstract.  Summary: Snakemake is a workflow engine that provides a readable Python-based workflow definition language and a powerful execution environment that},
	language = {en},
	number = {19},
	urldate = {2020-05-16},
	journal = {Bioinformatics},
	author = {Köster, Johannes and Rahmann, Sven},
	month = oct,
	year = {2012},
	note = {Publisher: Oxford Academic},
	pages = {2520--2522},
}

@article{van_der_walt_numpy_2011,
	title = {The {NumPy} {Array}: {A} {Structure} for {Efficient} {Numerical} {Computation}},
	volume = {13},
	issn = {1558-366X},
	shorttitle = {The {NumPy} {Array}},
	doi = {10.1109/MCSE.2011.37},
	abstract = {In the Python world, NumPy arrays are the standard representation for numerical data and enable efficient implementation of numerical computations in a high-level language. As this effort shows, NumPy performance can be improved through three techniques: vectorizing calculations, avoiding copying data in memory, and minimizing operation counts.},
	number = {2},
	journal = {Computing in Science Engineering},
	author = {van der Walt, Stefan and Colbert, S. Chris and Varoquaux, Gael},
	month = mar,
	year = {2011},
	note = {Conference Name: Computing in Science Engineering},
	keywords = {Arrays, Computational efficiency, Finite element methods, NumPy, Numerical analysis, Performance evaluation, Python, Python programming language, Resource management, Vector quantization, data structures, high level language, high level languages, mathematics computing, numerical analysis, numerical computation, numerical computations, numerical data, numpy array, programming libraries, scientific programming},
	pages = {22--30},
}

@article{caballero_nature_2015,
	title = {The {Nature} of {Genetic} {Variation} for {Complex} {Traits} {Revealed} by {GWAS} and {Regional} {Heritability} {Mapping} {Analyses}},
	volume = {201},
	copyright = {Copyright © 2015 by the Genetics Society of America},
	issn = {0016-6731, 1943-2631},
	url = {https://www.genetics.org/content/201/4/1601},
	doi = {10.1534/genetics.115.177220},
	abstract = {We use computer simulations to investigate the amount of genetic variation for complex traits that can be revealed by single-SNP genome-wide association studies (GWAS) or regional heritability mapping (RHM) analyses based on full genome sequence data or SNP chips. We model a large population subject to mutation, recombination, selection, and drift, assuming a pleiotropic model of mutations sampled from a bivariate distribution of effects of mutations on a quantitative trait and fitness. The pleiotropic model investigated, in contrast to previous models, implies that common mutations of large effect are responsible for most of the genetic variation for quantitative traits, except when the trait is fitness itself. We show that GWAS applied to the full sequence increases the number of QTL detected by as much as 50\% compared to the number found with SNP chips but only modestly increases the amount of additive genetic variance explained. Even with full sequence data, the total amount of additive variance explained is generally below 50\%. Using RHM on the full sequence data, a slightly larger number of QTL are detected than by GWAS if the same probability threshold is assumed, but these QTL explain a slightly smaller amount of genetic variance. Our results also suggest that most of the missing heritability is due to the inability to detect variants of moderate effect (∼0.03–0.3 phenotypic SDs) segregating at substantial frequencies. Very rare variants, which are more difficult to detect by GWAS, are expected to contribute little genetic variation, so their eventual detection is less relevant for resolving the missing heritability problem.},
	language = {en},
	number = {4},
	urldate = {2020-05-15},
	journal = {Genetics},
	author = {Caballero, Armando and Tenesa, Albert and Keightley, Peter D.},
	month = dec,
	year = {2015},
	pmid = {26482794},
	note = {Publisher: Genetics
Section: Investigations},
	keywords = {additive genetic variance, complex traits, fitness, missing heritability, quantitative trait variation},
	pages = {1601--1613},
}

@article{sikora_ancient_2017,
	title = {Ancient genomes show social and reproductive behavior of early {Upper} {Paleolithic} foragers},
	volume = {358},
	copyright = {Copyright © 2017 The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. No claim to original U.S. Government Works. http://www.sciencemag.org/about/science-licenses-journal-article-reuseThis is an article distributed under the terms of the Science Journals Default License.},
	issn = {0036-8075, 1095-9203},
	url = {https://science.sciencemag.org/content/358/6363/659},
	doi = {10.1126/science.aao1807},
	abstract = {How early human groups were organized
Sequencing ancient hominid remains has provided insights into the relatedness between individuals. However, it is not clear whether ancient humans bred among close relatives, as is common in some modern human cultures. Sikora et al. report genome sequences from four early humans buried close together in western Russia about 34,000 years ago (see the Perspective by Bergström and Tyler-Smith). The individuals clustered together genetically and came from a population with a small effective size, but they were not very closely related. Thus, these people may represent a single social group that was part of a larger mating network, similar to contemporary hunter-gatherers. The lack of close inbreeding might help to explain the survival advantage of anatomically modern humans.
Science, this issue p. 659; see also p. 586
Present-day hunter-gatherers (HGs) live in multilevel social groups essential to sustain a population structure characterized by limited levels of within-band relatedness and inbreeding. When these wider social networks evolved among HGs is unknown. To investigate whether the contemporary HG strategy was already present in the Upper Paleolithic, we used complete genome sequences from Sunghir, a site dated to {\textasciitilde}34,000 years before the present, containing multiple anatomically modern human individuals. We show that individuals at Sunghir derive from a population of small effective size, with limited kinship and levels of inbreeding similar to HG populations. Our findings suggest that Upper Paleolithic social organization was similar to that of living HGs, with limited relatedness within residential groups embedded in a larger mating network.
Early Eurasian genomes identify Upper Paleolithic social organization similar to that observed in present-day hunter-gatherers.
Early Eurasian genomes identify Upper Paleolithic social organization similar to that observed in present-day hunter-gatherers.},
	language = {en},
	number = {6363},
	urldate = {2020-05-15},
	journal = {Science},
	author = {Sikora, Martin and Seguin-Orlando, Andaine and Sousa, Vitor C. and Albrechtsen, Anders and Korneliussen, Thorfinn and Ko, Amy and Rasmussen, Simon and Dupanloup, Isabelle and Nigst, Philip R. and Bosch, Marjolein D. and Renaud, Gabriel and Allentoft, Morten E. and Margaryan, Ashot and Vasilyev, Sergey V. and Veselovskaya, Elizaveta V. and Borutskaya, Svetlana B. and Deviese, Thibaut and Comeskey, Dan and Higham, Tom and Manica, Andrea and Foley, Robert and Meltzer, David J. and Nielsen, Rasmus and Excoffier, Laurent and Lahr, Marta Mirazon and Orlando, Ludovic and Willerslev, Eske},
	month = nov,
	year = {2017},
	pmid = {28982795},
	note = {Publisher: American Association for the Advancement of Science
Section: Report},
	pages = {659--662},
}

@article{pedersen_effect_2017,
	title = {The {Effect} of an {Extreme} and {Prolonged} {Population} {Bottleneck} on {Patterns} of {Deleterious} {Variation}: {Insights} from the {Greenlandic} {Inuit}},
	volume = {205},
	copyright = {Copyright © 2017 by the Genetics Society of America},
	issn = {0016-6731, 1943-2631},
	shorttitle = {The {Effect} of an {Extreme} and {Prolonged} {Population} {Bottleneck} on {Patterns} of {Deleterious} {Variation}},
	url = {https://www.genetics.org/content/205/2/787},
	doi = {10.1534/genetics.116.193821},
	abstract = {The genetic consequences of population bottlenecks on patterns of deleterious genetic variation in human populations are of tremendous interest. Based on exome sequencing of 18 Greenlandic Inuit we show that the Inuit have undergone a severe ∼20,000-year-long bottleneck. This has led to a markedly more extreme distribution of allele frequencies than seen for any other human population tested to date, making the Inuit the perfect population for investigating the effect of a bottleneck on patterns of deleterious variation. When comparing proxies for genetic load that assume an additive effect of deleterious alleles, the Inuit show, at most, a slight increase in load compared to European, East Asian, and African populations. Specifically, we observe {\textless}4\% increase in the number of derived deleterious alleles in the Inuit. In contrast, proxies for genetic load under a recessive model suggest that the Inuit have a significantly higher load (20\% increase or more) compared to other less bottlenecked human populations. Forward simulations under realistic models of demography support our empirical findings, showing up to a 6\% increase in the genetic load for the Inuit population across all models of dominance. Further, the Inuit population carries fewer deleterious variants than other human populations, but those that are present tend to be at higher frequency than in other populations. Overall, our results show how recent demographic history has affected patterns of deleterious variants in human populations.},
	language = {en},
	number = {2},
	urldate = {2020-05-15},
	journal = {Genetics},
	author = {Pedersen, Casper-Emil T. and Lohmueller, Kirk E. and Grarup, Niels and Bjerregaard, Peter and Hansen, Torben and Siegismund, Hans R. and Moltke, Ida and Albrechtsen, Anders},
	month = feb,
	year = {2017},
	pmid = {27903613},
	note = {Publisher: Genetics
Section: Investigations},
	keywords = {disease mapping, founder population, genetic load, isolated human populations, neutral theory},
	pages = {787--801},
}

@article{haller_slim_2019,
	title = {{SLiM} 3: {Forward} {Genetic} {Simulations} {Beyond} the {Wright}–{Fisher} {Model}},
	volume = {36},
	issn = {0737-4038},
	shorttitle = {{SLiM} 3},
	url = {https://academic.oup.com/mbe/article/36/3/632/5229931},
	doi = {10.1093/molbev/msy228},
	abstract = {Abstract.  With the desire to model population genetic processes under increasingly realistic scenarios, forward genetic simulations have become a critical part},
	language = {en},
	number = {3},
	urldate = {2020-05-15},
	journal = {Molecular Biology and Evolution},
	author = {Haller, Benjamin C. and Messer, Philipp W.},
	month = mar,
	year = {2019},
	note = {Publisher: Oxford Academic},
	pages = {632--637},
}

@article{bodmer_migration_1968,
	title = {A migration matrix model for the study of random genetic drift},
	volume = {59},
	issn = {0016-6731},
	language = {eng},
	number = {4},
	journal = {Genetics},
	author = {Bodmer, W. F. and Cavalli-Sforza, L. L.},
	month = aug,
	year = {1968},
	pmid = {5708302},
	pmcid = {PMC1212023},
	keywords = {Computers, Gene Frequency, Genetics, Population, Mathematics, Models, Biological, Models, Theoretical},
	pages = {565--592},
}

@article{crow_motoo_1997,
	title = {Motoo {Kimura}. 13 {November} 1924—13 {November} 1994},
	volume = {43},
	url = {https://royalsocietypublishing.org/doi/10.1098/rsbm.1997.0014},
	doi = {10.1098/rsbm.1997.0014},
	abstract = {Motoo Kimura's research contributions can be divided into two parts. The first is a series of papers on theoretical population genetics, the quality and quantity of which place him as the successor to the great trinity, R.A. Fisher, J.B.S. Haldane and Sewall Wright. The second is his neutral theory, the idea that the bulk of molecular evolutionary changes are driven by mutation and random chance, rather than by natural selection. The neutral theory brought him fame far beyond the confines of population genetics, and has made the name Motoo Kimura well-known to evolutionary biologists. (Motoo is pronounced ‘Mo-toe’, not ‘Mo-two’. By repeating the letter O, Kimura sought to indicate that this syllable was to be protracted. Unfortunately, rather than producing the desired effect, this more often led to mispronunciation.)},
	urldate = {2020-05-15},
	journal = {Biographical Memoirs of Fellows of the Royal Society},
	author = {Crow, James F.},
	month = nov,
	year = {1997},
	note = {Publisher: Royal Society},
	pages = {255--265},
}

@article{kimura_stepping_1953,
	title = {"{Stepping} {Stone}" model of population},
	volume = {3},
	journal = {Ann. Rept. Natn Inst. Genetics},
	author = {Kimura, Motoo},
	year = {1953},
	pages = {62--63},
}

@misc{marcus_introduction_2016,
	title = {Introduction to the {Wright}-{Fisher} {Model}},
	url = {https://stephens999.github.io/fiveMinuteStats/wright_fisher_model.html#examples},
	urldate = {2020-05-15},
	author = {Marcus, Joseph H.},
	month = mar,
	year = {2016},
}

@book{protter_intermediate_2012,
	title = {Intermediate {Calculus}},
	isbn = {978-1-4612-1086-3},
	language = {en},
	publisher = {Springer Science \& Business Media},
	author = {Protter, Murray H. and Morrey, Charles B. Jr},
	month = dec,
	year = {2012},
	note = {Google-Books-ID: 3lTmBwAAQBAJ},
	keywords = {Mathematics / Mathematical Analysis},
}

@book{durrett_probability_2008,
	address = {New York},
	edition = {2},
	series = {Probability and {Its} {Applications}},
	title = {Probability {Models} for {DNA} {Sequence} {Evolution}},
	isbn = {978-0-387-78168-6},
	url = {https://www.springer.com/gp/book/9780387781686},
	abstract = {How is genetic variability shaped by natural selection, demographic factors, and random genetic drift? To approach this question, we introduce and analyze a number of probability models beginning with the basics, and ending at the frontiers of current research. Throughout the book, the theory is developed in close connection with examples from the biology literature that illustrate the use of these results. Along the way, there are many numerical examples and graphs to illustrate the conclusions. This is the second edition and is twice the size of the first one. The material on recombination and the stepping stone model have been greatly expanded, there are many results form the last five years, and two new chapters on diffusion processes develop that viewpoint. This book is written for mathematicians and for biologists alike. No previous knowledge of concepts from biology is assumed, and only a basic knowledge of probability, including some familiarity with Markov chains and Poisson processes. The book has been restructured into a large number of subsections and written in a theorem-proof style, to more clearly highlight the main results and allow readers to find the results they need and to skip the proofs if they desire. Rick Durrett received his Ph.D. in operations research from Stanford University in 1976. He taught in the UCLA mathematics department before coming to Cornell in 1985. He is the author of eight books and 160 research papers, most of which concern the use of probability models in genetics and ecology. He is the academic father of 39 Ph.D. students and was recently elected to the National Academy of Sciences.},
	language = {en},
	urldate = {2020-05-14},
	publisher = {Springer-Verlag},
	author = {Durrett, Richard},
	year = {2008},
	doi = {10.1007/978-0-387-78168-6},
}

@article{tran_introduction_2013,
	title = {An introduction to the mathematical structure of the {Wright}–{Fisher} model of population genetics},
	volume = {132},
	issn = {1611-7530},
	url = {https://doi.org/10.1007/s12064-012-0170-3},
	doi = {10.1007/s12064-012-0170-3},
	abstract = {In this paper, we develop the mathematical structure of the Wright–Fisher model for evolution of the relative frequencies of two alleles at a diploid locus under random genetic drift in a population of fixed size in its simplest form, that is, without mutation or selection. We establish a new concept of a global solution for the diffusion approximation (Fokker–Planck equation), prove its existence and uniqueness and then show how one can easily derive all the essential properties of this random genetic drift process from our solution. Thus, our solution turns out to be superior to the local solution constructed by Kimura.},
	language = {en},
	number = {2},
	urldate = {2020-05-14},
	journal = {Theory in Biosciences},
	author = {Tran, Tat Dat and Hofrichter, Julian and Jost, Jürgen},
	month = jun,
	year = {2013},
	pages = {73--82},
}

@article{wright_differential_1945,
	title = {The {Differential} {Equation} of the {Distribution} of {Gene} {Frequencies}},
	volume = {31},
	issn = {0027-8424},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1078851/},
	number = {12},
	urldate = {2020-05-14},
	journal = {Proceedings of the National Academy of Sciences of the United States of America},
	author = {Wright, Sewall},
	month = dec,
	year = {1945},
	pmid = {16588707},
	pmcid = {PMC1078851},
	pages = {382--389},
}

@inproceedings{rackauckas_intuitive_2014,
	title = {An {Intuitive} {Introduction} {For} {Understanding} and {Solving} {Stochastic} {Differential} {Equations}},
	abstract = {Stochastic differential equations (SDEs) are a generalization of deterministic differential equations that incorporate a “noise term”. These equations can be useful in many applications where we assume that there are deterministic changes combined with noisy fluctuations. Ito’s Calculus is the mathematics for handling such equations. In this article we introduce stochastic differential equations and Ito’s calculus from an intuitive point of view, building the ideas from relatable probability theory and only straying into measure-theoretic probability (defining all concepts along the way) as necessary. All of the proofs are discussed intuitively and rigorously: step by step proofs are provided. . We start by reviewing the relevant probability needed in order to develop the stochastic processes. We then develop the mathematics of stochastic processes in order to define the Poisson Counter Process. We then define Brownian Motion, or the Wiener Process, as a limit of the Poisson Counter Process. By doing the definition in this manner, we are able to solve for many of the major properties and theorems of the stochastic calculus without resorting to measure-theoretic approaches. Along the way, examples are given to show how the calculus is actually used to solve problems. After developing Ito’s calculus for solving SDEs, we briefly discuss how these SDEs can be computationally simulated in case the analytical solutions are difficult or impossible. After this, we turn to defining some relevant terms in measure-theoretic probability in order to develop ideas such as conditional expectation and martingales. The conclusion to this article is a set of four applications. We show how the rules of the stochastic calculus and some basic martingale theory can be applied to solve problems such as option pricing, genetic drift, stochastic control, and stochastic filtering. The end of this article is a cheat sheet that details the fundamental rules for “doing” Ito’s calculus, like one would find on the cover flap of a calculus book. These are the equations/properties/rules that one rules to solve stochastic differential equations that are explained and justified in the article but put together for convenience.},
	author = {Rackauckas, Chris},
	year = {2014},
}

@article{rackauckas_intuitive_nodate,
	title = {An {Intuitive} {Introduction} {For} {Understanding} and {Solving} {Stochastic} {Diﬀerential} {Equations}},
	abstract = {Stochastic diﬀerential equations (SDEs) are a generalization of deterministic diﬀerential equations that incorporate a “noise term”. These equations can be useful in many applications where we assume that there are deterministic changes combined with noisy ﬂuctuations. Ito’s Calculus is the mathematics for handling such equations. In this article we introduce stochastic diﬀerential equations and Ito’s calculus from an intuitive point of view, building the ideas from relatable probability theory and only straying into measure-theoretic probability (deﬁning all concepts along the way) as necessary. All of the proofs are discussed intuitively and rigorously: step by step proofs are provided. We start by reviewing the relevant probability needed in order to develop the stochastic processes. We then develop the mathematics of stochastic processes in order to deﬁne the Poisson Counter Process. We then deﬁne Brownian Motion, or the Wiener Process, as a limit of the Poisson Counter Process. By doing the deﬁnition in this manner, we are able to solve for many of the major properties and theorems of the stochastic calculus without resorting to measure-theoretic approaches. Along the way, examples are given to show how the calculus is actually used to solve problems. After developing Ito’s calculus for solving SDEs, we brieﬂy discuss how these SDEs can be computationally simulated in case the analytical solutions are diﬃcult or impossible. After this, we turn to deﬁning some relevant terms in measure-theoretic probability in order to develop ideas such as conditional expectation and martingales. The conclusion to this article is a set of four applications. We show how the rules of the stochastic calculus and some basic martingale theory can be applied to solve problems such as option pricing, genetic drift, stochastic control, and stochastic ﬁltering. The end of this article is a cheat sheet that details the fundamental rules for “doing” Ito’s calculus, like one would ﬁnd on the cover ﬂap of a calculus book. These are the equations/properties/rules that one uses to solve stochastic diﬀerential equations that are explained and justiﬁed in the article but put together for convenience.},
	language = {en},
	author = {Rackauckas, Chris},
	pages = {96},
}

@book{kiss_mathematics_2017,
	address = {Cham},
	series = {Interdisciplinary {Applied} {Mathematics}},
	title = {Mathematics of {Epidemics} on {Networks}: {From} {Exact} to {Approximate} {Models}},
	volume = {46},
	isbn = {978-3-319-50804-7 978-3-319-50806-1},
	shorttitle = {Mathematics of {Epidemics} on {Networks}},
	url = {http://link.springer.com/10.1007/978-3-319-50806-1},
	language = {en},
	urldate = {2020-05-14},
	publisher = {Springer International Publishing},
	author = {Kiss, István Z. and Miller, Joel C. and Simon, Péter L.},
	year = {2017},
	doi = {10.1007/978-3-319-50806-1},
}

@book{kiss_mathematics_2017-1,
	address = {Cham},
	series = {Interdisciplinary {Applied} {Mathematics}},
	title = {Mathematics of {Epidemics} on {Networks}: {From} {Exact} to {Approximate} {Models}},
	volume = {46},
	isbn = {978-3-319-50804-7 978-3-319-50806-1},
	shorttitle = {Mathematics of {Epidemics} on {Networks}},
	url = {http://link.springer.com/10.1007/978-3-319-50806-1},
	language = {en},
	urldate = {2020-05-14},
	publisher = {Springer International Publishing},
	author = {Kiss, István Z. and Miller, Joel C. and Simon, Péter L.},
	year = {2017},
	doi = {10.1007/978-3-319-50806-1},
}

@article{wright_differential_1945-1,
	title = {The {Differential} {Equation} of the {Distribution} of {Gene} {Frequencies}},
	volume = {31},
	issn = {0027-8424},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1078851/},
	number = {12},
	urldate = {2020-05-14},
	journal = {Proceedings of the National Academy of Sciences of the United States of America},
	author = {Wright, Sewall},
	month = dec,
	year = {1945},
	pmid = {16588707},
	pmcid = {PMC1078851},
	pages = {382--389},
}

@article{tran_introduction_2013-1,
	title = {An introduction to the mathematical structure of the {Wright}–{Fisher} model of population genetics},
	volume = {132},
	issn = {1611-7530},
	url = {https://doi.org/10.1007/s12064-012-0170-3},
	doi = {10.1007/s12064-012-0170-3},
	abstract = {In this paper, we develop the mathematical structure of the Wright–Fisher model for evolution of the relative frequencies of two alleles at a diploid locus under random genetic drift in a population of fixed size in its simplest form, that is, without mutation or selection. We establish a new concept of a global solution for the diffusion approximation (Fokker–Planck equation), prove its existence and uniqueness and then show how one can easily derive all the essential properties of this random genetic drift process from our solution. Thus, our solution turns out to be superior to the local solution constructed by Kimura.},
	language = {en},
	number = {2},
	urldate = {2020-05-14},
	journal = {Theory in Biosciences},
	author = {Tran, Tat Dat and Hofrichter, Julian and Jost, Jürgen},
	month = jun,
	year = {2013},
	pages = {73--82},
}

@article{marcus_visualizing_2017,
	title = {Visualizing the geography of genetic variants},
	volume = {33},
	issn = {1367-4803},
	url = {https://academic.oup.com/bioinformatics/article/33/4/594/2608633},
	doi = {10.1093/bioinformatics/btw643},
	abstract = {AbstractSummary.  One of the key characteristics of any genetic variant is its geographic distribution. The geographic distribution can shed light on where an a},
	language = {en},
	number = {4},
	urldate = {2020-05-14},
	journal = {Bioinformatics},
	author = {Marcus, Joseph H. and Novembre, John},
	month = feb,
	year = {2017},
	note = {Publisher: Oxford Academic},
	pages = {594--595},
}

@article{bomba_impact_2017,
	title = {The impact of rare and low-frequency genetic variants in common disease},
	volume = {18},
	issn = {1474-760X},
	url = {https://doi.org/10.1186/s13059-017-1212-4},
	doi = {10.1186/s13059-017-1212-4},
	abstract = {Despite thousands of genetic loci identified to date, a large proportion of genetic variation predisposing to complex disease and traits remains unaccounted for. Advances in sequencing technology enable focused explorations on the contribution of low-frequency and rare variants to human traits. Here we review experimental approaches and current knowledge on the contribution of these genetic variants in complex disease and discuss challenges and opportunities for personalised medicine.},
	number = {1},
	urldate = {2020-05-14},
	journal = {Genome Biology},
	author = {Bomba, Lorenzo and Walter, Klaudia and Soranzo, Nicole},
	month = apr,
	year = {2017},
	pages = {77},
}

@article{geerlings_geographic_2018,
	title = {Geographic distribution of rare variants associated with age-related macular degeneration},
	volume = {24},
	issn = {1090-0535},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5788811/},
	abstract = {Purpose
A recent genome-wide association study by the International Age-related Macular Degeneration Genomics Consortium (IAMDGC) identified seven rare variants that are individually associated with age-related macular degeneration (AMD), the most common cause of vision loss in the elderly. In literature, several of these rare variants have been reported with different frequencies and odds ratios across populations of Europe and North America. Here, we aim to describe the representation of these seven AMD-associated rare variants in different geographic regions based on 24 AMD studies.

Methods
We explored the occurrence of seven rare variants independently associated with AMD (CFH rs121913059 (p.Arg1210Cys), CFI rs141853578 (p.Gly119Arg), C3 rs147859257 (p.Lys155Gln), and C9 rs34882957 (p.Pro167Ser)) and three non-coding variants in or near the CFH gene (rs148553336, rs35292876, and rs191281603) in 24 AMD case-control studies. We studied the difference in distribution, interaction, and effect size for each of the rare variants based on the minor allele frequency within the different geographic regions.

Results
We demonstrate that two rare AMD-associated variants in the CFH gene (rs121913059 [p.Arg1210Cys] and rs35292876) deviate in frequency among different geographic regions (p=0.004 and p=0.001, respectively). The risk estimates of each of the seven rare variants were comparable across the geographic regions.

Conclusions
The results emphasize the importance of identifying population-specific rare variants, for example, by performing sequencing studies in case-control studies of various populations, because their identification may have implications for diagnostic screening and personalized treatment.},
	urldate = {2020-05-14},
	journal = {Molecular Vision},
	author = {Geerlings, Maartje J. and Kersten, Eveline and Groenewoud, Joannes M.M. and Fritsche, Lars G. and Hoyng, Carel B. and de Jong, Eiko K. and den Hollander, Anneke I.},
	month = jan,
	year = {2018},
	pmid = {29410599},
	pmcid = {PMC5788811},
	pages = {75--82},
}

@article{etheridge_modelling_nodate,
	title = {Modelling the genetics of spatially structured populations},
	language = {en},
	author = {Etheridge, Alison},
	pages = {2},
}

@misc{noauthor_genetical_nodate,
	title = {The genetical theory of natural selection.},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1461012/},
	urldate = {2020-05-13},
}

@article{visscher_r_2019,
	title = {From {R}.{A}. {Fisher}’s 1918 {Paper} to {GWAS} a {Century} {Later}},
	volume = {211},
	copyright = {Copyright © 2019 by the Genetics Society of America},
	issn = {0016-6731, 1943-2631},
	url = {https://www.genetics.org/content/211/4/1125},
	doi = {10.1534/genetics.118.301594},
	abstract = {The genetics and evolution of complex traits, including quantitative traits and disease, have been hotly debated ever since Darwin. A century ago, a paper from R.A. Fisher reconciled Mendelian and biometrical genetics in a landmark contribution that is now accepted as the main foundation stone of the field of quantitative genetics. Here, we give our perspective on Fisher’s 1918 paper in the context of how and why it is relevant in today’s genome era. We mostly focus on human trait variation, in part because Fisher did so too, but the conclusions are general and extend to other natural populations, and to populations undergoing artificial selection.},
	language = {en},
	number = {4},
	urldate = {2020-05-13},
	journal = {Genetics},
	author = {Visscher, Peter M. and Goddard, Michael E.},
	month = apr,
	year = {2019},
	pmid = {30967441},
	note = {Publisher: Genetics
Section: Perspectives},
	keywords = {Fisher 1918, GWAS, quantitative trait},
	pages = {1125--1130},
}

@article{charlesworth_population_2017,
	title = {Population genetics from 1966 to 2016},
	volume = {118},
	copyright = {2017 The Author(s)},
	issn = {1365-2540},
	url = {https://www.nature.com/articles/hdy201655},
	doi = {10.1038/hdy.2016.55},
	abstract = {We describe the astonishing changes and progress that have occurred in the field of population genetics over the past 50 years, slightly longer than the time since the first Population Genetics Group (PGG) meeting in January 1968. We review the major questions and controversies that have preoccupied population geneticists during this time (and were often hotly debated at PGG meetings). We show how theoretical and empirical work has combined to generate a highly productive interaction involving successive developments in the ability to characterise variability at the molecular level, to apply mathematical models to the interpretation of the data and to use the results to answer biologically important questions, even in nonmodel organisms. We also describe the changes from a field that was largely dominated by UK and North American biologists to a much more international one (with the PGG meetings having made important contributions to the increased number of population geneticists in several European countries). Although we concentrate on the earlier history of the field, because developments in recent years are more familiar to most contemporary researchers, we end with a brief outline of topics in which new understanding is still actively developing.},
	language = {en},
	number = {1},
	urldate = {2020-05-13},
	journal = {Heredity},
	author = {Charlesworth, B. and Charlesworth, D.},
	month = jan,
	year = {2017},
	note = {Number: 1
Publisher: Nature Publishing Group},
	pages = {2--9},
}

@misc{noauthor_spatial_nodate,
	title = {Spatial {Population} {Genetics}: {It}'s {About} {Time} {\textbar} {Annual} {Review} of {Ecology}, {Evolution}, and {Systematics}},
	url = {https://www.annualreviews.org/doi/full/10.1146/annurev-ecolsys-110316-022659},
	urldate = {2020-05-13},
}

@article{kern_neutral_2018,
	title = {The {Neutral} {Theory} in {Light} of {Natural} {Selection}},
	volume = {35},
	issn = {0737-4038},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5967545/},
	doi = {10.1093/molbev/msy092},
	abstract = {In this perspective, we evaluate the explanatory power of the neutral theory of molecular evolution, 50 years after its introduction by Kimura. We argue that the neutral theory was supported by unreliable theoretical and empirical evidence from the beginning, and that in light of modern, genome-scale data, we can firmly reject its universality. The ubiquity of adaptive variation both within and between species means that a more comprehensive theory of molecular evolution must be sought.},
	number = {6},
	urldate = {2020-05-13},
	journal = {Molecular Biology and Evolution},
	author = {Kern, Andrew D and Hahn, Matthew W},
	month = jun,
	year = {2018},
	pmid = {29722831},
	pmcid = {PMC5967545},
	pages = {1366--1371},
}

@article{novembre_interpreting_2008,
	title = {Interpreting principal component analyses of spatial population genetic variation},
	volume = {40},
	copyright = {2008 Nature Publishing Group},
	issn = {1546-1718},
	url = {https://www.nature.com/articles/ng.139},
	doi = {10.1038/ng.139},
	abstract = {Nearly 30 years ago, Cavalli-Sforza et al. pioneered the use of principal component analysis (PCA) in population genetics and used PCA to produce maps summarizing human genetic variation across continental regions1. They interpreted gradient and wave patterns in these maps as signatures of specific migration events1,2,3. These interpretations have been controversial4,5,6,7, but influential8, and the use of PCA has become widespread in analysis of population genetics data9,10,11,12,13. However, the behavior of PCA for genetic data showing continuous spatial variation, such as might exist within human continental groups, has been less well characterized. Here, we find that gradients and waves observed in Cavalli-Sforza et al.'s maps resemble sinusoidal mathematical artifacts that arise generally when PCA is applied to spatial data, implying that the patterns do not necessarily reflect specific migration events. Our findings aid interpretation of PCA results and suggest how PCA can help correct for continuous population structure in association studies.},
	language = {en},
	number = {5},
	urldate = {2020-05-13},
	journal = {Nature Genetics},
	author = {Novembre, John and Stephens, Matthew},
	month = may,
	year = {2008},
	note = {Number: 5
Publisher: Nature Publishing Group},
	pages = {646--649},
}

@article{malecot_heterozygosity_1975,
	title = {Heterozygosity and relationship in regularly subdivided populations},
	volume = {8},
	issn = {0040-5809},
	doi = {10.1016/0040-5809(75)90033-7},
	language = {eng},
	number = {2},
	journal = {Theoretical Population Biology},
	author = {Malécot, G.},
	month = oct,
	year = {1975},
	pmid = {1198353},
	keywords = {Fourier Analysis, Gene Frequency, Genetics, Population, Heterozygote, Models, Biological, Mutation},
	pages = {212--241},
}

@article{eyre-walker_evolution_2010,
	title = {Evolution in health and medicine {Sackler} colloquium: {Genetic} architecture of a complex trait and its implications for fitness and genome-wide association studies},
	volume = {107 Suppl 1},
	issn = {1091-6490},
	shorttitle = {Evolution in health and medicine {Sackler} colloquium},
	doi = {10.1073/pnas.0906182107},
	abstract = {A model is investigated in which mutations that affect a complex trait (e.g., heart disease) also affect fitness because the trait is a component of fitness or because the mutations have pleiotropic effects on fitness. The model predicts that the genetic variance, and hence the heritability, in the trait is contributed by mutations at low frequency in the population, unless the mean strength of selection of mutations that affect the trait is very small or weakly selected mutations tend to contribute disproportionately to the trait compared with strongly selected mutations. Furthermore, it is shown that each rare mutation tends to contribute more to the variance than each common mutation. These results may explain why most genome-wide association studies have failed to find associations that explain much of the variance. It is also shown that most of the variance in fitness contributed by new nonsynonymous mutations is caused by mutations at very low frequency in the population. This implies that most low-frequency SNPs, which are observed in current resequencing studies of, for example, 100 chromosomes, probably have little impact on the variance in fitness or traits. Finally, it is shown that the variance contributed by a category of mutations (e.g., coding or regulatory) depends largely upon the mean strength of selection; this has implications for understanding which types of mutations are likely to be responsible for the variance in fitness and inherited disease.},
	language = {eng},
	journal = {Proceedings of the National Academy of Sciences of the United States of America},
	author = {Eyre-Walker, Adam},
	month = jan,
	year = {2010},
	pmid = {20133822},
	pmcid = {PMC2868283},
	keywords = {Genetic Variation, Genome-Wide Association Study, Humans, Mutation, Selection, Genetic},
	pages = {1752--1756},
}

@article{kimura_stepping_1964,
	title = {The {Stepping} {Stone} {Model} of {Population} {Structure} and the {Decrease} of {Genetic} {Correlation} with {Distance}},
	volume = {49},
	issn = {0016-6731},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1210594/},
	number = {4},
	urldate = {2020-05-13},
	journal = {Genetics},
	author = {Kimura, Motoo and Weiss, George H.},
	month = apr,
	year = {1964},
	pmid = {17248204},
	pmcid = {PMC1210594},
	pages = {561--576},
}

@article{fisher_mathematical_1922,
	title = {On the mathematical foundations of theoretical statistics},
	volume = {222},
	url = {https://royalsocietypublishing.org/doi/abs/10.1098/rsta.1922.0009},
	doi = {10.1098/rsta.1922.0009},
	abstract = {Several reasons have contributed to the prolonged neglect into which the study of statistics, in its theoretical aspects, has fallen. In spite of the immense amount of fruitful labour which has been expended in its practical applications, the basic principles of this organ of science are still in a state of obscurity, and it cannot be denied that, during the recent rapid development of practical methods, fundamental problems have been ignored and fundamental paradoxes left unresolved. This anomalous state of statistical science is strikingly exemplified by a recent paper entitled "The Fundamental Problem of Practical Statistics," in which one of the most eminent of modern statisticians presents what purports to be a general proof of BAYES' postulate, a proof which, in the opinion of a second statistician of equal eminence, "seems to rest upon a very peculiar -- not to say hardly supposable -- relation."},
	number = {594-604},
	urldate = {2020-05-13},
	journal = {Philosophical Transactions of the Royal Society of London. Series A, Containing Papers of a Mathematical or Physical Character},
	author = {Fisher, R. A. and Russell, Edward John},
	month = jan,
	year = {1922},
	note = {Publisher: Royal Society},
	pages = {309--368},
}

@article{wright_evolution_1931,
	title = {Evolution in {Mendelian} {Populations}},
	volume = {16},
	issn = {0016-6731},
	language = {eng},
	number = {2},
	journal = {Genetics},
	author = {Wright, S.},
	month = mar,
	year = {1931},
	pmid = {17246615},
	pmcid = {PMC1201091},
	pages = {97--159},
}

@article{marcus_visualizing_2017-1,
	title = {Visualizing the geography of genetic variants},
	volume = {33},
	issn = {1367-4811},
	doi = {10.1093/bioinformatics/btw643},
	abstract = {Summary: One of the key characteristics of any genetic variant is its geographic distribution. The geographic distribution can shed light on where an allele first arose, what populations it has spread to, and in turn on how migration, genetic drift, and natural selection have acted. The geographic distribution of a genetic variant can also be of great utility for medical/clinical geneticists and collectively many genetic variants can reveal population structure. Here we develop an interactive visualization tool for rapidly displaying the geographic distribution of genetic variants. Through a REST API and dynamic front-end, the Geography of Genetic Variants (GGV) browser ( http://popgen.uchicago.edu/ggv/ ) provides maps of allele frequencies in populations distributed across the globe.
Availability and Implementation: GGV is implemented as a website ( http://popgen.uchicago.edu/ggv/ ) which employs an API to access frequency data ( http://popgen.uchicago.edu/freq\_api/ ). Python and javascript source code for the website and the API are available at: http://github.com/NovembreLab/ggv/ and http://github.com/NovembreLab/ggv-api/ .
Contact: jnovembre@uchicago.edu.
Supplementary information: Supplementary data are available at Bioinformatics online.},
	language = {eng},
	number = {4},
	journal = {Bioinformatics (Oxford, England)},
	author = {Marcus, Joseph H. and Novembre, John},
	year = {2017},
	pmid = {27742697},
	pmcid = {PMC5408806},
	keywords = {Genetic Variation, Genome, Human, Genomics, Humans, Phylogeography, Software},
	pages = {594--595},
}

@article{polechova_genetic_2011,
	title = {Genetic {Drift} {Widens} the {Expected} {Cline} but {Narrows} the {Expected} {Cline} {Width}},
	volume = {189},
	issn = {0016-6731},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3176109/},
	doi = {10.1534/genetics.111.129817},
	abstract = {Random genetic drift shifts clines in space, alters their width, and distorts their shape. Such random fluctuations complicate inferences from cline width and position. Notably, the effect of genetic drift on the expected shape of the cline is opposite to the naive (but quite common) misinterpretation of classic results on the expected cline. While random drift on average broadens the overall cline in expected allele frequency, it narrows the width of any particular cline. The opposing effects arise because locally, drift drives alleles to fixation—but fluctuations in position widen the expected cline. The effect of genetic drift can be predicted from standardized variance in allele frequencies, averaged across the habitat: 〈F〉. A cline maintained by spatially varying selection (step change) is expected to be narrower by a factor of 1−〈F〉 relative to the cline in the absence of drift. The expected cline is broader by the inverse of this factor. In a tension zone maintained by underdominance, the expected cline width is narrower by about 1 – 〈F〉 relative to the width in the absence of drift. Individual clines can differ substantially from the expectation, and we give quantitative predictions for the variance in cline position and width. The predictions apply to clines in almost one-dimensional circumstances such as hybrid zones in rivers, deep valleys, or along a coast line and give a guide to what patterns to expect in two dimensions.},
	number = {1},
	urldate = {2020-05-12},
	journal = {Genetics},
	author = {Polechová, Jitka and Barton, Nick},
	month = sep,
	year = {2011},
	pmid = {21705747},
	pmcid = {PMC3176109},
	pages = {227--235},
}

@article{vasemagi_adaptive_2006,
	title = {The {Adaptive} {Hypothesis} of {Clinal} {Variation} {Revisited}: {Single}-{Locus} {Clines} as a {Result} of {Spatially} {Restricted} {Gene} {Flow}},
	volume = {173},
	issn = {0016-6731},
	shorttitle = {The {Adaptive} {Hypothesis} of {Clinal} {Variation} {Revisited}},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1569722/},
	doi = {10.1534/genetics.106.059881},
	number = {4},
	urldate = {2020-05-12},
	journal = {Genetics},
	author = {Vasemägi, Anti},
	month = aug,
	year = {2006},
	pmid = {16849603},
	pmcid = {PMC1569722},
	pages = {2411--2414},
}

@article{baines_role_2004,
	title = {The {Role} of {Natural} {Selection} in {Genetic} {Differentiation} of {Worldwide} {Populations} of {Drosophila} ananassae},
	volume = {168},
	issn = {0016-6731},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1448739/},
	doi = {10.1534/genetics.104.027482},
	abstract = {The main evolutionary forces leading to genetic differentiation between populations are generally considered to be natural selection, random genetic drift, and limited migration. However, little empirical evidence exists to help explain the extent, mechanism, and relative role of these forces. In this study, we make use of the differential migration behavior of genes located in regions of low and high recombination to infer the role and demographic distribution of natural selection in Drosophila ananassae. Sequence data were obtained from 13 populations, representing almost the entire range of cosmopolitan D. ananassae. The pattern of variation at a 5.1-kb fragment of the furrowed gene, located in a region of very low recombination, appears strikingly different from that of 10 noncoding DNA fragments (introns) in regions of normal to high recombination. Most interestingly, two main haplotypes are present at furrowed, one being fixed in northern populations and the other being fixed or in high frequency in more southern populations. A cline in the frequency of one of these haplotypes occurs in parallel latitudinal transects. Taken together, significant clinal variation and a test against alternative models of natural selection provide evidence of two independent selective sweeps restricted to specific regions of the species range.},
	number = {4},
	urldate = {2020-05-12},
	journal = {Genetics},
	author = {Baines, John F. and Das, Aparup and Mousset, Sylvain and Stephan, Wolfgang},
	month = dec,
	year = {2004},
	pmid = {15611169},
	pmcid = {PMC1448739},
	pages = {1987--1998},
}

@article{ito_stochastic_1944,
	title = {Stochastic integral},
	volume = {20},
	issn = {0369-9846},
	url = {https://projecteuclid.org/euclid.pja/1195572786},
	doi = {10.3792/pia/1195572786},
	abstract = {Project Euclid - mathematics and statistics online},
	language = {EN},
	number = {8},
	urldate = {2020-05-12},
	journal = {Proceedings of the Imperial Academy},
	author = {Itô, Kiyosi},
	year = {1944},
	mrnumber = {MR0014633},
	zmnumber = {0060.29105},
	note = {Publisher: The Japan Academy},
	pages = {519--524},
}

@misc{noauthor_icahn_nodate,
	title = {Icahn {School} of {Medicine} at {Mount} {Sinai}/{Mount} {Sinai} {School} of {Medicine} - {Application}},
	url = {https://applymd.mssm.edu/Applicant/PreMatric/Orientation.aspx},
	urldate = {2020-05-12},
}

@article{felsenstein_pain_1975,
	title = {A {Pain} in the {Torus}: {Some} {Difficulties} with {Models} of {Isolation} by {Distance}},
	volume = {109},
	issn = {0003-0147},
	shorttitle = {A {Pain} in the {Torus}},
	url = {https://www.jstor.org/stable/2459700},
	abstract = {Malecot's models of isolation by distance seem to assume random distribution in space, Poisson distribution of offspring number, and independent migration of individuals. These assumptions are inconsistent. By calculating the probability of joint occupancy of two locations at a distance x, it is shown that a haploid model embodying the latter two assumptions violates the first, forming larger and larger clumps of individuals separated by greater and greater distances. Thus the model is biologically irrelevant, even though it is possible to use it to calculate probabilities of genetic identity at different distances. The formation of these clumps is verified by computer simulation. The same phenomenon occurs in two- and three-dimensional models. In models of finite regions (a circle or a torus), the formation of clumps takes the form of the certainty of ultimate local extinction of the organism in each region. This does not occur if we hold the number of individuals on the circle or torus constant, but considerable departure from random distribution still results. The stepping-stone models appear for the present to be the only well-defined models of geographically structured populations with finite population density.},
	number = {967},
	urldate = {2020-05-12},
	journal = {The American Naturalist},
	author = {Felsenstein, Joseph},
	year = {1975},
	note = {Publisher: [University of Chicago Press, American Society of Naturalists]},
	pages = {359--368},
}

@article{felsenstein_pain_1975-1,
	title = {A {Pain} in the {Torus}: {Some} {Difficulties} with {Models} of {Isolation} by {Distance}},
	volume = {109},
	issn = {0003-0147},
	shorttitle = {A {Pain} in the {Torus}},
	url = {https://www.jstor.org/stable/2459700},
	abstract = {Malecot's models of isolation by distance seem to assume random distribution in space, Poisson distribution of offspring number, and independent migration of individuals. These assumptions are inconsistent. By calculating the probability of joint occupancy of two locations at a distance x, it is shown that a haploid model embodying the latter two assumptions violates the first, forming larger and larger clumps of individuals separated by greater and greater distances. Thus the model is biologically irrelevant, even though it is possible to use it to calculate probabilities of genetic identity at different distances. The formation of these clumps is verified by computer simulation. The same phenomenon occurs in two- and three-dimensional models. In models of finite regions (a circle or a torus), the formation of clumps takes the form of the certainty of ultimate local extinction of the organism in each region. This does not occur if we hold the number of individuals on the circle or torus constant, but considerable departure from random distribution still results. The stepping-stone models appear for the present to be the only well-defined models of geographically structured populations with finite population density.},
	number = {967},
	urldate = {2020-05-12},
	journal = {The American Naturalist},
	author = {Felsenstein, Joseph},
	year = {1975},
	note = {Publisher: [University of Chicago Press, American Society of Naturalists]},
	pages = {359--368},
}

@misc{noauthor_reanalysis_nodate,
	title = {A reanalysis of mouse {ENCODE} comparative gene... {\textbar} {F1000Research}},
	url = {https://f1000research.com/articles/4-121/v1},
	urldate = {2020-05-12},
}

@article{woolston_potential_2015,
	title = {Potential flaws in genomics paper scrutinized on {Twitter}},
	volume = {521},
	url = {http://www.nature.com/news/potential-flaws-in-genomics-paper-scrutinized-on-twitter-1.17591},
	doi = {10.1038/521397f},
	abstract = {Reanalysis of a study that compared gene expression in mice and humans tests social media as a forum for discussing research results.},
	language = {en},
	number = {7553},
	urldate = {2020-05-12},
	journal = {Nature News},
	author = {Woolston, Chris},
	month = may,
	year = {2015},
	note = {Section: Research Highlights: Social Selection},
	pages = {397},
}

@article{goh_why_2017,
	series = {Special {Issue}: {Computation} and {Modeling}},
	title = {Why {Batch} {Effects} {Matter} in {Omics} {Data}, and {How} to {Avoid} {Them}},
	volume = {35},
	issn = {0167-7799},
	url = {http://www.sciencedirect.com/science/article/pii/S0167779917300367},
	doi = {10.1016/j.tibtech.2017.02.012},
	abstract = {Effective integration and analysis of new high-throughput data, especially gene-expression and proteomic-profiling data, are expected to deliver novel clinical insights and therapeutic options. Unfortunately, technical heterogeneity or batch effects (different experiment times, handlers, reagent lots, etc.) have proven challenging. Although batch effect-correction algorithms (BECAs) exist, we know little about effective batch-effect mitigation: even now, new batch effect-associated problems are emerging. These include false effects due to misapplying BECAs and positive bias during model evaluations. Depending on the choice of algorithm and experimental set-up, biological heterogeneity can be mistaken for batch effects and wrongfully removed. Here, we examine these emerging batch effect-associated problems, propose a series of best practices, and discuss some of the challenges that lie ahead.},
	language = {en},
	number = {6},
	urldate = {2020-05-12},
	journal = {Trends in Biotechnology},
	author = {Goh, Wilson Wen Bin and Wang, Wei and Wong, Limsoon},
	month = jun,
	year = {2017},
	keywords = {batch effect, cross-validation, data integration, heterogeneity, reproducibility},
	pages = {498--507},
}

@article{park_distribution_2011,
	title = {Distribution of allele frequencies and effect sizes and their interrelationships for common genetic susceptibility variants},
	volume = {108},
	copyright = {©  . Freely available online through the PNAS open access option.},
	issn = {0027-8424, 1091-6490},
	url = {https://www.pnas.org/content/108/44/18026},
	doi = {10.1073/pnas.1114759108},
	abstract = {Recent discoveries of hundreds of common susceptibility SNPs from genome-wide association studies provide a unique opportunity to examine population genetic models for complex traits. In this report, we investigate distributions of various population genetic parameters and their interrelationships using estimates of allele frequencies and effect-size parameters for about 400 susceptibility SNPs across a spectrum of qualitative and quantitative traits. We calibrate our analysis by statistical power for detection of SNPs to account for overrepresentation of variants with larger effect sizes in currently known SNPs that are expected due to statistical power for discovery. Across all qualitative disease traits, minor alleles conferred “risk” more often than “protection.” Across all traits, an inverse relationship existed between “regression effects” and allele frequencies. Both of these trends were remarkably strong for type I diabetes, a trait that is most likely to be influenced by selection, but were modest for other traits such as human height or late-onset diseases such as type II diabetes and cancers. Across all traits, the estimated effect-size distribution suggested the existence of increasingly large numbers of susceptibility SNPs with decreasingly small effects. For most traits, the set of SNPs with intermediate minor allele frequencies (5–20\%) contained an unusually small number of susceptibility loci and explained a relatively small fraction of heritability compared with what would be expected from the distribution of SNPs in the general population. These trends could have several implications for future studies of common and uncommon variants.},
	language = {en},
	number = {44},
	urldate = {2020-05-12},
	journal = {Proceedings of the National Academy of Sciences},
	author = {Park, Ju-Hyun and Gail, Mitchell H. and Weinberg, Clarice R. and Carroll, Raymond J. and Chung, Charles C. and Wang, Zhaoming and Chanock, Stephen J. and Fraumeni, Joseph F. and Chatterjee, Nilanjan},
	month = nov,
	year = {2011},
	pmid = {22003128},
	note = {Publisher: National Academy of Sciences
Section: Biological Sciences},
	keywords = {genetic prediction, missing heritability, population genetics},
	pages = {18026--18031},
}

@article{meuwissen_prediction_2001,
	title = {Prediction of total genetic value using genome-wide dense marker maps},
	volume = {157},
	issn = {0016-6731},
	abstract = {Recent advances in molecular genetic techniques will make dense marker maps available and genotyping many individuals for these markers feasible. Here we attempted to estimate the effects of approximately 50,000 marker haplotypes simultaneously from a limited number of phenotypic records. A genome of 1000 cM was simulated with a marker spacing of 1 cM. The markers surrounding every 1-cM region were combined into marker haplotypes. Due to finite population size N(e) = 100, the marker haplotypes were in linkage disequilibrium with the QTL located between the markers. Using least squares, all haplotype effects could not be estimated simultaneously. When only the biggest effects were included, they were overestimated and the accuracy of predicting genetic values of the offspring of the recorded animals was only 0.32. Best linear unbiased prediction of haplotype effects assumed equal variances associated to each 1-cM chromosomal segment, which yielded an accuracy of 0.73, although this assumption was far from true. Bayesian methods that assumed a prior distribution of the variance associated with each chromosome segment increased this accuracy to 0.85, even when the prior was not correct. It was concluded that selection on genetic values predicted from markers could substantially increase the rate of genetic gain in animals and plants, especially if combined with reproductive techniques to shorten the generation interval.},
	language = {eng},
	number = {4},
	journal = {Genetics},
	author = {Meuwissen, T. H. and Hayes, B. J. and Goddard, M. E.},
	month = apr,
	year = {2001},
	pmid = {11290733},
	pmcid = {PMC1461589},
	keywords = {Animals, Bayes Theorem, Breeding, Computer Simulation, Genetic Markers, Haplotypes, Least-Squares Analysis, Linkage Disequilibrium, Models, Genetic, Quantitative Trait, Heritable},
	pages = {1819--1829},
}

@article{xie_efficiency_1998,
	title = {Efficiency of multistage marker-assisted selection in the improvement of multiple quantitative traits},
	volume = {80},
	copyright = {1998 The Genetical Society of Great Britain},
	issn = {1365-2540},
	url = {https://www.nature.com/articles/6883080},
	doi = {10.1046/j.1365-2540.1998.00308.x},
	abstract = {The application of marker-assisted selection (MAS) to breeding programmes depends on its relative cost and the expected economic return compared to conventional phenotypic selection. The relative efficiency of MAS can be increased through a two-stage selection scheme or through marker-based, multiple-trait improvement. However, the effectiveness of these alternatives has not been quantified. In this study, we evaluate the efficiency of MAS relative to conventional phenotypic selection and marker-only selection in multistage selection for the improvement of multiple traits. We further incorporate the costs of obtaining measurements on phenotypic characters and marker loci into the objective function to evaluate the efficiency of MAS with respect to the gain per unit cost. Deterministic analyses indicate that excluding costs, multiple-trait MAS can be used to increase the aggregate breeding values in quantitative characters and is expected to be more effective than conventional selection or single-trait MAS. Two-stage MAS has a slightly reduced gain because of culling in the first stage. If the objective function is to maximize the gain per unit cost, multiple-trait MAS is inferior to phenotypic selection in most of the selection schemes investigated when the cost ratio (r) of obtaining measurements on phenotypic characters to scoring marker loci is less than unity (r≤1.0) and the heritability (h2) is greater than 0.3. The efficiency of MAS increases as r increases and h2 decreases. For MAS to be more effective, it is necessary to decrease further the cost associated with molecular marker assays.},
	language = {en},
	number = {4},
	urldate = {2020-05-12},
	journal = {Heredity},
	author = {Xie, Chongqing and Xu, Shizhong},
	month = apr,
	year = {1998},
	note = {Number: 4
Publisher: Nature Publishing Group},
	pages = {489--498},
}

@article{dudbridge_power_2013,
	title = {Power and {Predictive} {Accuracy} of {Polygenic} {Risk} {Scores}},
	volume = {9},
	issn = {1553-7390},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3605113/},
	doi = {10.1371/journal.pgen.1003348},
	abstract = {Polygenic scores have recently been used to summarise genetic effects among an ensemble of markers that do not individually achieve significance in a large-scale association study. Markers are selected using an initial training sample and used to construct a score in an independent replication sample by forming the weighted sum of associated alleles within each subject. Association between a trait and this composite score implies that a genetic signal is present among the selected markers, and the score can then be used for prediction of individual trait values. This approach has been used to obtain evidence of a genetic effect when no single markers are significant, to establish a common genetic basis for related disorders, and to construct risk prediction models. In some cases, however, the desired association or prediction has not been achieved. Here, the power and predictive accuracy of a polygenic score are derived from a quantitative genetics model as a function of the sizes of the two samples, explained genetic variance, selection thresholds for including a marker in the score, and methods for weighting effect sizes in the score. Expressions are derived for quantitative and discrete traits, the latter allowing for case/control sampling. A novel approach to estimating the variance explained by a marker panel is also proposed. It is shown that published studies with significant association of polygenic scores have been well powered, whereas those with negative results can be explained by low sample size. It is also shown that useful levels of prediction may only be approached when predictors are estimated from very large samples, up to an order of magnitude greater than currently available. Therefore, polygenic scores currently have more utility for association testing than predicting complex traits, but prediction will become more feasible as sample sizes continue to grow., Recently there has been much interest in combining multiple genetic markers into a single score for predicting disease risk. Even if many of the individual markers have no detected effect, the combined score could be a strong predictor of disease. This has allowed researchers to demonstrate that some diseases have a strong genetic basis, even if few actual genes have been identified, and it has also revealed a common genetic basis for distinct diseases. These analyses have so far been performed opportunistically, with mixed results. Here I derive formulae based on the heritability of disease and size of the study, allowing researchers to plan their analyses from a more informed position. I show that discouraging results in some previous studies were due to the low number of subjects studied, but a modest increase in study size would allow more successful analysis. However, I also show that, for genetics to become useful for predicting individual risk of disease, hundreds of thousands of subjects may be needed to estimate the gene effects. This is larger than most existing studies, but will become more common in the near future, so that gene scores will become more useful for predicting disease than has appeared to date.},
	number = {3},
	urldate = {2020-05-12},
	journal = {PLoS Genetics},
	author = {Dudbridge, Frank},
	month = mar,
	year = {2013},
	pmid = {23555274},
	pmcid = {PMC3605113},
}

@article{hamada_equilibrium_2019,
	title = {Equilibrium properties of the spatial {SIS} model as a point pattern dynamics - {How} is infection distributed over space?},
	volume = {468},
	issn = {0022-5193},
	url = {http://www.sciencedirect.com/science/article/pii/S0022519319300608},
	doi = {10.1016/j.jtbi.2019.02.005},
	abstract = {We revisit the classical epidemiological SIS model as a stochastic point pattern dynamics with special focus on its spatial distribution at equilibrium. In this model, each point on a continuous space is either susceptible S or infectious I, and infection occurs with an infection kernel as a function of distance from I to S. This stochastic process has been mathematically described by the hierarchical dynamics of the probabilities that a point, a pair made by two points, and a triplet made by three points, etc., is in a specific configuration of status. Using a simple closure thereby triplet probabilities that appear in the dynamics are approximated, we show that the average singlet probabilities and the pair probabilities that describe spatial distributions of Ss and Is at equilibrium can be explicitly derived using the infection kernel; Is are spatially clustered in the same order of the infection kernel. The results highlight the advantage of point pattern approach to model spatial population dynamics in general ecology where local interactions among individuals likely depend on distance between them.},
	language = {en},
	urldate = {2020-05-06},
	journal = {Journal of Theoretical Biology},
	author = {Hamada, Miki and Takasu, Fugo},
	month = may,
	year = {2019},
	keywords = {Distance-dependent interactions, Epidemiology, Spatial ecology, Spatial population dynamics},
	pages = {12--26},
}

@article{visscher_r_2019-1,
	title = {From {R}.{A}. {Fisher}’s 1918 {Paper} to {GWAS} a {Century} {Later}},
	volume = {211},
	copyright = {Copyright © 2019 by the Genetics Society of America},
	issn = {0016-6731, 1943-2631},
	url = {https://www.genetics.org/content/211/4/1125},
	doi = {10.1534/genetics.118.301594},
	abstract = {The genetics and evolution of complex traits, including quantitative traits and disease, have been hotly debated ever since Darwin. A century ago, a paper from R.A. Fisher reconciled Mendelian and biometrical genetics in a landmark contribution that is now accepted as the main foundation stone of the field of quantitative genetics. Here, we give our perspective on Fisher’s 1918 paper in the context of how and why it is relevant in today’s genome era. We mostly focus on human trait variation, in part because Fisher did so too, but the conclusions are general and extend to other natural populations, and to populations undergoing artificial selection.},
	language = {en},
	number = {4},
	urldate = {2020-05-05},
	journal = {Genetics},
	author = {Visscher, Peter M. and Goddard, Michael E.},
	month = apr,
	year = {2019},
	pmid = {30967441},
	note = {Publisher: Genetics
Section: Perspectives},
	keywords = {Fisher 1918, GWAS, quantitative trait},
	pages = {1125--1130},
}

@article{frs_has_1936,
	title = {Has {Mendel}'s work been rediscovered?},
	volume = {1},
	issn = {0003-3790},
	url = {https://doi.org/10.1080/00033793600200111},
	doi = {10.1080/00033793600200111},
	number = {2},
	urldate = {2020-05-05},
	journal = {Annals of Science},
	author = {F.R.S, R. A. Fisher M. A. Sc D.},
	month = apr,
	year = {1936},
	note = {Publisher: Taylor \& Francis
\_eprint: https://doi.org/10.1080/00033793600200111},
	pages = {115--137},
}

@book{bowler_evolution_2003,
	title = {Evolution : the history of an idea},
	shorttitle = {Evolution},
	url = {http://archive.org/details/evolutionhistory0000bowl_n7y8},
	abstract = {xix, 464 p. : 23 cm; Includes bibliographical references (p. 383-449) and index},
	language = {eng},
	urldate = {2020-05-05},
	publisher = {Berkeley : University of California Press},
	author = {Bowler, Peter J.},
	collaborator = {{Internet Archive}},
	year = {2003},
	keywords = {Evolution (Biology) -- History},
}

@article{mendel_experiments_nodate,
	title = {{EXPERIMENTS} {IN} {PLANT} {HYBRIDIZATION} (1865)},
	language = {en},
	author = {Mendel, Gregor},
	pages = {41},
}

@article{cui_spatial_2016,
	title = {A spatial {SIS} model in advective heterogeneous environments},
	volume = {261},
	issn = {0022-0396},
	url = {http://www.sciencedirect.com/science/article/pii/S0022039616301103},
	doi = {10.1016/j.jde.2016.05.025},
	abstract = {We study the effects of diffusion and advection for a susceptible-infected-susceptible epidemic reaction–diffusion model in heterogeneous environments. The definition of the basic reproduction number R0 is given. If R0{\textless}1, the unique disease-free equilibrium (DFE) is globally asymptotically stable. Asymptotic behaviors of R0 for advection rate and mobility of the infected individuals (denoted by dI) are established, and the existence of the endemic equilibrium when R0{\textgreater}1 is studied. The effects of diffusion and advection rates on the stability of the DFE are further investigated. Among other things, we find that if the habitat is a low-risk domain, there may exist one critical value for the advection rate, under which the DFE changes its stability at least twice as dI varies from zero to infinity, while the DFE is unstable for any dI when the advection rate is larger than the critical value. These results are in strong contrast with the case of no advection, where the DFE changes its stability at most once as dI varies from zero to infinity.},
	language = {en},
	number = {6},
	urldate = {2020-05-05},
	journal = {Journal of Differential Equations},
	author = {Cui, Renhao and Lou, Yuan},
	month = sep,
	year = {2016},
	keywords = {Disease-free equilibrium, Endemic equilibrium, Reaction–diffusion–advection, SIS epidemic model, Spatial heterogeneity},
	pages = {3305--3343},
}

@book{henig_monk_2000,
	title = {The monk in the garden : the lost and found genius of {Gregor} {Mendel}, the father of genetics},
	shorttitle = {The monk in the garden},
	url = {http://archive.org/details/monkingardenlost00heni},
	abstract = {Includes bibliographical references (p. 278-279) and index; expanded bibliography and footnotes found only on the World Wide Web; A study of the groundbreaking work in genetics conducted by Gregor Mendel, acclaimed as the father of modern genetics, argues that the Moravian monk was far ahead of his time; Spring 1900 -- In the glasshouse -- Southern exposure -- Between science and God -- Breakdown in Vienna -- Back to the garden -- Crossings -- First harvest -- Eve's homunculus -- The flowering of Darwinism -- Garden reflections -- Full moon in February -- The silence -- "My time will come" -- Synchronicity -- Mendel redux -- The monk's bulldog -- A death in Oxford -- Inventing Mendelism -- A statue in Meldelplatz -- Another spring},
	language = {eng},
	urldate = {2020-05-05},
	publisher = {Boston : Houghton Mifflin},
	author = {Henig, Robin Marantz},
	collaborator = {{Internet Archive}},
	year = {2000},
	keywords = {Mendel, Gregor, 1822-1884},
}

@book{bateson_mendels_2013,
	title = {Mendel's {Principles} of {Heredity}},
	isbn = {978-0-486-14837-3},
	abstract = {Six years after Charles Darwin announced his theory of evolution to the world, Gregor Mendel began studying the inheritance of traits in pea plants. Mendel\&\#39;s research led to his discovery of dominant and recessive traits and other facts of evolution, which he reported in his groundbreaking 1865 paper, Experiments in Plant Hybridization. His findings languished until 1902, when William Bateson revived interest in the subject with this book, a succinct account of Mendel\&\#39;s heredity-related discoveries. Bateson coined the term \&quot;genetics\&quot; to refer to heredity and inherited traits, and his rediscovery of Mendel\&\#39;s work forms the foundation of today\&\#39;s field of genetics.Suitable for biology and general science students at the undergraduate and graduate levels, this volume is essential reading for anyone with an interest in science and genetics. In addition to Bateson\&\#39;s commentary, it features two of Mendel\&\#39;s papers—including the original Experiments—plus a biography of Mendel, a detailed bibliography, and indexes of subjects and authors. Numerous figures complement the text, along with eight pages of color illustrations.},
	language = {en},
	publisher = {Courier Corporation},
	author = {Bateson, William and Mendel, Gregor},
	month = mar,
	year = {2013},
	note = {Google-Books-ID: x7ici4OUmkYC},
	keywords = {Science / Life Sciences / Biology},
}

@book{darwin_variation_2010,
	title = {The {Variation} of {Animals} and {Plants} {Under} {Domestication}},
	isbn = {978-1-108-01423-6},
	abstract = {Charles Darwin (1809-1882) first published this work in 1868 in two volumes. The book began as an expansion of the first two chapters of On the Origin of Species: 'Variation under Domestication' and 'Variation under Nature' and it developed into one of his largest works; Darwin referred to it as his 'big book'. In volume 2, concerned with how species inherit particular characteristics, Darwin first published his 'provisional hypothesis' of pangenesis. This theory of 'gemmules' was not met with much acceptance and today is not valuable as scientific explanation, but it was important in laying down the key questions that needed to be answered regarding the processes of genetic inheritance. Darwin also used volume 2 to challenge the theories of evolution by design, expounded by the botanist Asa Gray. Darwin's arguments were some of the very first in a long debate that remains hot today.},
	language = {en},
	publisher = {Cambridge University Press},
	author = {Darwin, Charles},
	month = jun,
	year = {2010},
	note = {Google-Books-ID: \_uON0AO7qwYC},
	keywords = {Science / Life Sciences / Evolution},
}

@book{darwin_origin_1936,
	address = {New York},
	title = {The origin of species by means of natural selection: or the preservation of favored races in the struggle for life[,] and the descent of man and selection in relation to sex},
	isbn = {978-0-394-60398-8},
	shorttitle = {The origin of species by means of natural selection},
	language = {English},
	publisher = {Modern library},
	author = {Darwin, Charles},
	year = {1936},
	note = {OCLC: 352242},
}

@article{kempthorne_correlation_1968,
	title = {The correlation between relatives on the supposition of mendelian inheritance},
	volume = {20},
	issn = {0002-9297},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1706357/},
	number = {4},
	urldate = {2020-05-05},
	journal = {American Journal of Human Genetics},
	author = {Kempthorne, Oscar},
	month = jul,
	year = {1968},
	pmid = {null},
	pmcid = {PMC1706357},
	pages = {402--403},
}

@book{lamarck_zoological_1914,
	title = {Zoological philosophy;},
	copyright = {http://creativecommons.org/publicdomain/zero/1.0/},
	url = {http://archive.org/details/ZoologicalPhilosophy},
	abstract = {an exposition with regard to the natural history of animals, with an introduction by Hugh Elliot.  From the Brittle Books digitization program at the Ohio State University Libraries.},
	urldate = {2020-05-05},
	publisher = {London,Macmillan and Co., limited},
	author = {Lamarck, Jean Baptiste Pierre Antoine de Monet de},
	year = {1914},
	keywords = {Zoology},
}

@misc{noauthor_internet_nodate,
	title = {The {Internet} {Classics} {Archive} {\textbar} {On} {Airs}, {Waters}, and {Places} by {Hippocrates}},
	url = {http://classics.mit.edu/Hippocrates/airwatpl.14.14.html},
	urldate = {2020-05-04},
}

@article{darwin_pangenesis_1871,
	title = {Pangenesis},
	volume = {3},
	copyright = {1871 Nature Publishing Group},
	issn = {1476-4687},
	url = {https://www.nature.com/articles/003502a0},
	doi = {10.1038/003502a0},
	abstract = {IN a paper, read March 30, 1871, before the Royal Society, and just published in the Proceedings, Mr. Galton gives the results of his interesting experiments on the inter-transfusion of the blood of distinct varieties of rabbits. These experiments were undertaken to test whether there was any truth in my provisional hypothesis of Pangenesis. Mr. Galton, in recapitulating “the cardinal points,” says that the gemmules are supposed “to swarm in the blood. “He enlarges on this head, and remarks, “Under Mr. Darwin's theory, the gemmules in each individual must, therefore, be looked upon as entozoa of his blood,” \&c. Now, in the chapter on Pangenesis in my “Variation of Animals and Plants under Domestication,” I have not said one word about the blood, or about any fluid proper to any circulating system. It is, indeed, obvious that the presence of gemmules in the blood can form no necessary part of my hypothesis; for I refer in illustration of it to the lowest animals, such as the Protozoa, which do not possess blood or any vessels; and I refer to plants in which the fluid, when present in the vessels, cannot be considered as true blood. The fundamental laws of growth, reproduction, inheritance, \&c, are so closely similar throughout the whole organic kingdom, that the means by which the gemmules (assuming for the moment their existence) are diffused through the body, would probably be the same in all beings; therefore the means can hardly be diffusion through the blood. Nevertheless, when I first heard of Mr. Galton's experiments, I did not sufficiently reflect on the subject, and saw not the difficulty of believing in the presence of gemmules in the blood. I have said (Variation, \&c, vol. ii., p. 379) that “the gemmules in each organism must be thoroughly diffused; nor does this seem improbable, considering their minuteness, and the steady circulation of fluids throughout the body.” But when I used these latter words and other similar ones, I presume that I was thinking of the diffusion of the gemmules through the tissues, or from cell to cell, independently of the presence of vessels,—as in the remarkable experiments by Dr. Bence Jones, in which chemical elements absorbed by the stomach were detected in the course of some minutes in the crystalline lens of the eye; or again as in the repeated loss of colour and its recovery after a few days by the hair, in the singular case of a neuralgic lady recorded by Mr. Paget. Nor can it be objected that the gemmules could not pass through tissues or cell-walls, for the contents of each pollen-grain have to pass through the coats, both of the pollen-tube and embryonic sack. I may add, with respect to the passage of fluids through membrane, that they pass from cell to cell in the absorbing hairs of the roots of living plants at a rate, as I have myself observed under the microscope, which is truly surprising.},
	language = {en},
	number = {78},
	urldate = {2020-05-03},
	journal = {Nature},
	author = {Darwin, Charles},
	month = apr,
	year = {1871},
	note = {Number: 78
Publisher: Nature Publishing Group},
	pages = {502--503},
}

@article{holterhoff_history_2014,
	title = {The {History} and {Reception} of {Charles} {Darwin}’s {Hypothesis} of {Pangenesis}},
	volume = {47},
	issn = {1573-0387},
	url = {https://doi.org/10.1007/s10739-014-9377-0},
	doi = {10.1007/s10739-014-9377-0},
	abstract = {This paper explores Charles Darwin’s hypothesis of pangenesis through a popular and professional reception history. First published in The Variation of Animals and Plants under Domestication (1868), pangenesis stated that inheritance can be explained by sub-cellular “gemmules” which aggregated in the sexual organs during intercourse. Pangenesis thereby accounted for the seemingly arbitrary absence and presence of traits in offspring while also clarifying some botanical and invertebrates’ limb regeneration abilities. I argue that critics largely interpreted Variation as an extension of On the Origin of Species by Means of Natural Selection (1859), while pangenesis was an extension of natural selection. Contrary to claims that pangenesis was divorced from natural selection by its reliance on the inheritance of acquired characters, pangenesis’s mid nineteenth-century reception suggests that Darwin’s hypothesis responded directly to selection’s critics. Using Variation’s several editions, periodical reviews, and personal correspondence I assess pangenesis popularly, professionally, and biographically to better understand Variation’s impact on 1860s and 70s British evolutionism and inheritance.},
	language = {en},
	number = {4},
	urldate = {2020-05-03},
	journal = {Journal of the History of Biology},
	author = {Holterhoff, Kate},
	month = nov,
	year = {2014},
	pages = {661--695},
}

@article{chakravarti_perspectives_2015,
	title = {Perspectives on {Human} {Variation} through the {Lens} of {Diversity} and {Race}},
	volume = {7},
	issn = {1943-0264},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4563709/},
	doi = {10.1101/cshperspect.a023358},
	abstract = {Human populations, however defined, differ in the distribution and frequency of traits they display and diseases to which individuals are susceptible. These need to be understood with respect to three recent advances. First, these differences are multicausal and a result of not only genetic but also epigenetic and environmental factors. Second, the actions of genes, although crucial, turn out to be quite dynamic and modifiable, which contrasts with the classical view that they are inflexible machines. Third, the diverse human populations across the globe have spent too little time apart from our common origin 50,000 years ago to have developed many individually adapted traits. Human trait and disease differences by continental ancestry are thus as much the result of nongenetic as genetic forces., Humans across the globe display variation in numerous different traits. But these differences are caused by both genetic and nongenetic factors and do not define distinct “races” in biological terms.},
	number = {9},
	urldate = {2020-05-03},
	journal = {Cold Spring Harbor Perspectives in Biology},
	author = {Chakravarti, Aravinda},
	month = sep,
	year = {2015},
	pmid = {26330522},
	pmcid = {PMC4563709},
}

@article{reich_opinion_2018,
	chapter = {Opinion},
	title = {Opinion {\textbar} {How} {Genetics} {Is} {Changing} {Our} {Understanding} of ‘{Race}’},
	issn = {0362-4331},
	url = {https://www.nytimes.com/2018/03/23/opinion/sunday/genetics-race.html},
	abstract = {If scientists avoid discussing the topic candidly, racist theories will fill the vacuum.},
	language = {en-US},
	urldate = {2020-05-03},
	journal = {The New York Times},
	author = {Reich, David},
	month = mar,
	year = {2018},
	keywords = {Genetics and Heredity, Race and Ethnicity},
}

@article{zirkle_inheritance_1935,
	title = {The {Inheritance} of {Acquired} {Characters} and the {Provisional} {Hypothesis} of {Pangenesis}},
	volume = {69},
	issn = {0003-0147},
	url = {https://www.journals.uchicago.edu/doi/10.1086/280617},
	doi = {10.1086/280617},
	abstract = {1. Lamarck was neither the first nor the most distinguished biologist to believe in the inheritance of acquired characters. He merely endorsed a belief which had been generally accepted for at least 2,200 years before his time and used it to explain how evolution could have taken place. The inheritance of acquired characters had been accepted previously by Hippocrates, Aristotle, Galen (?), Roger Bacon, Jerome Cardan, Levinus Lemnius, John Ray, Michael Adanson, Jo. Fried. Blumenbach and Erasmus Darwin among others. 2. If we wish to trace the history of evolution, we should search for naturalists who lived before Lamarck and who did not believe in the inheritance of acquired characters. Brock listed but two, (1) the unknown editor of Aristotle's "Historia Animalium," and (2) the philosopher, Immanuel Kant. 3. The dogma of the immutability of species met with general acceptance only late in the eighteenth and early in the nineteenth century. Botanists from the time of Theophrastos to the time of Linnaeus believed in "degeneration." "Degeneration" did not then have its modern meaning, but was synonymous with De Vriesian mutation. Belief in degeneration was not incompatible with belief in special creation, as its effects were supposed to be neither orderly nor cumulative. 4. In order to understand what Charles Darwin meant by his provisional hypothesis of pangenesis, it is necessary that we do not give to the terms he used the meanings which they acquired during the twentieth century. Darwin's conception of the germ-plasm was not the one currently accepted. He considered the theory that the foetus was produced from an egg fertilized by a single spermatozoan but rejected it. He believed that the whole semen was a fertilizing substance and that the foetus resembled the father in proportion to the amount of semen ejaculated in coition. Furthermore, he believed in telegony and in the specific influence of semen upon the mother's body. 5. The hypothesis of pangenesis is as old as the belief in the inheritance of acquired characters. It was endorsed by Hippocrates, Democritus, Galen (?), Clement of Alexandria, Lactantius, St. Isidore of Seville, Bartholomeus Anglicus, St. Albert the Great, St. Thomas of Aquinas, Peter of Crescentius (?), Paracelsus, Jerome Cardan, Levinus Lemnius, Venette, John Ray, Buffon, Bonnet, Maupertius, von Haller and Herbert Spencer. A careful search of our available records should add a number of names to this list.},
	number = {724},
	urldate = {2020-05-03},
	journal = {The American Naturalist},
	author = {Zirkle, Conway},
	month = sep,
	year = {1935},
	pages = {417--445},
}

@misc{noauthor_pii_nodate,
	title = {{PII}: 0092-8674(87)90348-5 {\textbar} {Elsevier} {Enhanced} {Reader}},
	shorttitle = {{PII}},
	url = {https://reader.elsevier.com/reader/sd/pii/0092867487903485?token=8A77DF07C4D2B69D5A58030EC5E05BAD86FF7BBEEBF037F22154D3764460AE04914CEA5A8FC0D1B38AF44857B9AF108B},
	language = {en},
	urldate = {2020-05-02},
	doi = {10.1016/0092-8674(87)90348-5},
}

@article{clayton_mutation_1955,
	title = {Mutation and {Quantitative} {Variation}},
	volume = {89},
	issn = {0003-0147},
	url = {https://www.jstor.org/stable/2458343},
	abstract = {Selection for abdominal chaetae has been carried out in an inbred line of D. melanogaster, both with and without irradiation of 1800 r of X-rays each generation. The response in the control stocks in 17 generations was not significant. The irradiated lines responded to selection but slowly compared with wild populations. This is discussed in relation to the results of other workers. Two papers by Mather and co-workers are found to give consistent estimates of the rate of spontaneous production of new variance in abdominal chaetae of the order of 0.01 units each generation, which is not inconsistent with our results. The variance found in several wild populations is about 5 units. The evolutionary aspect of these results is discussed.},
	number = {846},
	urldate = {2020-05-02},
	journal = {The American Naturalist},
	author = {Clayton, G. and Robertson, Alan},
	year = {1955},
	pages = {151--158},
}

@article{clayton_mutation_1955-1,
	title = {Mutation and {Quantitative} {Variation}},
	volume = {89},
	issn = {0003-0147},
	url = {https://www.jstor.org/stable/2458343},
	abstract = {Selection for abdominal chaetae has been carried out in an inbred line of D. melanogaster, both with and without irradiation of 1800 r of X-rays each generation. The response in the control stocks in 17 generations was not significant. The irradiated lines responded to selection but slowly compared with wild populations. This is discussed in relation to the results of other workers. Two papers by Mather and co-workers are found to give consistent estimates of the rate of spontaneous production of new variance in abdominal chaetae of the order of 0.01 units each generation, which is not inconsistent with our results. The variance found in several wild populations is about 5 units. The evolutionary aspect of these results is discussed.},
	number = {846},
	urldate = {2020-05-02},
	journal = {The American Naturalist},
	author = {Clayton, G. and Robertson, Alan},
	year = {1955},
	pages = {151--158},
}

@article{clayton_mutation_1955-2,
	title = {Mutation and {Quantitative} {Variation}},
	volume = {89},
	issn = {0003-0147},
	url = {https://www.jstor.org/stable/2458343},
	abstract = {Selection for abdominal chaetae has been carried out in an inbred line of D. melanogaster, both with and without irradiation of 1800 r of X-rays each generation. The response in the control stocks in 17 generations was not significant. The irradiated lines responded to selection but slowly compared with wild populations. This is discussed in relation to the results of other workers. Two papers by Mather and co-workers are found to give consistent estimates of the rate of spontaneous production of new variance in abdominal chaetae of the order of 0.01 units each generation, which is not inconsistent with our results. The variance found in several wild populations is about 5 units. The evolutionary aspect of these results is discussed.},
	number = {846},
	urldate = {2020-05-02},
	journal = {The American Naturalist},
	author = {Clayton, G. and Robertson, Alan},
	year = {1955},
	pages = {151--158},
}

@misc{noauthor_heads_nodate,
	title = {heads up - chrisporras1@uchicago.edu - {The} {University} of {Chicago} {Mail}},
	url = {https://mail.google.com/mail/u/1/#inbox/FMfcgxwGDWvBCXbVPDJDbZzfjrtPFGSL},
	urldate = {2020-02-27},
}

@misc{noauthor_heads_nodate-1,
	title = {heads up - chrisporras1@uchicago.edu - {The} {University} of {Chicago} {Mail}},
	url = {https://mail.google.com/mail/u/1/#inbox/FMfcgxwGDWvBCXbVPDJDbZzfjrtPFGSL},
	urldate = {2020-02-27},
}

@misc{noauthor_heads_nodate-2,
	title = {heads up - chrisporras1@uchicago.edu - {The} {University} of {Chicago} {Mail}},
	url = {https://mail.google.com/mail/u/1/#inbox/FMfcgxwGDWvBCXbVPDJDbZzfjrtPFGSL},
	urldate = {2020-02-27},
}

@misc{noauthor_debunked_nodate,
	title = {{DEBUNKED}: {Obama}'s {Lies} {About} {Trump} {Economy} {\textbar} {Louder} with {Crowder}},
	shorttitle = {{DEBUNKED}},
	url = {https://www.youtube.com/watch?v=2wly3eAr6Ko},
	abstract = {Steven debunks all the leftist lies about Trump's booming economy.

Want to watch the full show every day? Join \#MugClub! http://louderwithcrowder.com/mugclub

Use promo codes "student" "veteran" "military" to get daily access for \$69/year!

Shop the official \#LWC store: http://louderwithcrowdershop.com

Follow me on Twitter: https://twitter.com/scrowder
Like me on Facebook: https://www.facebook.com/stevencrowder 

Find behind the scenes footage on instagram: http://www.instagram.com/louderwithcr...},
	urldate = {2020-02-20},
}

@misc{noauthor_notitle_nodate,
	url = {https://3000-d2deb77d-c0bd-4692-a19a-bce6da133fce.ws-us02.gitpod.io/payment/results?input_apr=4.35&input_years=30&input_pv=235000},
	urldate = {2020-02-03},
}

@misc{noauthor_nomethoderror_nodate,
	title = {{NoMethodError} at /flexible/square/5},
	url = {https://3000-d2deb77d-c0bd-4692-a19a-bce6da133fce.ws-us02.gitpod.io/flexible/square/5},
	urldate = {2020-02-03},
}

@misc{noauthor_hyuxley_nodate,
	title = {hyuxley modern synthesis - {Google} {Search}},
	url = {https://www.google.com/search?q=hyuxley+modern+synthesis&rlz=1C1CHBF_enUS802US802&oq=hyuxley+modern+synthesis&aqs=chrome..69i57j0l4.2705j0j1&sourceid=chrome&ie=UTF-8},
	urldate = {2020-01-22},
}

@misc{noauthor_hyuxley_nodate-1,
	title = {hyuxley modern synthesis - {Google} {Search}},
	url = {https://www.google.com/search?q=hyuxley+modern+synthesis&rlz=1C1CHBF_enUS802US802&oq=hyuxley+modern+synthesis&aqs=chrome..69i57j0l4.2705j0j1&sourceid=chrome&ie=UTF-8},
	urldate = {2020-01-22},
}

@misc{noauthor_hyuxley_nodate-2,
	title = {hyuxley modern synthesis - {Google} {Search}},
	url = {https://www.google.com/search?q=hyuxley+modern+synthesis&rlz=1C1CHBF_enUS802US802&oq=hyuxley+modern+synthesis&aqs=chrome..69i57j0l4.2705j0j1&sourceid=chrome&ie=UTF-8},
	urldate = {2020-01-22},
}

@misc{noauthor_hyuxley_nodate-3,
	title = {hyuxley modern synthesis - {Google} {Search}},
	url = {https://www.google.com/search?q=hyuxley+modern+synthesis&rlz=1C1CHBF_enUS802US802&oq=hyuxley+modern+synthesis&aqs=chrome..69i57j0l4.2705j0j1&sourceid=chrome&ie=UTF-8},
	urldate = {2020-01-22},
}

@misc{noauthor_hyuxley_nodate-4,
	title = {hyuxley modern synthesis - {Google} {Search}},
	url = {https://www.google.com/search?q=hyuxley+modern+synthesis&rlz=1C1CHBF_enUS802US802&oq=hyuxley+modern+synthesis&aqs=chrome..69i57j0l4.2705j0j1&sourceid=chrome&ie=UTF-8},
	urldate = {2020-01-22},
}

@article{soneji_association_2019,
	title = {Association of {Maternal} {Cigarette} {Smoking} and {Smoking} {Cessation} {With} {Preterm} {Birth}},
	volume = {2},
	issn = {2574-3805},
	doi = {10.1001/jamanetworkopen.2019.2514},
	abstract = {Importance: Cigarette smoking during pregnancy increases the risk of preterm birth, low birth weight, and infant mortality.
Objective: To assess the probability of preterm birth among expectant mothers who smoked cigarettes before pregnancy and quit smoking at the start or during pregnancy.
Design, Setting, and Participants: This cross-sectional study analyzed information provided on live birth certificates from 2011 through 2017 that were obtained from US states that implemented the 2003 revision of the US live birth certificate. In total, 25 233 503 expectant mothers who delivered live neonates and had known prepregnancy and trimester-specific cigarette smoking frequency were included in the analyses.
Exposures: Cigarette smoking frequency (1-9, 10-19, and ≥20 cigarettes per day) 3 months prior to pregnancy and for each trimester during pregnancy.
Main Outcomes and Measures: Cigarette smoking cessation throughout pregnancy, after the first trimester, after the second trimester, and during the third trimester irrespective of first and second trimester smoking. Probability of preterm birth ({\textless}37 weeks' gestation).
Results: Of 25 233 503 expectant mothers who delivered live neonates between 2011 and 2017, the modal age at delivery was 25 to 29 years; 52.9\% were non-Hispanic white, 23.6\% were Hispanic, and 14.2\% were non-Hispanic black women; 22 600 196 mothers did not smoke during the 3 months prior to pregnancy, and 2 633 307 smoked during the 3 months prior to pregnancy. The proportion of prepregnancy smokers who quit throughout pregnancy was 24.3\% in 2011 and 24.6\% in 2017. The proportion of prepregnancy smokers who quit during the third trimester was 39.5\% in 2011 and 39.7\% in 2017. High-frequency cigarette smoking often occurred among expectant mothers who smoked during pregnancy (eg, 46.9\% of third-trimester smokers smoked ≥10 cigarettes per day in 2017). The probability of preterm birth decreased more the earlier smoking cessation occurred in pregnancy. For example, the probability of preterm birth was 9.8\% (95\% CI, 9.7\%-10.0\%) among 25- to 29-year-old, non-Hispanic white, primigravida and primiparous expectant mothers (ie, pregnant for the first time and not yet delivered) who smoked 1 to 9 cigarettes per day prior to pregnancy and maintained this frequency throughout their pregnancy. The probability of preterm birth was 9.0\% (95\% CI, 8.8\%-9.1\%) if smoking cessation occurred at the start of the second trimester (an 8.9\% relative decrease), and 7.8\% (95\% CI, 7.7\%-8.0\%) if cessation occurred at the start of pregnancy (a 20.3\% relative decrease).
Conclusions and Relevance: Quitting smoking-and quitting early in pregnancy-was associated with reduced risk of preterm birth even for high-frequency cigarette smokers.},
	language = {eng},
	number = {4},
	journal = {JAMA network open},
	author = {Soneji, Samir and Beltrán-Sánchez, Hiram},
	month = apr,
	year = {2019},
	pmid = {31002320},
	pmcid = {PMC6481448},
	pages = {e192514},
}

@article{shipton_reliability_2009,
	title = {Reliability of self reported smoking status by pregnant women for estimating smoking prevalence: a retrospective, cross sectional study},
	volume = {339},
	issn = {0959-8138},
	shorttitle = {Reliability of self reported smoking status by pregnant women for estimating smoking prevalence},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2771076/},
	doi = {10.1136/bmj.b4347},
	abstract = {Objective To determine what impact reliance on self reported smoking status during pregnancy has on both the accuracy of smoking prevalence figures and access to smoking cessation services for pregnant women in Scotland., Design Retrospective, cross sectional study of cotinine measurements in stored blood samples., Participants Random sample (n=3475) of the 21 029 pregnant women in the West of Scotland who opted for second trimester prenatal screening over a one year period., Main outcome measure Smoking status validated with cotinine measurement by maternal area deprivation category (Scottish Index of Multiple Deprivation)., Results Reliance on self reported smoking status underestimated true smoking by 25\% (1046/3475 (30\%) from cotinine measurement v 839/3475 (24\%) from self reporting, z score 8.27, P{\textless}0.001). Projected figures suggest that in Scotland more than 2400 pregnant smokers go undetected each year. A greater proportion of smokers in the least deprived areas (deprivation categories 1+2) did not report their smoking (39\%) compared with women in the most deprived areas (22\% in deprivation categories 4+5), but, because smoking was far more common in the most deprived areas (706 (40\%) in deprived areas compared with 142 (14\%) in affluent areas), projected figures for Scotland suggest that twice as many women in the most deprived areas are undetected (n=1196) than in the least deprived areas (n=642)., Conclusion Reliance on self reporting to identify pregnant smokers significantly underestimates the number of pregnant smokers in Scotland and results in a failure to detect over 2400 smokers each year who are therefore not offered smoking cessation services.},
	urldate = {2019-12-10},
	journal = {The BMJ},
	author = {Shipton, Deborah and Tappin, David M and Vadiveloo, Thenmalar and Crossley, Jennifer A and Aitken, David A and Chalmers, Jim},
	month = oct,
	year = {2009},
	pmid = {19875845},
	pmcid = {PMC2771076},
}

@article{rocha_respiratory_2018,
	title = {Respiratory {Care} for the {Ventilated} {Neonate}},
	volume = {2018},
	issn = {1198-2241},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6110042/},
	doi = {10.1155/2018/7472964},
	abstract = {Invasive ventilation is often necessary for the treatment of newborn infants with respiratory insufficiency. The neonatal patient has unique physiological characteristics such as small airway caliber, few collateral airways, compliant chest wall, poor airway stability, and low functional residual capacity. Pathologies affecting the newborn's lung are also different from many others observed later in life. Several different ventilation modes and strategies are available to optimize mechanical ventilation and to prevent ventilator-induced lung injury. Important aspects to be considered in ventilating neonates include the use of correct sized endotracheal tube to minimize airway resistance and work of breathing, positioning of the patient, the nursing care, respiratory kinesiotherapy, sedation and analgesia, and infection prevention, namely, the ventilator-associated pneumonia and nosocomial infection, as well as prevention and treatment of complications such as air leaks and pulmonary hemorrhage. Aspects of ventilation in patients under ECMO (extracorporeal membrane oxygenation) and in palliative care are of increasing interest nowadays. Online pulmonary mechanics and function testing as well as capnography are becoming more commonly used. Echocardiography is now a routine in most neonatal units. Near infrared spectroscopy (NIRS) is an attractive tool potentially helping in preventing intraventricular hemorrhage and periventricular leukomalacia. Lung ultrasound is an emerging tool of diagnosis and can be of added value in helping monitoring the ventilated neonate. The aim of this scientific literature review is to address relevant aspects concerning the respiratory care and monitoring of the invasively ventilated newborn in order to help physicians to optimize the efficacy of care.},
	urldate = {2019-12-10},
	journal = {Canadian Respiratory Journal},
	author = {Rocha, Gustavo and Soares, Paulo and Gonçalves, Américo and Silva, Ana Isabel and Almeida, Diana and Figueiredo, Sara and Pissarra, Susana and Costa, Sandra and Soares, Henrique and Flôr-de-Lima, Filipa and Guimarães, Hercília},
	month = aug,
	year = {2018},
	pmid = {30186538},
	pmcid = {PMC6110042},
}

@article{kahn_reexamination_2002,
	title = {A reexamination of smoking before, during, and after pregnancy},
	volume = {92},
	issn = {0090-0036},
	doi = {10.2105/ajph.92.11.1801},
	abstract = {OBJECTIVES: This study examined the patterns and correlates of maternal smoking before, during, and after pregnancy.
METHODS: We examined socioeconomic, demographic, and clinical risk factors associated with maternal smoking in a nationally representative cohort of women (n = 8285) who were surveyed 17 +/- 5 months and again 35 +/- 5 months after delivery.
RESULTS: Smoking rates among women with a college degree decreased 30\% from before pregnancy to 35 months postpartum but did not change among the least educated women. Risk factors clustered, and a gradient linked the number of risk factors (0, 2, 4) to the percentage smoking (6\%, 31\%, 58\%, P {\textless}.0001).
CONCLUSIONS: The period of pregnancy and early parenthood is associated with worsening education-related disparities in smoking as well as substantial clustering of risk factors. These observations could influence the targeting and design of maternal smoking interventions.},
	language = {eng},
	number = {11},
	journal = {American Journal of Public Health},
	author = {Kahn, Robert S. and Certain, Laura and Whitaker, Robert C.},
	month = nov,
	year = {2002},
	pmid = {12406812},
	pmcid = {PMC1447332},
	keywords = {Adult, Behavioral Risk Factor Surveillance System, Cohort Studies, Depression, Family Characteristics, Female, Health Behavior, Humans, Mothers, Population Surveillance, Pregnancy, Pregnancy Complications, Pregnant Women, Prevalence, Probability, Recurrence, Risk Factors, Smoking, Smoking Cessation, Socioeconomic Factors, Surveys and Questionnaires, United States},
	pages = {1801--1808},
}

@article{mcevoy_pulmonary_2017,
	title = {“{Pulmonary} {Effects} of {Maternal} {Smoking} on the {Fetus} and {Child}: {Effects} on {Lung} {Development}, {Respiratory} {Morbidities}, and {Life} {Long} {Lung} {Health}”},
	volume = {21},
	issn = {1526-0542},
	shorttitle = {“{Pulmonary} {Effects} of {Maternal} {Smoking} on the {Fetus} and {Child}},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5303131/},
	doi = {10.1016/j.prrv.2016.08.005},
	abstract = {Maternal smoking during pregnancy is the largest preventable cause of abnormal in-utero lung development. Despite well known risks, rates of smoking during pregnancy have only slightly decreased over the last ten years, with rates varying from 5-40\% worldwide resulting in tens of millions of fetal exposures. Despite multiple approaches to smoking cessation about 50\% of smokers will continue to smoke during pregnancy. Maternal genotype plays an important role in the likelihood of continued smoking during pregnancy and the degree to which maternal smoking will affect the fetus. The primary effects of maternal smoking on offspring lung function and health are decreases in forced expiratory flows, decreased passive respiratory compliance, increased hospitalization for respiratory infections, and an increased prevalence of childhood wheeze and asthma. Nicotine appears to be the responsible component of tobacco smoke that affects lung development, and some of the effects of maternal smoking on lung development can be prevented by supplemental vitamin C. Because nicotine is the key agent for affecting lung development, e-cigarette usage during pregnancy is likely to be as dangerous to fetal lung development as is maternal smoking.},
	urldate = {2019-12-10},
	journal = {Paediatric respiratory reviews},
	author = {McEvoy, Cindy T. and Spindel, Eliot R.},
	month = jan,
	year = {2017},
	pmid = {27639458},
	pmcid = {PMC5303131},
	pages = {27--33},
}

@misc{noauthor_zotero_nodate,
	title = {Zotero {\textbar} {Downloads}},
	url = {https://www.zotero.org/download/},
	urldate = {2019-10-08},
}

@misc{noauthor_zotero_nodate-1,
	title = {Zotero {\textbar} {Downloads}},
	url = {https://www.zotero.org/download/},
	urldate = {2019-10-08},
}

@article{noauthor_initial_2001,
	title = {Initial sequencing and analysis of the human genome},
	volume = {409},
	copyright = {2001 Macmillan Magazines Ltd.},
	issn = {1476-4687},
	url = {https://www.nature.com/articles/35057062},
	doi = {10.1038/35057062},
	abstract = {The human genome holds an extraordinary trove of information about human development, physiology, medicine and evolution. Here we report the results of an international collaboration to produce and make freely available a draft sequence of the human genome. We also present an initial analysis of the data, describing some of the insights that can be gleaned from the sequence.},
	language = {En},
	number = {6822},
	urldate = {2019-08-01},
	journal = {Nature},
	month = feb,
	year = {2001},
	pages = {860},
}

@article{lawrence_discovery_2014,
	title = {Discovery and saturation analysis of cancer genes across 21 tumor types},
	volume = {505},
	issn = {0028-0836},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4048962/},
	doi = {10.1038/nature12912},
	abstract = {While a few cancer genes are mutated in a high proportion of tumors of a given type ({\textgreater}20\%), most are mutated at intermediate frequencies (2–20\%). To explore the feasibility of creating a comprehensive catalog of cancer genes, we analyzed somatic point mutations in exome sequence from 4,742 tumor-normal pairs across 21 cancer types. We found that large-scale genomic analysis can identify nearly all known cancer genes in these tumor types. Our analysis also identified 33 genes not previously known to be significantly mutated, including genes related to proliferation, apoptosis, genome stability, chromatin regulation, immune evasion, RNA processing and protein homeostasis. Down-sampling analysis indicates that larger sample sizes will reveal many more genes, mutated at clinically important frequencies. We estimate that near-saturation may be achieved with 600–5000 samples per tumor type, depending on background mutation rate. The results help guide the next stage of cancer genomics.},
	number = {7484},
	urldate = {2019-08-01},
	journal = {Nature},
	author = {Lawrence, Michael S. and Stojanov, Petar and Mermel, Craig H. and Garraway, Levi A. and Golub, Todd R. and Meyerson, Matthew and Gabriel, Stacey B. and Lander, Eric S. and Getz, Gad},
	month = jan,
	year = {2014},
	pmid = {24390350},
	pmcid = {PMC4048962},
	pages = {495--501},
}

@article{do_multiple_2015,
	title = {Multiple rare alleles at {LDLR} and {APOA5} confer risk for early-onset myocardial infarction},
	volume = {518},
	issn = {0028-0836},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4319990/},
	doi = {10.1038/nature13917},
	abstract = {Myocardial infarction (MI), a leading cause of death around the world, displays a complex pattern of inheritance,. When MI occurs early in life, the role of inheritance is substantially greater. Previously, rare mutations in low-density lipoprotein (LDL) genes have been shown to contribute to MI risk in individual families– whereas common variants at more than 45 loci have been associated with MI risk in the population–. Here, we evaluate the contribution of rare mutations to MI risk in the population. We sequenced the protein-coding regions of 9,793 genomes from patients with MI at an early age (≤50 years in males and ≤60 years in females) along with MI-free controls. We identified two genes where rare coding-sequence mutations were more frequent in cases versus controls at exome-wide significance. At low-density lipoprotein receptor (LDLR), carriers of rare, damaging mutations (3.1\% of cases versus 1.3\% of controls) were at 2.4-fold increased risk for MI; carriers of null alleles at LDLR were at even higher risk (13-fold difference). This sequence-based estimate of the proportion of early MI cases due to LDLR mutations is remarkably similar to an estimate made more than 40 years ago using total cholesterol. At apolipoprotein A-V (APOA5), carriers of rare nonsynonymous mutations (1.4\% of cases versus 0.6\% of controls) were at 2.2-fold increased risk for MI. When compared with non-carriers, LDLR mutation carriers had higher plasma LDL cholesterol whereas APOA5 mutation carriers had higher plasma triglycerides. Recent evidence has connected MI risk with coding sequence mutations at two genes functionally related to APOA5, namely lipoprotein lipase, and apolipoprotein C3,. When combined, these observations suggest that, beyond LDL cholesterol, disordered metabolism of triglyceride-rich lipoproteins contributes to MI risk.},
	number = {7537},
	urldate = {2019-08-01},
	journal = {Nature},
	author = {Do, Ron and Stitziel, Nathan O. and Won, Hong-Hee and Jørgensen, Anders Berg and Duga, Stefano and Merlini, Pier Angelica and Kiezun, Adam and Farrall, Martin and Goel, Anuj and Zuk, Or and Guella, Illaria and Asselta, Rosanna and Lange, Leslie A. and Peloso, Gina M. and Auer, Paul L. and Girelli, Domenico and Martinelli, Nicola and Farlow, Deborah N. and DePristo, Mark A. and Roberts, Robert and Stewart, Alexander F.R. and Saleheen, Danish and Danesh, John and Epstein, Stephen E. and Sivapalaratnam, Suthesh and Hovingh, G. Kees and Kastelein, John J. and Samani, Nilesh J. and Schunkert, Heribert and Erdmann, Jeanette and Shah, Svati H. and Kraus, William E. and Davies, Robert and Nikpay, Majid and Johansen, Christopher T. and Wang, Jian and Hegele, Robert A. and Hechter, Eliana and Marz, Winfried and Kleber, Marcus E. and Huang, Jie and Johnson, Andrew D. and Li, Mingyao and Burke, Greg L. and Gross, Myron and Liu, Yongmei and Assimes, Themistocles L. and Heiss, Gerardo and Lange, Ethan M. and Folsom, Aaron R. and Taylor, Herman A. and Olivieri, Oliviero and Hamsten, Anders and Clarke, Robert and Reilly, Dermot F. and Yin, Wu and Rivas, Manuel A. and Donnelly, Peter and Rossouw, Jacques E. and Psaty, Bruce M. and Herrington, David M. and Wilson, James G. and Rich, Stephen S. and Bamshad, Michael J. and Tracy, Russell P. and Cupples, L. Adrienne and Rader, Daniel J. and Reilly, Muredach P. and Spertus, John A. and Cresci, Sharon and Hartiala, Jaana and Tang, W.H. Wilson and Hazen, Stanley L. and Allayee, Hooman and Reiner, Alex P. and Carlson, Christopher S. and Kooperberg, Charles and Jackson, Rebecca D. and Boerwinkle, Eric and Lander, Eric S. and Schwartz, Stephen M. and Siscovick, David S. and McPherson, Ruth and Tybjaerg-Hansen, Anne and Abecasis, Goncalo R. and Watkins, Hugh and Nickerson, Deborah A. and Ardissino, Diego and Sunyaev, Shamil R. and O’Donnell, Christopher J. and Altshuler, David and Gabriel, Stacey and Kathiresan, Sekar},
	month = feb,
	year = {2015},
	pmid = {25487149},
	pmcid = {PMC4319990},
	pages = {102--106},
}

@article{novembre_technical_nodate,
	title = {Technical {Report}: {Assessment} of the genetic analyses of {Rasmussen} et al. (2015)},
	language = {en},
	author = {Novembre, John and Witonsky, David and Rienzo, Anna Di},
	pages = {22},
}

@article{preston_lost_1997,
	title = {{THE} {LOST} {MAN}},
	issn = {0028-792X},
	url = {https://www.newyorker.com/magazine/1997/06/16/the-lost-man},
	abstract = {A REPORTER AT LARGE about the discovery of an ancient skeleton in Kennewick, Washington... On Sunday, July 28, 1996, in the middle of the afternoon, two …},
	language = {en},
	urldate = {2019-07-23},
	author = {Preston, Douglas},
	month = jun,
	year = {1997},
	keywords = {anthropology, archeology, bering strait},
}

@article{rasmussen_ancestry_2015,
	title = {The ancestry and affiliations of {Kennewick} {Man}},
	volume = {523},
	copyright = {2015 Nature Publishing Group},
	issn = {1476-4687},
	url = {https://www.nature.com/articles/nature14625},
	doi = {10.1038/nature14625},
	abstract = {Kennewick Man, referred to as the Ancient One by Native Americans, is a male human skeleton discovered in Washington state (USA) in 1996 and initially radiocarbon dated to 8,340–9,200 calibrated years before present (bp)1. His population affinities have been the subject of scientific debate and legal controversy. Based on an initial study of cranial morphology it was asserted that Kennewick Man was neither Native American nor closely related to the claimant Plateau tribes of the Pacific Northwest, who claimed ancestral relationship and requested repatriation under the Native American Graves Protection and Repatriation Act (NAGPRA). The morphological analysis was important to judicial decisions that Kennewick Man was not Native American and that therefore NAGPRA did not apply. Instead of repatriation, additional studies of the remains were permitted2. Subsequent craniometric analysis affirmed Kennewick Man to be more closely related to circumpacific groups such as the Ainu and Polynesians than he is to modern Native Americans2. In order to resolve Kennewick Man’s ancestry and affiliations, we have sequenced his genome to ∼1× coverage and compared it to worldwide genomic data including for the Ainu and Polynesians. We find that Kennewick Man is closer to modern Native Americans than to any other population worldwide. Among the Native American groups for whom genome-wide data are available for comparison, several seem to be descended from a population closely related to that of Kennewick Man, including the Confederated Tribes of the Colville Reservation (Colville), one of the five tribes claiming Kennewick Man. We revisit the cranial analyses and find that, as opposed to genome-wide comparisons, it is not possible on that basis to affiliate Kennewick Man to specific contemporary groups. We therefore conclude based on genetic comparisons that Kennewick Man shows continuity with Native North Americans over at least the last eight millennia.},
	language = {en},
	number = {7561},
	urldate = {2019-07-23},
	journal = {Nature},
	author = {Rasmussen, Morten and Sikora, Martin and Albrechtsen, Anders and Korneliussen, Thorfinn Sand and Moreno-Mayar, J. Víctor and Poznik, G. David and Zollikofer, Christoph P. E. and Ponce de León, Marcia S. and Allentoft, Morten E. and Moltke, Ida and Jónsson, Hákon and Valdiosera, Cristina and Malhi, Ripan S. and Orlando, Ludovic and Bustamante, Carlos D. and Stafford Jr, Thomas W. and Meltzer, David J. and Nielsen, Rasmus and Willerslev, Eske},
	month = jul,
	year = {2015},
	pages = {455--458},
}

@article{preston_kennewicks_1997,
	title = {Kennewick's {Message} of {Unification}},
	volume = {38},
	issn = {1556-3502},
	url = {https://anthrosource.onlinelibrary.wiley.com/doi/abs/10.1111/an.1997.38.9.2.1},
	doi = {10.1111/an.1997.38.9.2.1},
	language = {en},
	number = {9},
	urldate = {2019-07-23},
	journal = {Anthropology News},
	author = {Preston, Douglas},
	year = {1997},
	pages = {2--2},
}

@article{chatters_recovery_2000,
	title = {The {Recovery} and {First} {Analysis} of an {Early} {Holocene} {Human} {Skeleton} from {Kennewick}, {Washington}},
	volume = {65},
	issn = {0002-7316},
	url = {https://www.jstor.org/stable/2694060},
	doi = {10.2307/2694060},
	abstract = {The nearly-complete, well-preserved skeleton of a Paleoamerican male was found by chance near Kennewick, Washington, in 1996. Although analysis was quickly suspended by the U.S. government, initial osteological, archaeological, and geological studies provide a glimpse into the age and life of this individual. A radiocarbon age of 8410 ± 60 B.P., stratigraphic position in a widely-dated alluvial terrace, and an early-Cascade style projectile point healed into the pelvis date the find to the late Early Holocene. Initial osteological analysis describes the man as middle-aged, standing 173.1 ± 3.6 cm tall and weighing approximately 70-75 kg. Healthy as a child, he later suffered repeatedly from injuries to his skull, left arm, chest, and hip, in addition to minor osteoarthritis and periodontal disease. His physical features, teeth, and skeletal measurements show him to be an outlier relative to modern human populations, but place him closer to Pacific Islanders and Ainu than to Late Prehistoric Amerinds or any other modern group. Despite his uniqueness relative to modern peoples, he is not significantly different from other Paleoamerican males in most characteristics.},
	number = {2},
	urldate = {2019-07-23},
	journal = {American Antiquity},
	author = {Chatters, James C.},
	year = {2000},
	pages = {291--316},
}

@article{boyko_assessing_2008,
	title = {Assessing the {Evolutionary} {Impact} of {Amino} {Acid} {Mutations} in the {Human} {Genome}},
	volume = {4},
	issn = {1553-7404},
	url = {https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000083},
	doi = {10.1371/journal.pgen.1000083},
	abstract = {Quantifying the distribution of fitness effects among newly arising mutations in the human genome is key to resolving important debates in medical and evolutionary genetics. Here, we present a method for inferring this distribution using Single Nucleotide Polymorphism (SNP) data from a population with non-stationary demographic history (such as that of modern humans). Application of our method to 47,576 coding SNPs found by direct resequencing of 11,404 protein coding-genes in 35 individuals (20 European Americans and 15 African Americans) allows us to assess the relative contribution of demographic and selective effects to patterning amino acid variation in the human genome. We find evidence of an ancient population expansion in the sample with African ancestry and a relatively recent bottleneck in the sample with European ancestry. After accounting for these demographic effects, we find strong evidence for great variability in the selective effects of new amino acid replacing mutations. In both populations, the patterns of variation are consistent with a leptokurtic distribution of selection coefficients (e.g., gamma or log-normal) peaked near neutrality. Specifically, we predict 27–29\% of amino acid changing (nonsynonymous) mutations are neutral or nearly neutral ({\textbar}s{\textbar}{\textless}0.01\%), 30–42\% are moderately deleterious (0.01\%{\textless}{\textbar}s{\textbar}{\textless}1\%), and nearly all the remainder are highly deleterious or lethal ({\textbar}s{\textbar}{\textgreater}1\%). Our results are consistent with 10–20\% of amino acid differences between humans and chimpanzees having been fixed by positive selection with the remainder of differences being neutral or nearly neutral. Our analysis also predicts that many of the alleles identified via whole-genome association mapping may be selectively neutral or (formerly) positively selected, implying that deleterious genetic variation affecting disease phenotype may be missed by this widely used approach for mapping genes underlying complex traits.},
	language = {en},
	number = {5},
	urldate = {2019-07-10},
	journal = {PLOS Genetics},
	author = {Boyko, Adam R. and Williamson, Scott H. and Indap, Amit R. and Degenhardt, Jeremiah D. and Hernandez, Ryan D. and Lohmueller, Kirk E. and Adams, Mark D. and Schmidt, Steffen and Sninsky, John J. and Sunyaev, Shamil R. and White, Thomas J. and Nielsen, Rasmus and Clark, Andrew G. and Bustamante, Carlos D.},
	month = may,
	year = {2008},
	keywords = {African American people, Deletion mutation, Europe, Evolutionary genetics, Gamma spectrometry, Molecular genetics, Mutation, Natural selection},
	pages = {e1000083},
}

@article{eyre-walker_distribution_2007,
	title = {The distribution of fitness effects of new mutations},
	volume = {8},
	copyright = {2007 Nature Publishing Group},
	issn = {1471-0064},
	url = {https://www.nature.com/articles/nrg2146},
	doi = {10.1038/nrg2146},
	abstract = {The distribution of fitness effects (DFE) of new mutations is a fundamental entity in genetics that has implications ranging from the genetic basis of complex disease to the stability of the molecular clock. It has been studied by two different approaches: mutation accumulation and mutagenesis experiments, and the analysis of DNA sequence data. The proportion of mutations that are advantageous, effectively neutral and deleterious varies between species, and the DFE differs between coding and non-coding DNA. Despite these differences between species and genomic regions, some general principles have emerged: advantageous mutations are rare, and those that are strongly selected are exponentially distributed; and the DFE of deleterious mutations is complex and multi-modal.},
	language = {en},
	number = {8},
	urldate = {2019-07-10},
	journal = {Nature Reviews Genetics},
	author = {Eyre-Walker, Adam and Keightley, Peter D.},
	month = aug,
	year = {2007},
	pages = {610--618},
}

@techreport{battey_space_2019,
	type = {preprint},
	title = {Space is the {Place}: {Effects} of {Continuous} {Spatial} {Structure} on {Analysis} of {Population} {Genetic} {Data}},
	shorttitle = {Space is the {Place}},
	url = {http://biorxiv.org/lookup/doi/10.1101/659235},
	abstract = {ABSTRACT
          Real geography is continuous, but standard models in population genetics are based on discrete, well-mixed populations. As a result many methods of analyzing genetic data assume that samples are a random draw from a well-mixed population, but are applied to clustered samples from populations that are structured clinally over space. Here we use simulations of populations living in continuous geography to study the impacts of dispersal and sampling strategy on population genetic summary statistics, demographic inference, and genome-wide association studies. We find that most common summary statistics have distributions that differ substantially from that seen in well-mixed populations, especially when Wright’s neighborhood size is less than 100 and sampling is spatially clustered. The combination of low dispersal and clustered sampling causes demographic inference from the site frequency spectrum to infer more turbulent demographic histories, but averaged results across multiple simulations were surprisingly robust to isolation by distance. We also show that the combination of spatially autocorrelated environments and limited dispersal causes genome-wide association studies to identify spurious signals of genetic association with purely environmentally determined phenotypes, and that this bias is only partially corrected by regressing out principal components of ancestry. Last, we discuss the relevance of our simulation results for inference from genetic variation in real organisms.},
	language = {en},
	urldate = {2019-07-10},
	institution = {Genetics},
	author = {Battey, C.J. and Ralph, Peter L. and Kern, Andrew D.},
	month = jun,
	year = {2019},
	doi = {10.1101/659235},
}

@article{visscher_r.._2019,
	title = {From {R}.{A}. {Fisher}’s 1918 {Paper} to {GWAS} a {Century} {Later}},
	volume = {211},
	copyright = {Copyright © 2019 by the Genetics Society of America},
	issn = {0016-6731, 1943-2631},
	url = {https://www.genetics.org/content/211/4/1125},
	doi = {10.1534/genetics.118.301594},
	abstract = {The genetics and evolution of complex traits, including quantitative traits and disease, have been hotly debated ever since Darwin. A century ago, a paper from R.A. Fisher reconciled Mendelian and biometrical genetics in a landmark contribution that is now accepted as the main foundation stone of the field of quantitative genetics. Here, we give our perspective on Fisher’s 1918 paper in the context of how and why it is relevant in today’s genome era. We mostly focus on human trait variation, in part because Fisher did so too, but the conclusions are general and extend to other natural populations, and to populations undergoing artificial selection.},
	language = {en},
	number = {4},
	urldate = {2019-07-10},
	journal = {Genetics},
	author = {Visscher, Peter M. and Goddard, Michael E.},
	month = apr,
	year = {2019},
	pmid = {30967441},
	keywords = {Fisher 1918, GWAS, quantitative trait},
	pages = {1125--1130},
}

@article{marouli_rare_2017,
	title = {Rare and low-frequency coding variants alter human adult height},
	volume = {542},
	issn = {0028-0836},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5302847/},
	doi = {10.1038/nature21039},
	abstract = {Height is a highly heritable, classic polygenic trait with ∼700 common associated variants identified so far through genome-wide association studies. Here, we report 83 height-associated coding variants with lower minor allele frequencies (range of 0.1-4.8\%) and effects of up to 2 cm/allele (e.g. in IHH, STC2, AR and CRISPLD2), {\textgreater}10 times the average effect of common variants. In functional follow-up studies, rare height-increasing alleles of STC2 (+1-2 cm/allele) compromised proteolytic inhibition of PAPP-A and increased cleavage of IGFBP-4 in vitro, resulting in higher bioavailability of insulin-like growth factors. These 83 height-associated variants overlap genes mutated in monogenic growth disorders and highlight new biological candidates (e.g. ADAMTS3, IL11RA, NOX4) and pathways (e.g. proteoglycan/glycosaminoglycan synthesis) involved in growth. Our results demonstrate that sufficiently large sample sizes can uncover rare and low-frequency variants of moderate to large effect associated with polygenic human phenotypes, and that these variants implicate relevant genes and pathways.},
	number = {7640},
	urldate = {2019-07-10},
	journal = {Nature},
	author = {Marouli, Eirini and Graff, Mariaelisa and Medina-Gomez, Carolina and Lo, Ken Sin and Wood, Andrew R and Kjaer, Troels R and Fine, Rebecca S and Lu, Yingchang and Schurmann, Claudia and Highland, Heather M and Rüeger, Sina and Thorleifsson, Gudmar and Justice, Anne E and Lamparter, David and Stirrups, Kathleen E and Turcot, Valérie and Young, Kristin L and Winkler, Thomas W and Esko, Tõnu and Karaderi, Tugce and Locke, Adam E and Masca, Nicholas GD and Ng, Maggie CY and Mudgal, Poorva and Rivas, Manuel A and Vedantam, Sailaja and Mahajan, Anubha and Guo, Xiuqing and Abecasis, Goncalo and Aben, Katja K and Adair, Linda S and Alam, Dewan S and Albrecht, Eva and Allin, Kristine H and Allison, Matthew and Amouyel, Philippe and Appel, Emil V and Arveiler, Dominique and Asselbergs, Folkert W and Auer, Paul L and Balkau, Beverley and Banas, Bernhard and Bang, Lia E and Benn, Marianne and Bergmann, Sven and Bielak, Lawrence F and Blüher, Matthias and Boeing, Heiner and Boerwinkle, Eric and Böger, Carsten A and Bonnycastle, Lori L and Bork-Jensen, Jette and Bots, Michiel L and Bottinger, Erwin P and Bowden, Donald W and Brandslund, Ivan and Breen, Gerome and Brilliant, Murray H and Broer, Linda and Burt, Amber A and Butterworth, Adam S and Carey, David J and Caulfield, Mark J and Chambers, John C and Chasman, Daniel I and Chen, Yii-Der Ida and Chowdhury, Rajiv and Christensen, Cramer and Chu, Audrey Y and Cocca, Massimiliano and Collins, Francis S and Cook, James P and Corley, Janie and Galbany, Jordi Corominas and Cox, Amanda J and Cuellar-Partida, Gabriel and Danesh, John and Davies, Gail and de Bakker, Paul IW and de Borst, Gert J. and de Denus, Simon and de Groot, Mark CH and de Mutsert, Renée and Deary, Ian J and Dedoussis, George and Demerath, Ellen W and den Hollander, Anneke I and Dennis, Joe G and Di Angelantonio, Emanuele and Drenos, Fotios and Du, Mengmeng and Dunning, Alison M and Easton, Douglas F and Ebeling, Tapani and Edwards, Todd L and Ellinor, Patrick T and Elliott, Paul and Evangelou, Evangelos and Farmaki, Aliki-Eleni and Faul, Jessica D and Feitosa, Mary F and Feng, Shuang and Ferrannini, Ele and Ferrario, Marco M and Ferrieres, Jean and Florez, Jose C and Ford, Ian and Fornage, Myriam and Franks, Paul W and Frikke-Schmidt, Ruth and Galesloot, Tessel E and Gan, Wei and Gandin, Ilaria and Gasparini, Paolo and Giedraitis, Vilmantas and Giri, Ayush and Girotto, Giorgia and Gordon, Scott D and Gordon-Larsen, Penny and Gorski, Mathias and Grarup, Niels and Grove, Megan L. and Gudnason, Vilmundur and Gustafsson, Stefan and Hansen, Torben and Harris, Kathleen Mullan and Harris, Tamara B and Hattersley, Andrew T and Hayward, Caroline and He, Liang and Heid, Iris M and Heikkilä, Kauko and Helgeland, Øyvind and Hernesniemi, Jussi and Hewitt, Alex W and Hocking, Lynne J and Hollensted, Mette and Holmen, Oddgeir L and Hovingh, G. Kees and Howson, Joanna MM and Hoyng, Carel B and Huang, Paul L and Hveem, Kristian and Ikram, M. Arfan and Ingelsson, Erik and Jackson, Anne U and Jansson, Jan-Håkan and Jarvik, Gail P and Jensen, Gorm B and Jhun, Min A and Jia, Yucheng and Jiang, Xuejuan and Johansson, Stefan and Jørgensen, Marit E and Jørgensen, Torben and Jousilahti, Pekka and Jukema, J Wouter and Kahali, Bratati and Kahn, René S and Kähönen, Mika and Kamstrup, Pia R and Kanoni, Stavroula and Kaprio, Jaakko and Karaleftheri, Maria and Kardia, Sharon LR and Karpe, Fredrik and Kee, Frank and Keeman, Renske and Kiemeney, Lambertus A and Kitajima, Hidetoshi and Kluivers, Kirsten B and Kocher, Thomas and Komulainen, Pirjo and Kontto, Jukka and Kooner, Jaspal S and Kooperberg, Charles and Kovacs, Peter and Kriebel, Jennifer and Kuivaniemi, Helena and Küry, Sébastien and Kuusisto, Johanna and La Bianca, Martina and Laakso, Markku and Lakka, Timo A and Lange, Ethan M and Lange, Leslie A and Langefeld, Carl D and Langenberg, Claudia and Larson, Eric B and Lee, I-Te and Lehtimäki, Terho and Lewis, Cora E and Li, Huaixing and Li, Jin and Li-Gao, Ruifang and Lin, Honghuang and Lin, Li-An and Lin, Xu and Lind, Lars and Lindström, Jaana and Linneberg, Allan and Liu, Yeheng and Liu, Yongmei and Lophatananon, Artitaya and Luan, Jian'an and Lubitz, Steven A and Lyytikäinen, Leo-Pekka and Mackey, David A and Madden, Pamela AF and Manning, Alisa K and Männistö, Satu and Marenne, Gaëlle and Marten, Jonathan and Martin, Nicholas G and Mazul, Angela L and Meidtner, Karina and Metspalu, Andres and Mitchell, Paul and Mohlke, Karen L and Mook-Kanamori, Dennis O and Morgan, Anna and Morris, Andrew D and Morris, Andrew P and Müller-Nurasyid, Martina and Munroe, Patricia B and Nalls, Mike A and Nauck, Matthias and Nelson, Christopher P and Neville, Matt and Nielsen, Sune F and Nikus, Kjell and Njølstad, Pål R and Nordestgaard, Børge G and Ntalla, Ioanna and O'Connel, Jeffrey R and Oksa, Heikki and Loohuis, Loes M Olde and Ophoff, Roel A and Owen, Katharine R and Packard, Chris J and Padmanabhan, Sandosh and Palmer, Colin NA and Pasterkamp, Gerard and Patel, Aniruddh P and Pattie, Alison and Pedersen, Oluf and Peissig, Peggy L and Peloso, Gina M and Pennell, Craig E and Perola, Markus and Perry, James A and Perry, John R.B. and Person, Thomas N and Pirie, Ailith and Polasek, Ozren and Posthuma, Danielle and Raitakari, Olli T and Rasheed, Asif and Rauramaa, Rainer and Reilly, Dermot F and Reiner, Alex P and Renström, Frida and Ridker, Paul M and Rioux, John D and Robertson, Neil and Robino, Antonietta and Rolandsson, Olov and Rudan, Igor and Ruth, Katherine S and Saleheen, Danish and Salomaa, Veikko and Samani, Nilesh J and Sandow, Kevin and Sapkota, Yadav and Sattar, Naveed and Schmidt, Marjanka K and Schreiner, Pamela J and Schulze, Matthias B and Scott, Robert A and Segura-Lepe, Marcelo P and Shah, Svati and Sim, Xueling and Sivapalaratnam, Suthesh and Small, Kerrin S and Smith, Albert Vernon and Smith, Jennifer A and Southam, Lorraine and Spector, Timothy D and Speliotes, Elizabeth K and Starr, John M and Steinthorsdottir, Valgerdur and Stringham, Heather M and Stumvoll, Michael and Surendran, Praveen and Hart, Leen M ‘t and Tansey, Katherine E and Tardif, Jean-Claude and Taylor, Kent D and Teumer, Alexander and Thompson, Deborah J and Thorsteinsdottir, Unnur and Thuesen, Betina H and Tönjes, Anke and Tromp, Gerard and Trompet, Stella and Tsafantakis, Emmanouil and Tuomilehto, Jaakko and Tybjaerg-Hansen, Anne and Tyrer, Jonathan P and Uher, Rudolf and Uitterlinden, André G and Ulivi, Sheila and van der Laan, Sander W and Van Der Leij, Andries R and van Duijn, Cornelia M and van Schoor, Natasja M and van Setten, Jessica and Varbo, Anette and Varga, Tibor V and Varma, Rohit and Edwards, Digna R Velez and Vermeulen, Sita H and Vestergaard, Henrik and Vitart, Veronique and Vogt, Thomas F and Vozzi, Diego and Walker, Mark and Wang, Feijie and Wang, Carol A and Wang, Shuai and Wang, Yiqin and Wareham, Nicholas J and Warren, Helen R and Wessel, Jennifer and Willems, Sara M and Wilson, James G and Witte, Daniel R and Woods, Michael O and Wu, Ying and Yaghootkar, Hanieh and Yao, Jie and Yao, Pang and Yerges-Armstrong, Laura M and Young, Robin and Zeggini, Eleftheria and Zhan, Xiaowei and Zhang, Weihua and Zhao, Jing Hua and Zhao, Wei and Zhao, Wei and Zheng, He and Zhou, Wei and Rotter, Jerome I and Boehnke, Michael and Kathiresan, Sekar and McCarthy, Mark I and Willer, Cristen J and Stefansson, Kari and Borecki, Ingrid B and Liu, Dajiang J and North, Kari E and Heard-Costa, Nancy L and Pers, Tune H and Lindgren, Cecilia M and Oxvig, Claus and Kutalik, Zoltán and Rivadeneira, Fernando and Loos, Ruth JF and Frayling, Timothy M and Hirschhorn, Joel N and Deloukas, Panos and Lettre, Guillaume},
	month = feb,
	year = {2017},
	pmid = {28146470},
	pmcid = {PMC5302847},
	pages = {186--190},
}

@article{slatkin_estimating_2000,
	title = {Estimating {Allele} {Age}},
	volume = {1},
	url = {https://doi.org/10.1146/annurev.genom.1.1.225},
	doi = {10.1146/annurev.genom.1.1.225},
	abstract = {The age of an allele can be estimated both from genetic variation among different copies (intra-allelic variation) and from its frequency. Estimates based on intra-allelic variation follow from the exponential decay of linkage disequilibrium because of recombination and mutation. The confidence interval depends both on the uncertainty in recombination and mutation rates and on randomness of the genealogy of chromosomes that carry the allele (the intra-allelic genealogy). Several approximate methods to account for variation in the intra-allelic genealogy have been derived. Allele frequency alone also provides an estimate of age. Estimates based on frequency and on intra-allelic variability can be combined to provide a more accurate estimate or can be contrasted to show that an allele has been subject to natural selection. These methods have been applied to numerous cases, including alleles associated with cystic fibrosis, idiopathic torsion dystonia, and resistance to infection by HIV. We emphasize that estimates of allele age depend on assumptions about demographic history and natural selection.},
	number = {1},
	urldate = {2019-07-10},
	journal = {Annual Review of Genomics and Human Genetics},
	author = {Slatkin, Montgomery and Rannala, Bruce},
	year = {2000},
	pmid = {11701630},
	pages = {225--249},
}

@article{slatkin_spatial_1978,
	title = {The {Spatial} {Distribution} of {Transient} {Alleles} in a {Subdivided} {Population}: {A} {Simulation} {Study}},
	volume = {89},
	issn = {0016-6731},
	shorttitle = {The {Spatial} {Distribution} of {Transient} {Alleles} in a {Subdivided} {Population}},
	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1213868/},
	abstract = {The spatial distributions of newly introducted alleles in a subdivided population are generated using a computer program to model the processes of selection, gene flow and genetic drift. Advantageous, neutral and deleterious alleles are considered, and certain aspects of the patterns generated by new alleles that are ultimately fixed and ultimately lost are examined. To characterize the spatial pattern of rare alleles, the distribution, Pi, the probability that the new allele is found in exactly i local populations before it is lost, is defined and estimated from the simulations. The shape of the Pi distribution is surprisingly similar for selected and neutral alleles. For advantageous alleles going to fixation, the "wave of advance" is set up quickly, but stochastic effects reduce the wave speed from Fisher's (1937) value. Gene flow is much more effective in dispersing alleles in a two-dimensional array than in one dimension. Long distance gene flow has a much smaller effect in two dimensions than in one dimension.},
	number = {4},
	urldate = {2019-07-10},
	journal = {Genetics},
	author = {Slatkin, Montgomery and Charlesworth, Deborah},
	month = aug,
	year = {1978},
	pmid = {17248852},
	pmcid = {PMC1213868},
	pages = {793--810},
}

@article{kim_inference_2017,
	title = {Inference of the {Distribution} of {Selection} {Coefficients} for {New} {Nonsynonymous} {Mutations} {Using} {Large} {Samples}},
	volume = {206},
	copyright = {Copyright © 2017 by the Genetics Society of America. Available freely online through the author-supported open access option.},
	issn = {0016-6731, 1943-2631},
	url = {https://www.genetics.org/content/206/1/345},
	doi = {10.1534/genetics.116.197145},
	abstract = {The distribution of fitness effects (DFE) has considerable importance in population genetics. To date, estimates of the DFE come from studies using a small number of individuals. Thus, estimates of the proportion of moderately to strongly deleterious new mutations may be unreliable because such variants are unlikely to be segregating in the data. Additionally, the true functional form of the DFE is unknown, and estimates of the DFE differ significantly between studies. Here we present a flexible and computationally tractable method, called Fit∂a∂i, to estimate the DFE of new mutations using the site frequency spectrum from a large number of individuals. We apply our approach to the frequency spectrum of 1300 Europeans from the Exome Sequencing Project ESP6400 data set, 1298 Danes from the LuCamp data set, and 432 Europeans from the 1000 Genomes Project to estimate the DFE of deleterious nonsynonymous mutations. We infer significantly fewer (0.38–0.84 fold) strongly deleterious mutations with selection coefficient {\textbar}s{\textbar} {\textgreater} 0.01 and more (1.24–1.43 fold) weakly deleterious mutations with selection coefficient {\textbar}s{\textbar} {\textless} 0.001 compared to previous estimates. Furthermore, a DFE that is a mixture distribution of a point mass at neutrality plus a gamma distribution fits better than a gamma distribution in two of the three data sets. Our results suggest that nearly neutral forces play a larger role in human evolution than previously thought.},
	language = {en},
	number = {1},
	urldate = {2019-07-10},
	journal = {Genetics},
	author = {Kim, Bernard Y. and Huber, Christian D. and Lohmueller, Kirk E.},
	month = may,
	year = {2017},
	pmid = {28249985},
	keywords = {deleterious mutations, diffusion theory, population genetics, site frequency spectrum},
	pages = {345--361},
}

@techreport{ragsdale_perspective:_2018,
	type = {preprint},
	title = {Perspective: {Genomic} inference using diffusion models and the allele frequency spectrum},
	shorttitle = {Perspective},
	url = {http://biorxiv.org/lookup/doi/10.1101/375048},
	abstract = {Evolutionary, biological, and demographic processes combine to shape the variation observed in populations. Understanding how these processes are expected to inﬂuence variation allows us to infer past demographic events and the nature of selection in human populations. Forward models such as the diﬀusion approximation provide a powerful tool for analyzing the distribution of allele frequencies in contemporary populations due to their computational tractability and model ﬂexibility. Here, we discuss recent computational developments and their application to reconstructing human demographic history and patterns of selection at new mutations. We also reexamine how some classical assumptions that are still commonly used in inference studies fare when applied to modern data. We use whole-genome sequence data for 797 French Canadian individuals to examine the neutrality of synonymous sites. We ﬁnd that selection can lead to strong biases in the inferred demography, mutation rate, and distributions of ﬁtness eﬀects. We use these distributions of ﬁtness eﬀects together with demographic and phenotype-ﬁtness models to predict the relationship between eﬀect size and allele frequency, and contrast those predictions to commonly used models in statistical genetics. Thus the simple evolutionary models investigated by Kimura and Ohta still provide important insight into modern genetic research.},
	language = {en},
	urldate = {2019-07-10},
	institution = {Genetics},
	author = {Ragsdale, Aaron P and Moreau, Claudia and Gravel, Simon},
	month = jul,
	year = {2018},
	doi = {10.1101/375048},
}

@article{mazet_importance_2016,
	title = {On the importance of being structured: instantaneous coalescence rates and human evolution—lessons for ancestral population size inference?},
	volume = {116},
	copyright = {2015 Nature Publishing Group},
	issn = {1365-2540},
	shorttitle = {On the importance of being structured},
	url = {https://www.nature.com/articles/hdy2015104},
	doi = {10.1038/hdy.2015.104},
	abstract = {Most species are structured and influenced by processes that either increased or reduced gene flow between populations. However, most population genetic inference methods assume panmixia and reconstruct a history characterized by population size changes. This is potentially problematic as population structure can generate spurious signals of population size change through time. Moreover, when the model assumed for demographic inference is misspecified, genomic data will likely increase the precision of misleading if not meaningless parameters. For instance, if data were generated under an n-island model (characterized by the number of islands and migrants exchanged) inference based on a model of population size change would produce precise estimates of a bottleneck that would be meaningless. In addition, archaeological or climatic events around the bottleneck’s timing might provide a reasonable but potentially misleading scenario. In a context of model uncertainty (panmixia versus structure) genomic data may thus not necessarily lead to improved statistical inference. We consider two haploid genomes and develop a theory that explains why any demographic model with structure will necessarily be interpreted as a series of changes in population size by inference methods ignoring structure. We formalize a parameter, the inverse instantaneous coalescence rate, and show that it is equivalent to a population size only in panmictic models, and is mostly misleading for structured models. We argue that this issue affects all population genetics methods ignoring population structure which may thus infer population size changes that never took place. We apply our approach to human genomic data.},
	language = {en},
	number = {4},
	urldate = {2019-07-10},
	journal = {Heredity},
	author = {Mazet, O. and Rodríguez, W. and Grusea, S. and Boitard, S. and Chikhi, L.},
	month = apr,
	year = {2016},
	pages = {362--371},
}

@article{lapierre_accuracy_2017,
	title = {Accuracy of {Demographic} {Inferences} from the {Site} {Frequency} {Spectrum}: {The} {Case} of the {Yoruba} {Population}},
	volume = {206},
	copyright = {Copyright © 2017 by the Genetics Society of America},
	issn = {0016-6731, 1943-2631},
	shorttitle = {Accuracy of {Demographic} {Inferences} from the {Site} {Frequency} {Spectrum}},
	url = {https://www.genetics.org/content/206/1/439},
	doi = {10.1534/genetics.116.192708},
	abstract = {Some methods for demographic inference based on the observed genetic diversity of current populations rely on the use of summary statistics such as the Site Frequency Spectrum (SFS). Demographic models can be either model-constrained with numerous parameters, such as growth rates, timing of demographic events, and migration rates, or model-flexible, with an unbounded collection of piecewise constant sizes. It is still debated whether demographic histories can be accurately inferred based on the SFS. Here, we illustrate this theoretical issue on an example of demographic inference for an African population. The SFS of the Yoruba population (data from the 1000 Genomes Project) is fit to a simple model of population growth described with a single parameter (e.g., founding time). We infer a time to the most recent common ancestor of 1.7 million years (MY) for this population. However, we show that the Yoruba SFS is not informative enough to discriminate between several different models of growth. We also show that for such simple demographies, the fit of one-parameter models outperforms the stairway plot, a recently developed model-flexible method. The use of this method on simulated data suggests that it is biased by the noise intrinsically present in the data.},
	language = {en},
	number = {1},
	urldate = {2019-07-10},
	journal = {Genetics},
	author = {Lapierre, Marguerite and Lambert, Amaury and Achaz, Guillaume},
	month = may,
	year = {2017},
	pmid = {28341655},
	keywords = {coalescent theory, human demography, model identifiability, site frequency spectrum},
	pages = {439--449},
}